{"cells":[{"cell_type":"markdown","id":"d234c639","metadata":{},"source":["# UKBiobank\n","\n","* **Project:** ADRD Genetic Diversity in Biobanks\n","* **Version:** Python/3.9 and 3.10\n","* **Last Updated:** 22-August-2024\n","\n","## Notebook Overview\n","Fetch cohorts, ancestry info, remove related individuals, filter non-WGS, combine pVCFs, normalize VCFs, annotate, allele freqs, APOE genotyping, phenotype data, resilience/protective variants"]},{"cell_type":"markdown","id":"9b69a02d","metadata":{},"source":["# Initialize Notebook"]},{"cell_type":"markdown","id":"b11a388c-5ca1-4097-91a5-29e8dc6bd559","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Import packages"]},{"cell_type":"code","execution_count":1,"id":"cd14b439-3a34-4910-96d6-fc7afaede06f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["import pyspark\n","import dxdata\n","import dxpy\n","import pandas as pd\n","from datetime import date, datetime\n","import os \n","import numpy as np\n","import random\n","import shutil\n","import glob\n","import requests\n","from functools import reduce\n","\n","sc = pyspark.SparkContext()\n","spark = pyspark.sql.SparkSession(sc)\n"]},{"cell_type":"markdown","id":"a579174a-8b5a-488d-af45-aa9ed13205d2","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Initialize variables"]},{"cell_type":"code","execution_count":2,"id":"adff6de3-eef6-4029-ae05-b3905bf83d43","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["gene_names = [\n","    \"APOE\",\n","    \"APP\",\n","    \"GBA1\",\n","    \"GBA1LP\",\n","    \"GRN\",\n","    \"MAPT\",\n","    \"PSEN1\",\n","    \"PSEN2\",\n","    \"SNCA\",\n","    \"TREM2\",\n","]\n"]},{"cell_type":"markdown","id":"2efc43ef-80dd-4450-9c64-98735c8a2216","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Initialize helper functions"]},{"cell_type":"code","execution_count":3,"id":"3c7822be-2a15-455a-ab3f-28a8211ab69b","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["def fetch_gene_info_ensembl(gene_names, species='human', genome_version='GRCh38'):\n","    gene_info_dict = {}\n","    server = \"https://rest.ensembl.org\"\n","    \n","    for gene_name in gene_names:\n","        endpoint = f\"/lookup/symbol/{species}/{gene_name}\"\n","        headers = {\"Content-Type\": \"application/json\"}\n","\n","        response = requests.get(server + endpoint, headers=headers, params={\"expand\": \"1\"})\n","        if not response.ok:\n","            print(f\"Fetching failed for {gene_name}\")\n","            continue\n","\n","        data = response.json()\n","        gene_info = {\n","            \"gene_name\": data.get(\"display_name\", gene_name),\n","            \"chromosome\": f\"chr{data['seq_region_name']}\",\n","            \"start\": int(data[\"start\"]),\n","            \"end\": int(data[\"end\"]),\n","            \"genome_version\": genome_version\n","        }\n","\n","        gene_info_dict[gene_name] = gene_info\n","\n","    return gene_info_dict\n"]},{"cell_type":"markdown","id":"6db4eb5c-7b62-4cca-9413-3f60389debfa","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Fetch cohorts"]},{"cell_type":"markdown","id":"4759399f-16f7-4135-b76d-309603968270","metadata":{"tags":[]},"source":["## Grab the dataset containing participant information"]},{"cell_type":"code","execution_count":4,"id":"cbfbf832-1d99-434a-9390-ede2b7245c42","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["dispensed_dataset_id = dxpy.find_one_data_object(typename=\"Dataset\", name=\"app*.dataset\", folder=\"/\", name_mode=\"glob\")[\"id\"]\n","dataset = dxdata.load_dataset(id=dispensed_dataset_id)\n","participant = dataset[\"participant\"]\n"]},{"cell_type":"markdown","id":"9ea3457f-503d-4b56-9aed-43ca4255495d","metadata":{"tags":[]},"source":["## Retrieve Cases"]},{"cell_type":"markdown","id":"afaebbea-2258-4d46-b4f0-1c7eda2e1a25","metadata":{"tags":[]},"source":["### AD + Non-AD Dementia Cohorts"]},{"cell_type":"markdown","id":"b85518c4-29ea-41cf-88d5-aee787f51bda","metadata":{"tags":[]},"source":["#### Pull down the fields we need \n","https://docs.google.com/document/d/1AebkQ-Nxrk63jhsDzZpn5QD-7EK4unsykHVj-saEm3U/edit?usp=sharing"]},{"cell_type":"code","execution_count":5,"id":"f917dc60-cc2c-4991-9732-5b632c39258f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["field_names = [\n","    \"eid\", \n","    \"p31\", \n","    \"p34\", \n","    \"p22189\", \n","    \"p22006\", \n","    \"p21022\", \n","    \"p42020\", \n","    \"p42018\", \n","    \"p22009_a1\", \n","    \"p22009_a2\", \n","    \"p22009_a3\", \n","    \"p22009_a4\", \n","    \"p22009_a5\", \n","    \"p40000_i0\",\n","    \"p120042\",\n","    \"p26302_i2\",\n","    \"p26302_i3\",\n","    \"p21625_i2\",\n","    \"p21625_i3\",\n","    \"p62_i0\",\n","    \"p62_i1\",\n","    \"p62_i2\",\n","    #\"p32104\",\n","    #\"p32105\",\n","    #\"p32106\",\n","    #\"p32107\",\n","    #\"p32108\",\n","    #\"p32109\",\n","]\n","cases_df = participant.retrieve_fields(names=field_names, coding_values=\"replace\", engine=dxdata.connect())\n","cases_df = cases_df.toPandas()\n"]},{"cell_type":"markdown","id":"04b0abf1-b83d-49ae-8513-d95afee0e91a","metadata":{"tags":[]},"source":["#### Rename columns to be human-readable"]},{"cell_type":"code","execution_count":6,"id":"f6709363-3dd9-4265-a3c0-324eab83c72f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["cases_df = cases_df.rename(columns={\n","    'eid':'ID',\n","    'p31':'GENETIC_SEX', \n","    'p34':'BIRTH_YEAR', \n","    'p22189':'TOWNSEND', \n","    'p22006':'ETHNICITY', \n","    'p21022':'AGE_OF_RECRUIT',\n","    'p42020':'AD_DATE',\n","    'p42018':'DEM_DATE',\n","    'p22009_a1':'PC1',\n","    'p22009_a2':'PC2',\n","    'p22009_a3':'PC3',\n","    'p22009_a4':'PC4',\n","    'p22009_a5':'PC5',\n","    'p40000_i0':'DATE_OF_DEATH',\n","    \"p120042\":\"COGNITIVE_SYMPTOMS_SEVERITY_PAST_WEEK\",\n","    \"p26302_i2\":\"SPECIFIC_COGNITIVE_ABILITY_2014\",\n","    \"p26302_i3\":\"SPECIFIC_COGNITIVE_ABILITY_2019\",\n","    \"p21625_i2\":\"TOUCHSCREEN_COGNITIVE_DURATION_2014\",\n","    \"p21625_i3\":\"TOUCHSCREEN_COGNITIVE_DURATION_2019\",\n","    \"p62_i0\":\"COGNITIVE_TEST_WILLINGNESS\",\n","    \"p62_i1\":\"COGNITIVE_TEST_WILLINGNESS\",\n","    \"p62_i2\":\"COGNITIVE_TEST_WILLINGNESS\",\n","    #\"p32104\":\"FORGETFULNESS_PAST_WEEK\",\n","    #\"p32105\":\"POOR_CONCENTRATION_PAST_WEEK\",\n","    #\"p32106\":\"TROUBLE_EXPRESSING_THOUGHTS_PAST_WEEK\",\n","    #\"p32107\":\"TROUBLE_FINDING_RIGHT_WORD_PAST_WEEK\",\n","    #\"p32108\":\"SLOW_THINKING_SPEED_PAST_WEEK\",\n","    #\"p32109\":\"TROUBLE_SOLVING_PROBLEMS_PAST_WEEK\",\n","})\n"]},{"cell_type":"markdown","id":"70b05066-05f2-460d-9566-22f8c2d06f01","metadata":{"tags":[]},"source":["#### Find participants with AD and RD"]},{"cell_type":"code","execution_count":7,"id":"4fe6d7fc-01b8-401f-a7f4-584ca57cdeef","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["# AD:\n","ad_df = cases_df[~cases_df[f'AD_DATE'].isna()]\n","ad_df = ad_df[[\n","    'ID', 'GENETIC_SEX', 'BIRTH_YEAR', 'TOWNSEND', 'ETHNICITY', 'AGE_OF_RECRUIT', \n","    f'AD_DATE', 'PC1', 'PC2', 'PC3', 'PC4', 'PC5', 'DATE_OF_DEATH', \n","    \"COGNITIVE_SYMPTOMS_SEVERITY_PAST_WEEK\", \"SPECIFIC_COGNITIVE_ABILITY_2014\",\n","    \"SPECIFIC_COGNITIVE_ABILITY_2019\", \"TOUCHSCREEN_COGNITIVE_DURATION_2014\",\n","    \"TOUCHSCREEN_COGNITIVE_DURATION_2019\", \"COGNITIVE_TEST_WILLINGNESS\",\n","    \"COGNITIVE_TEST_WILLINGNESS\", \"COGNITIVE_TEST_WILLINGNESS\",\n","]]\n","ad_df[\"ID\"] = pd.to_numeric(ad_df[\"ID\"])\n","    \n","# RD:\n","rd_df = cases_df[cases_df['AD_DATE'].isna() & ~cases_df['DEM_DATE'].isna()]\n","rd_df = rd_df[[\n","    'ID', 'GENETIC_SEX', 'BIRTH_YEAR', 'TOWNSEND', 'ETHNICITY', 'AGE_OF_RECRUIT', \n","    'DEM_DATE', 'PC1', 'PC2', 'PC3', 'PC4', 'PC5', 'DATE_OF_DEATH', \n","    \"COGNITIVE_SYMPTOMS_SEVERITY_PAST_WEEK\", \"SPECIFIC_COGNITIVE_ABILITY_2014\",\n","    \"SPECIFIC_COGNITIVE_ABILITY_2019\", \"TOUCHSCREEN_COGNITIVE_DURATION_2014\",\n","    \"TOUCHSCREEN_COGNITIVE_DURATION_2019\", \"COGNITIVE_TEST_WILLINGNESS\",\n","    \"COGNITIVE_TEST_WILLINGNESS\", \"COGNITIVE_TEST_WILLINGNESS\",\n","]]\n","rd_df[\"ID\"] = pd.to_numeric(rd_df[\"ID\"])\n"]},{"cell_type":"markdown","id":"eba6b23a-ece6-40c2-9ec1-71b88e992e48","metadata":{"tags":[]},"source":["## Retrieve Controls"]},{"cell_type":"markdown","id":"96fb1da9-2893-45de-b9f2-93f45f7cc63c","metadata":{"tags":[]},"source":["### Age 65+, no NDD, no parents with AD or PD"]},{"cell_type":"markdown","id":"8c8baa9a-cdf5-4aae-bfa3-c2123f2d9f26","metadata":{"tags":[]},"source":["#### Retrieve field names of interest for each participant"]},{"cell_type":"code","execution_count":8,"id":"e8bfdd82-c980-45fb-9051-8fb3106f1f39","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["# Date G10 first reported (huntington's disease),\n","# Date D11 first reported (hereditary ataxia), \n","# Date G12 first reported (spinal muscular atrophy and related syndromes), \n","# Date G13 first reported (systemic atrophies primarily affecting central nervous system in diseases classified elswhere), \n","# Date G14 first reported (postpolio syndrome), \n","# Date G20 first reported (parkinson's disease), \n","# Date G21 first reported (secondary parkinsonism), \n","# Date G22 first reported (parkinsonism in diseases classified elsewhere), \n","# Date G23 first reported (other degenerative diseases of basal ganglia), \n","# Date G24 first reported (dystonia), \n","# Date G25 first reported (other extrapyramidal and movement disorders), \n","# Date G30 first reported (alzheimer's disease), \n","# Date G31 first reported (other degenerative diseases of nervous system, not elsewhere classified), \n","# Date G32 first reported (other degenerative disorders of nervous system in diseases classified elsewhere), \n","# Date G35 first reported (multiple sclerosis), \n","# Date G36 first reported (other acute disseminated demyelination), \n","# Date G37 first reported (other demyelinating diseases of central nervous system), \n","# Date G45 first reported (transient cerebral ischaemic attacks and related syndromes), \n","# Date G46 first reported (vascular syndromes of brain in cerebrovascular diseases), \n","# Date G50 first reported (disorders of trigeminal nerve), \n","# Date G52 first reported (disorders of other cranial nerves), \n","# Date G53 first reported (cranial nerve disorders in diseases classified elsewhere), \n","# Date G54 first reported (nerve root and plexus disorders), \n","# Date G55 first reported (nerve root and plexus compressions in diseases classified elsewhere), \n","# Date G56 first reported (mononeuropathies of upper limb), \n","# Date G57 first reported (mononeuropathies of lower limb), \n","# Date G58 first reported (other mononeuropathies), \n","# Date G59 first reported (mononeuropathy in diseases classified elsewhere), \n","# Date G60 first reported (hereditary and idiopathic neuropathy), \n","# Date G61 first reported (inflammatory polyneuropathy), \n","# Date G62 first reported (other polyneuropathies), \n","# Date G63 first reported (polyneuropathy in diseases classified elsewhere), \n","# Date G64 first reported (other disorders of peripheral nervous system), \n","# Date G70 first reported (myasthenia gravis and other myoneural disorders), \n","# Date G71 first reported (primary disorders of muscles), \n","# Date G72 first reported (other myopathies), \n","# Date G73 first reported (disorders of myoneural junction and muscle in diseases classified elsewhere), \n","# Date G80 first reported (infantile cerebral palsy), \n","# Date G81 first reported (hemiplegia), \n","# Date G82 first reported (paraplegia and tetraplegia), \n","# Date G83 first reported (other paralytic syndromes), \n","# Date G90 first reported (disorders of autonomic nervous system),\n","# Date G91 first reported (hydrocephalus), \n","# Date G92 first reported (toxic encephalopathy), \n","# Date G93 first reported (other disorders of brain), \n","# Date G94 first reported (other disorders of brain in diseases classified elsewhere), \n","# Date G96 first reported (other disorders of central nervous system), \n","# Date G97 first reported (postprocedural disorders of nervous system, not elsewhere classified),  \n","# Date G98 first reported (other disorders of nervous system, not elsewhere classified), \n","# Date G99 first reported (other disorders of nervous system in diseases classified elsewhere), \n","# Date of all cause dementia report, \n","# Date of alzheimer's disease report, \n","# Date of vascular dementia report, \n","# Date of frontotemporal dementia report, \n","# Date of motor neurone disease report, \n","# Date of all cause parkinsonism report, \n","# Date of parkinson's disease report, \n","# Date of progressive supranuclear palsy report, \n","# Date of multiple system atrophy report, \n","# Genetic ethnic grouping, \n","# Age at recruitment, \n","# Townsend deprivation index at recruitment, \n","# Sex, \n","# Genetic Principal components | Array 1, \n","# Genetic Principal components | Array 2, \n","# Genetic Principal components | Array 3, \n","# Genetic Principal components | Array 4, \n","# Genetic Principal components | Array 5\n","\n","field_names = ['eid', 'p131012', 'p131016', 'p131018', 'p131020', 'p131022', 'p131024', 'p131026', 'p131028', 'p131030', 'p131036', 'p131038', 'p131040', 'p131042',\n","               'p131046', 'p131056', 'p131058', 'p131062', 'p131066', 'p131068', 'p131070', 'p131074', 'p131076', 'p131078', 'p131080', 'p131082', 'p131084', 'p131086', \n","               'p131088', 'p131090', 'p131092', 'p131094', 'p131096', 'p131098', 'p131100', 'p131102', 'p131104', 'p131106', 'p131108', 'p131110', 'p131112', 'p131114', \n","               'p131116', 'p131120', 'p131122', 'p131124', 'p131126',  'p42018', 'p42020', 'p42022', 'p42024', 'p42028', 'p42030', 'p42032', 'p42034', 'p42036', 'p22006', \n","               'p21022', 'p22189', 'p31', 'p22009_a1', 'p22009_a2', 'p22009_a3', 'p22009_a4', 'p22009_a5', 'p34', 'p40000_i0', 'p20110_i0', 'p20110_i1', 'p20110_i2', \n","               'p20110_i3', 'p20107_i0', 'p20107_i1', 'p20107_i2', 'p20107_i3']\n","control_df = participant.retrieve_fields(names=field_names, coding_values=\"replace\", engine=dxdata.connect())\n","control_df = control_df.toPandas()\n"]},{"cell_type":"markdown","id":"236a6214-b21d-4b93-a9b4-a6e09aa60699","metadata":{"tags":[]},"source":["#### Remove participants with any of the listed conditions"]},{"cell_type":"code","execution_count":9,"id":"88d075bb-f716-4006-a411-2d8559db092a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["control_df = control_df[control_df['p131012'].isnull() & control_df['p131016'].isnull() & control_df['p131018'].isnull() & control_df['p131020'].isnull() \n","                        & control_df['p131022'].isnull() & control_df['p131024'].isnull() & control_df['p131026'].isnull() & control_df['p131028'].isnull() \n","                        & control_df['p131030'].isnull() & control_df['p131036'].isnull() & control_df['p131038'].isnull() & control_df['p131040'].isnull() \n","                        & control_df['p131042'].isnull() & control_df['p131046'].isnull() & control_df['p131056'].isnull() & control_df['p131058'].isnull() \n","                        & control_df['p131062'].isnull() & control_df['p131066'].isnull() & control_df['p131068'].isnull() & control_df['p131070'].isnull() \n","                        & control_df['p131074'].isnull() & control_df['p131076'].isnull() & control_df['p131078'].isnull() & control_df['p131080'].isnull() \n","                        & control_df['p131082'].isnull() & control_df['p131084'].isnull() & control_df['p131086'].isnull() & control_df['p131088'].isnull() \n","                        & control_df['p131090'].isnull() & control_df['p131092'].isnull() & control_df['p131094'].isnull() & control_df['p131096'].isnull() \n","                        & control_df['p131098'].isnull() & control_df['p131100'].isnull() & control_df['p131102'].isnull() & control_df['p131104'].isnull() \n","                        & control_df['p131106'].isnull() & control_df['p131108'].isnull() & control_df['p131110'].isnull() & control_df['p131112'].isnull() \n","                        & control_df['p131114'].isnull() & control_df['p131116'].isnull() & control_df['p131120'].isnull() & control_df['p131122'].isnull() \n","                        & control_df['p131124'].isnull() & control_df['p131126'].isnull() & control_df['p42018'].isnull() & control_df['p42020'].isnull() \n","                        & control_df['p42022'].isnull() & control_df['p42024'].isnull() & control_df['p42028'].isnull() & control_df['p42030'].isnull() \n","                        & control_df['p42032'].isnull() & control_df['p42034'].isnull() & control_df['p42036'].isnull()]\n"]},{"cell_type":"markdown","id":"ee348fde-73f0-42cf-b700-a4d21cf96144","metadata":{"tags":[]},"source":["#### Remove participants whose parents have AD or PD"]},{"cell_type":"code","execution_count":10,"id":"940fc55e-e654-48f2-8b05-5e51c9fc64f2","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["# Columns defining all instances of parent illness\n","parent_illness_cols = ['p20110_i0', 'p20110_i1', 'p20110_i2', 'p20110_i3', 'p20107_i0', 'p20107_i1', 'p20107_i2', 'p20107_i3']\n","\n","# Convert None values to empty lists\n","for illness_col in parent_illness_cols:\n","    control_df[illness_col] = control_df[illness_col].apply(lambda l: l if isinstance(l, list) else [])\n","\n","# Define a condition as anybody who has never reported a parent as having AD or PD\n","condition = lambda participant: all((\"Alzheimer's disease/dementia\" not in illnesses and \"Parkinson's disease\" not in illnesses) for illnesses in participant[parent_illness_cols])\n","\n","# Apply the condition to give all participants who have a parent who has/had AD or PD\n","control_df = control_df[control_df.apply(condition, axis=1)]\n"]},{"cell_type":"markdown","id":"5568a2f0-906d-458a-bdc1-1f2b2912ee11","metadata":{"tags":[]},"source":["#### Remove participants below the defined age threshold"]},{"cell_type":"code","execution_count":11,"id":"22cc264e-fb68-4847-ac74-6bc16445d431","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["control_df = control_df[control_df['p21022'] >= 65]\n"]},{"cell_type":"markdown","id":"01cc3aa6-86af-46d2-b865-0631e97dfe1f","metadata":{"tags":[]},"source":["#### Rename columns"]},{"cell_type":"code","execution_count":12,"id":"c4fabba8-2442-4fc0-8bb3-ff9acf870d4d","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["<class 'pandas.core.frame.DataFrame'>\n","Int64Index: 59611 entries, 38 to 502267\n","Data columns (total 12 columns):\n"," #   Column          Non-Null Count  Dtype  \n","---  ------          --------------  -----  \n"," 0   ID              59611 non-null  int64  \n"," 1   AGE_OF_RECRUIT  59611 non-null  int64  \n"," 2   TOWNSEND        59570 non-null  float64\n"," 3   GENETIC_SEX     59611 non-null  object \n"," 4   PC1             57980 non-null  float64\n"," 5   PC2             57980 non-null  float64\n"," 6   PC3             57980 non-null  float64\n"," 7   PC4             57980 non-null  float64\n"," 8   PC5             57980 non-null  float64\n"," 9   BIRTH_YEAR      59611 non-null  int64  \n"," 10  ETHNICITY       50743 non-null  object \n"," 11  DATE_OF_DEATH   8986 non-null   object \n","dtypes: float64(6), int64(3), object(3)\n","memory usage: 5.9+ MB\n"]}],"source":["control_df = control_df[['eid', 'p21022', 'p22189', 'p31', 'p22009_a1', 'p22009_a2', 'p22009_a3', 'p22009_a4', 'p22009_a5', 'p34', 'p22006', 'p40000_i0']]\n","control_df.rename(columns={\n","    'eid':'ID',\n","    'p21022':'AGE_OF_RECRUIT', \n","    'p22189':'TOWNSEND', \n","    'p31':'GENETIC_SEX', \n","    'p22009_a1':'PC1', \n","    'p22009_a2':'PC2', \n","    'p22009_a3':'PC3', \n","    'p22009_a4':'PC4', \n","    'p22009_a5':'PC5', \n","    'p34':'BIRTH_YEAR', \n","    'p22006':'ETHNICITY', \n","    'p40000_i0':'DATE_OF_DEATH',\n","}, inplace=True)\n","control_df[\"ID\"] = pd.to_numeric(control_df[\"ID\"])\n","control_df.info()\n"]},{"cell_type":"markdown","id":"38a4b6e3-b820-458f-b265-b7796c53dc54","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Find ancestry information about each cohort"]},{"cell_type":"markdown","id":"f1a34052-9000-4728-ad7a-3a560e4e8682","metadata":{"tags":[]},"source":["#### Read ancestry label mappings"]},{"cell_type":"code","execution_count":13,"id":"5d9e8a36-4ef8-4371-afba-988a32d30643","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ancestries = pd.read_csv(\"../../mnt/project/wgs_analysis/data/ukbb_imputed_genotypes_umap_linearsvc_predicted_labels.txt\", sep=\"\\t\")\n"]},{"cell_type":"markdown","id":"2b44441e-e349-42bf-afb5-6d0fd800b1f3","metadata":{"tags":[]},"source":["#### Add labels to cohort dataframes"]},{"cell_type":"code","execution_count":14,"id":"6e7aec1a-40d1-443b-895a-ea55fbfd382b","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["control_df = control_df.merge(ancestries[[\"IID\",\"label\"]], left_on=\"ID\", right_on=\"IID\").drop(\"IID\", axis=1)\n","ad_df = ad_df.merge(ancestries[[\"IID\",\"label\"]], left_on=\"ID\", right_on=\"IID\").drop(\"IID\", axis=1)\n","rd_df = rd_df.merge(ancestries[[\"IID\",\"label\"]], left_on=\"ID\", right_on=\"IID\").drop(\"IID\", axis=1)\n"]},{"cell_type":"markdown","id":"db80f25b-5e95-4d9b-8130-9cf553cc7c16","metadata":{"tags":[]},"source":["#### Get list of IDs for each cohort"]},{"cell_type":"code","execution_count":15,"id":"da0af415-fda1-441e-a24e-a2b997c9fe87","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ad_ids = ad_df[\"ID\"].tolist()\n","rd_ids = rd_df[\"ID\"].tolist()\n","control_ids = control_df[\"ID\"].tolist()\n"]},{"cell_type":"markdown","id":"e703dfb9-c45f-4f4e-b7c1-5371fcd1f38e","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Remove related individuals"]},{"cell_type":"markdown","id":"e75c2575-7728-4f0c-a8a2-c16e4909ed29","metadata":{"tags":[]},"source":["#### Fetch relatedness data"]},{"cell_type":"code","execution_count":16,"id":"3d9514f1-6ea1-498a-9442-75b43e5b302c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["full_related_df = pd.read_csv('../../mnt/project/Bulk/Genotype Results/Genotype calls/ukb_rel.dat', sep = ' ')\n","full_related_df = full_related_df[full_related_df['Kinship'] > 0.0884]\n"]},{"cell_type":"markdown","id":"068ab431-8251-4f3e-9eb7-0d928ec69183","metadata":{"tags":[]},"source":["#### Define cohorts to maximize cases included"]},{"cell_type":"code","execution_count":17,"id":"0883dbb0-e079-434a-8937-e10d6390514e","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["full_cohort_ids = ad_ids + rd_ids + control_ids\n","case_ids = ad_ids + rd_ids\n"]},{"cell_type":"markdown","id":"75f9c846-5b7d-4c2e-aa22-cf56fd9f106b","metadata":{"tags":[]},"source":["#### Keep only rows with both participants in cohorts of interest"]},{"cell_type":"code","execution_count":18,"id":"858045e4-fe3e-48e0-9181-f8239ae098b6","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["related_cohort_df = full_related_df.loc[full_related_df['ID1'].isin(full_cohort_ids) & full_related_df['ID2'].isin(full_cohort_ids)]\n","related_cohort_df = related_cohort_df.reset_index(drop=True)\n"]},{"cell_type":"markdown","id":"3ac59ac1-0f14-4fef-8b81-407a598870c7","metadata":{"tags":[]},"source":["#### Maximize the number of cases included"]},{"cell_type":"code","execution_count":19,"id":"d728ac8d-d930-421c-a8e5-f15a91bb2a08","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["flipped_df = related_cohort_df[related_cohort_df[\"ID1\"].isin(control_ids) & related_cohort_df[\"ID2\"].isin(case_ids)].copy()\n","related_cohort_df = related_cohort_df[~(related_cohort_df[\"ID1\"].isin(control_ids) & related_cohort_df[\"ID2\"].isin(case_ids))]\n","flipped_df.rename(columns={\"ID1\":\"ID2\", \"ID2\":\"ID1\"}, inplace=True)\n","related_cohort_df = pd.concat([related_cohort_df, flipped_df])\n"]},{"cell_type":"markdown","id":"758e07fe-ec1c-457d-b8ab-d0b678f41017","metadata":{"tags":[]},"source":["#### Get set of participants to remove"]},{"cell_type":"code","execution_count":20,"id":"b5dbe38d-6111-47a7-b8cd-04fcd5261643","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["Removing 848 participants\n"]}],"source":["ids_to_remove = set(related_cohort_df[\"ID2\"])\n","print(f\"Removing {len(ids_to_remove)} participants\")\n"]},{"cell_type":"markdown","id":"98554e3c-a3b2-42ca-9a3e-07a3ebbfce22","metadata":{"tags":[]},"source":["#### Filter ID lists accordingly"]},{"cell_type":"code","execution_count":21,"id":"0963afa9-d662-446d-98ca-d4b62d258a9f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ad_ids = [iid for iid in ad_ids if iid not in ids_to_remove]\n","rd_ids = [iid for iid in rd_ids if iid not in ids_to_remove]\n","control_ids = [iid for iid in control_ids if iid not in ids_to_remove]\n","total_ids = ad_ids + rd_ids + control_ids\n"]},{"cell_type":"markdown","id":"4384b677-27ec-47f6-813b-2280a0b7205e","metadata":{"tags":[]},"source":["#### Save the IDs of each participant to a txt file"]},{"cell_type":"code","execution_count":22,"id":"ebac2b16-bb6d-4efe-a1e7-1341037867b6","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["with open('ad_ids_pre_VCF.txt', 'w') as file:\n","    for iid in ad_ids:\n","        file.write(f\"{iid}\\n\")\n","        "]},{"cell_type":"code","execution_count":23,"id":"c00fb845-0668-4961-ba0a-623eac074715","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["with open('rd_ids_pre_VCF.txt', 'w') as file:\n","    for iid in rd_ids:\n","        file.write(f\"{iid}\\n\")\n"]},{"cell_type":"code","execution_count":24,"id":"6a0997d6-3bb3-4106-9e76-49bb4da0fae0","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["with open('control_ids_pre_VCF.txt', 'w') as file:\n","    for iid in control_ids:\n","        file.write(f\"{iid}\\n\")\n"]},{"cell_type":"code","execution_count":25,"id":"f581ecc9-00d1-4ee5-ad23-ea0c8eb2105e","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["with open('ids_pre_VCF.txt', 'w') as file:\n","    for iid in total_ids:\n","        file.write(f\"{iid}\\n\")\n"]},{"cell_type":"markdown","id":"0873ee2d-bfa5-4cc1-9deb-9eaafde85999","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Filter out participants without WGS data"]},{"cell_type":"code","execution_count":null,"id":"f6119e27-701e-4e7d-9ef3-c6c626643bd6","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr1/ukb24310_c1_b7761_v1.vcf.gz\" \\\n","-iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr1/ukb24310_c1_b7761_v1.vcf.gz.tbi\" \\\n","-icmd=\"bcftools query -l ukb24310_c1_b7761_v1.vcf.gz > pvcf_full_ids.txt\" \\\n","--instance-type mem1_hdd1_v2_x16 \\\n","--destination \"${projectid}:/wgs_analysis/results\"\n"]},{"cell_type":"code","execution_count":null,"id":"0ea2e900-da98-454e-ab67-95feb6fb8ae6","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":26,"id":"0d7f0a68-4066-404c-91b1-648c96b8fbc6","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["Error: path \"/opt/notebooks/pvcf_full_ids.txt\" already exists but\n","-f/--overwrite was not set\n"]}],"source":["!dx download /wgs_analysis/results/pvcf_full_ids.txt\n","!grep -Fwf pvcf_full_ids.txt ids_pre_VCF.txt > filtered_sample_ids.txt\n","!grep -Fwf pvcf_full_ids.txt ad_ids_pre_VCF.txt > filtered_ad_ids.txt\n","!grep -Fwf pvcf_full_ids.txt rd_ids_pre_VCF.txt > filtered_rd_ids.txt\n","!grep -Fwf pvcf_full_ids.txt control_ids_pre_VCF.txt > filtered_control_ids.txt\n"]},{"cell_type":"code","execution_count":27,"id":"618075de-394b-4149-80f3-f0834b7d3a3a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["with open('filtered_ad_ids.txt', 'r') as file:\n","    ad_ids = [int(line.strip()) for line in file]\n","with open('filtered_rd_ids.txt', 'r') as file:\n","    rd_ids = [int(line.strip()) for line in file]\n","with open('filtered_control_ids.txt', 'r') as file:\n","    control_ids = [int(line.strip()) for line in file]\n"]},{"cell_type":"markdown","id":"1f84405d-6953-402f-8e1b-9af96e3fa536","metadata":{"tags":[]},"source":["#### Get list of IDs for each cohort"]},{"cell_type":"code","execution_count":28,"id":"71fe61ad-b1dd-4816-b625-310e0382201a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ad_df = ad_df[ad_df[\"ID\"].isin(ad_ids)]\n","rd_df = rd_df[rd_df[\"ID\"].isin(rd_ids)]\n","control_df = control_df[control_df[\"ID\"].isin(control_ids)]\n"]},{"cell_type":"code","execution_count":29,"id":"9626b5ed-6274-439d-8a93-33ad761fa874","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["Number of AD participants:       4225\n","Number of RD participants:       5306\n","Number of Control participants:  56741\n"]}],"source":["print(f\"Number of AD participants:       {len(ad_ids)}\")\n","print(f\"Number of RD participants:       {len(rd_ids)}\")\n","print(f\"Number of Control participants:  {len(control_ids)}\")\n"]},{"cell_type":"code","execution_count":30,"id":"f4fa38b3-d52e-400b-b7a1-e6d9acdd3213","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 530,176 of 530,176 bytes (100%) filtered_sample_ids.txt\n","ID                                file-GpkG0ZjJYBPb0z3gzjX4jBX7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results\n","Name                              sample_ids.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:31:15 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:31:16 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 33,800 of 33,800 bytes (100%) filtered_ad_ids.txt\n","ID                                file-GpkG0b8JYBPb0z3gzjX4jBX9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results\n","Name                              ad_ids.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:31:17 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:31:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 42,448 of 42,448 bytes (100%) filtered_rd_ids.txt\n","ID                                file-GpkG0bjJYBPpjFV3qkq31bzG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results\n","Name                              rd_ids.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:31:19 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:31:19 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 453,928 of 453,928 bytes (100%) filtered_control_ids.txt\n","ID                                file-GpkG0f0JYBPpKK0J8XP6JVXg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results\n","Name                              control_ids.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:31:20 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:31:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n"]}],"source":["!dx upload filtered_sample_ids.txt --path /wgs_analysis/results/sample_ids.txt\n","!dx upload filtered_ad_ids.txt --path /wgs_analysis/results/ad_ids.txt\n","!dx upload filtered_rd_ids.txt --path /wgs_analysis/results/rd_ids.txt\n","!dx upload filtered_control_ids.txt --path /wgs_analysis/results/control_ids.txt\n"]},{"cell_type":"markdown","id":"c96a78b8-db32-4e8f-937f-407092475a68","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Save and print cohort statistics"]},{"cell_type":"code","execution_count":31,"id":"13af93db-5aa4-4da2-8255-fa07c5e48b75","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["control_df.to_csv(\"Controls.csv\", header=True, index=False)\n","ad_df.to_csv(f'AD_cases.csv', header=True, index=False)\n","rd_df.to_csv(f'RD_cases.csv', header=True, index=False)\n"]},{"cell_type":"code","execution_count":32,"id":"2eaf30ad-fb5b-4f58-8463-53da85f58a4c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 4,877,644 of 4,877,644 bytes (100%) Controls.csv\n","ID                                file-GpkG0fjJYBPY6gzq1v1K0jxZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results\n","Name                              Controls.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:31:23 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:31:24 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 650,692 of 650,692 bytes (100%) AD_cases.csv\n","ID                                file-GpkG0g8JYBPy975KV0z8F9V0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results\n","Name                              AD_cases.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:31:25 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:31:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 817,762 of 817,762 bytes (100%) RD_cases.csv\n","ID                                file-GpkG0gQJYBPv4G4vJ818fkQ2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results\n","Name                              RD_cases.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:31:26 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:31:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n"]}],"source":["! dx upload Controls.csv --path /wgs_analysis/results/Controls.csv\n","! dx upload AD_cases.csv --path /wgs_analysis/results/AD_cases.csv\n","! dx upload RD_cases.csv --path /wgs_analysis/results/RD_cases.csv\n"]},{"cell_type":"code","execution_count":33,"id":"0ae940a7-abb4-43b2-93b4-b6d07760838a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["EUR    54423\n","SAS      672\n","AFR      445\n","AJ       416\n","CAS      204\n","CAH      169\n","EAS      154\n","AAC      133\n","MDE       62\n","AMR       48\n","FIN       15\n","Name: label, dtype: int64\n","EUR    4051\n","AFR      52\n","SAS      49\n","AJ       27\n","AAC      12\n","CAS      10\n","MDE       9\n","CAH       8\n","EAS       7\n","Name: label, dtype: int64\n","EUR    5028\n","SAS      70\n","AFR      64\n","AJ       43\n","CAH      27\n","AAC      24\n","CAS      23\n","MDE      13\n","EAS      10\n","AMR       3\n","FIN       1\n","Name: label, dtype: int64\n","\n","\n","Female    28400\n","Male      28341\n","Name: GENETIC_SEX, dtype: int64\n","Female    2226\n","Male      1999\n","Name: GENETIC_SEX, dtype: int64\n","Male      2960\n","Female    2346\n","Name: GENETIC_SEX, dtype: int64\n","\n","\n","66.88828905119792 +/- 1.4783824909537675\n","64.68934467233616 +/- 4.394240647897201\n","63.82364864864865 +/- 5.075567668349701\n","66.88031690140845 +/- 1.4778455623561713\n","64.59883198562444 +/- 4.097286411577796\n","63.731031543052005 +/- 5.245986623303864\n","\n","\n"]}],"source":["print(control_df[\"label\"].value_counts())\n","print(ad_df[\"label\"].value_counts())\n","print(rd_df[\"label\"].value_counts())\n","print(\"\\n\")\n","\n","print(control_df[\"GENETIC_SEX\"].value_counts())\n","print(ad_df[\"GENETIC_SEX\"].value_counts())\n","print(rd_df[\"GENETIC_SEX\"].value_counts())\n","print(\"\\n\")\n","\n","print(f'{control_df[control_df[\"GENETIC_SEX\"] == \"Male\"][\"AGE_OF_RECRUIT\"].mean()} +/- {control_df[control_df[\"GENETIC_SEX\"] == \"Male\"][\"AGE_OF_RECRUIT\"].std()}')\n","print(f'{ad_df[ad_df[\"GENETIC_SEX\"] == \"Male\"][\"AGE_OF_RECRUIT\"].mean()} +/- {ad_df[ad_df[\"GENETIC_SEX\"] == \"Male\"][\"AGE_OF_RECRUIT\"].std()}')\n","print(f'{rd_df[rd_df[\"GENETIC_SEX\"] == \"Male\"][\"AGE_OF_RECRUIT\"].mean()} +/- {rd_df[rd_df[\"GENETIC_SEX\"] == \"Male\"][\"AGE_OF_RECRUIT\"].std()}')\n","print(f'{control_df[control_df[\"GENETIC_SEX\"] == \"Female\"][\"AGE_OF_RECRUIT\"].mean()} +/- {control_df[control_df[\"GENETIC_SEX\"] == \"Female\"][\"AGE_OF_RECRUIT\"].std()}')\n","print(f'{ad_df[ad_df[\"GENETIC_SEX\"] == \"Female\"][\"AGE_OF_RECRUIT\"].mean()} +/- {ad_df[ad_df[\"GENETIC_SEX\"] == \"Female\"][\"AGE_OF_RECRUIT\"].std()}')\n","print(f'{rd_df[rd_df[\"GENETIC_SEX\"] == \"Female\"][\"AGE_OF_RECRUIT\"].mean()} +/- {rd_df[rd_df[\"GENETIC_SEX\"] == \"Female\"][\"AGE_OF_RECRUIT\"].std()}')\n","print(\"\\n\")\n"]},{"cell_type":"markdown","id":"cfc4bc7c-09fd-412d-88b2-9435b141e066","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Fetch pVCF chunks for each gene of interest"]},{"cell_type":"markdown","id":"13c0d910-d47b-4ef0-95e0-523fff791b46","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## GBA (chr1: 155,225,002 - 155,254,507) (b: 7761 - 7763)"]},{"cell_type":"code","execution_count":null,"id":"5106ceff-1566-4d98-b5ee-302e89494c10","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {7761..7763};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr1/ukb24310_c1_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr1/ukb24310_c1_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c1_b${b_val}_v1.vcf.gz -o GBA_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done"]},{"cell_type":"markdown","id":"2f5119ab-b11a-49c3-bcf8-cc73f55dbeb4","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## PSEN2 (chr1: 226,860,648 - 226,905,565) (b: 11343 - 11346)"]},{"cell_type":"code","execution_count":null,"id":"4f551298-6d71-4a17-9e39-89080bfcdcc2","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {11343..11346};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr1/ukb24310_c1_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr1/ukb24310_c1_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c1_b${b_val}_v1.vcf.gz -o PSEN2_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"e4f9748d-c4ae-4fd7-87c3-0384a677c10a","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## SNCA (chr4: 89,716,632 - 89,848,254) (b: 4485 - 4493)"]},{"cell_type":"code","execution_count":null,"id":"bf544205-edb1-493c-9cfa-0c0b0251a30d","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {4485..4493};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr4/ukb24310_c4_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr4/ukb24310_c4_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c4_b${b_val}_v1.vcf.gz -o SNCA_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"5769055c-34e0-43d1-a4f7-452635529085","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## TREM2 (chr6: 41,148,607 - 41,173,076) (b: 2057 - 2059)"]},{"cell_type":"code","execution_count":null,"id":"b2c41e93-9d78-4733-8581-30415b2de455","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {2057..2059};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr6/ukb24310_c6_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr6/ukb24310_c6_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c6_b${b_val}_v1.vcf.gz -o TREM2_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"dd049084-fd8f-47b1-90bb-59c90e06e7ba","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## PSEN1 (chr14: 73,126,590 - 73,229,275) (b: 3656 - 3662)"]},{"cell_type":"code","execution_count":null,"id":"c28d5a51-3335-4db6-a5a3-8aa3b1f62592","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {3656..3662};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr14/ukb24310_c14_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr14/ukb24310_c14_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c14_b${b_val}_v1.vcf.gz -o PSEN1_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"8e20081d-226d-4457-a168-e8790c65fdec","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## GRN (chr17: 44,335,332 - 44,362,797) (b: 2216 - 2219)"]},{"cell_type":"code","execution_count":null,"id":"f842fea0-fd3a-4537-b6a6-52bbc1796649","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {2216..2219};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr17/ukb24310_c17_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr17/ukb24310_c17_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c17_b${b_val}_v1.vcf.gz -o GRN_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"faf74c91-2f56-4585-90d4-1ecd03438330","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## MAPT (chr17: 45,884,685 - 46,035,185) (b: 2294 - 2302)"]},{"cell_type":"code","execution_count":null,"id":"00a7b5e7-5518-4b18-9b24-7ba86f4f05e1","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {2294..2302};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr17/ukb24310_c17_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr17/ukb24310_c17_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c17_b${b_val}_v1.vcf.gz -o MAPT_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"cc307a9f-b50d-447e-98b2-c10deaa086fe","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## APOE (chr19: 44,895,840 - 44,919,238) (b: 2244 - 2246)"]},{"cell_type":"code","execution_count":null,"id":"59e5404d-6d53-48b1-aba9-907b9735aa4c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {2244..2246};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr19/ukb24310_c19_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr19/ukb24310_c19_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c19_b${b_val}_v1.vcf.gz -o APOE_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"5f7c1af2-7586-4614-bb18-be3e7be45d3a","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## APP (chr21: 25,871,670 - 26,180,987) (b: 1293 - 1310)"]},{"cell_type":"code","execution_count":null,"id":"77af7716-b3b9-4c25-83f4-46274e678f87","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","for b_val in {1293..1310};\n","do\n","    dx run swiss-army-knife \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr21/ukb24310_c21_b${b_val}_v1.vcf.gz\" \\\n","    -iin=\"/Bulk/DRAGEN\\ WGS/DRAGEN\\ population\\ level\\ WGS\\ variants,\\ pVCF\\ format\\ [500k\\ release]/chr21/ukb24310_c21_b${b_val}_v1.vcf.gz.tbi\" \\\n","    -iin=\"/wgs_analysis/results/sample_ids.txt\" \\\n","    -icmd=\"bcftools view -O z -S sample_ids.txt ukb24310_c21_b${b_val}_v1.vcf.gz -o APP_b${b_val}.vcf.gz\" \\\n","    --instance-type mem2_ssd1_v2_x32 \\\n","    --destination \"${projectid}:/wgs_analysis/results/1_pvcf_chunks\"\n","done\n"]},{"cell_type":"markdown","id":"5a97d311-e969-43c6-b2c8-eacff5b473d3","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Combine pVCF chunks into one file for each gene"]},{"cell_type":"code","execution_count":null,"id":"4ed1146d-aaee-4cd5-b100-3b31c40cb52b","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"markdown","id":"e7376d10-4dbc-4919-a5c3-5f8884c81f73","metadata":{"jp-MarkdownHeadingCollapsed":true,"jupyter":{"outputs_hidden":true},"tags":[]},"source":["## GBA"]},{"cell_type":"code","execution_count":null,"id":"f99872b4-090e-4c15-8db4-706eb2a7144f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/GBA_b7761.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/GBA_b7762.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/GBA_b7763.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z GBA_b7761.vcf.gz GBA_b7762.vcf.gz GBA_b7763.vcf.gz -o GBA.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"798e8e1e-63e2-4068-bf29-6c489116b2fb","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## PSEN2"]},{"cell_type":"code","execution_count":null,"id":"fafe52e3-7115-4bc0-9a03-c38fdc7f5a74","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN2_b11343.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN2_b11344.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN2_b11345.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN2_b11346.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z PSEN2_b11343.vcf.gz PSEN2_b11344.vcf.gz PSEN2_b11345.vcf.gz PSEN2_b11346.vcf.gz -o PSEN2.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"1660a493-06da-4690-b5ad-0970ddb327e4","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## SNCA"]},{"cell_type":"code","execution_count":null,"id":"f376de9d-f37c-43ec-a7da-47f202dbb92d","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4485.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4486.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4487.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4488.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4489.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4490.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4491.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4492.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/SNCA_b4493.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z SNCA_b4485.vcf.gz SNCA_b4486.vcf.gz SNCA_b4487.vcf.gz SNCA_b4488.vcf.gz SNCA_b4489.vcf.gz SNCA_b4490.vcf.gz SNCA_b4491.vcf.gz SNCA_b4492.vcf.gz SNCA_b4493.vcf.gz -o SNCA.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"e5ee1b41-3bdb-42ea-a4aa-da581924a543","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## TREM2"]},{"cell_type":"code","execution_count":null,"id":"aad87835-6238-4582-adbd-b305f5c213c9","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/TREM2_b2057.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/TREM2_b2058.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/TREM2_b2059.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z TREM2_b2057.vcf.gz TREM2_b2058.vcf.gz TREM2_b2059.vcf.gz -o TREM2.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"2f00e800-d9db-41f1-8c8d-34989a00155a","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## PSEN1"]},{"cell_type":"code","execution_count":null,"id":"11dae603-196b-4aea-b5f6-87383a5024e6","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN1_b3656.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN1_b3657.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN1_b3658.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN1_b3659.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN1_b3660.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN1_b3661.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/PSEN1_b3662.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z PSEN1_b3656.vcf.gz PSEN1_b3657.vcf.gz PSEN1_b3658.vcf.gz PSEN1_b3659.vcf.gz PSEN1_b3660.vcf.gz PSEN1_b3661.vcf.gz PSEN1_b3662.vcf.gz -o PSEN1.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"ce81cd0e-74fd-44e0-8ccd-17e59803be65","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## GRN"]},{"cell_type":"code","execution_count":null,"id":"257db9b7-d7e5-494e-a005-336a0b5f16d7","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/GRN_b2216.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/GRN_b2217.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/GRN_b2218.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/GRN_b2219.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z GRN_b2216.vcf.gz GRN_b2217.vcf.gz GRN_b2218.vcf.gz GRN_b2219.vcf.gz -o GRN.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"c9f650cc-f149-438a-93e9-0c5e93f2b8c7","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## MAPT"]},{"cell_type":"code","execution_count":null,"id":"2d822680-a9f7-4c3d-835f-6e7a5967a50f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2294.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2295.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2296.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2297.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2298.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2299.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2300.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2301.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/MAPT_b2302.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z MAPT_b2294.vcf.gz MAPT_b2295.vcf.gz MAPT_b2296.vcf.gz MAPT_b2297.vcf.gz MAPT_b2298.vcf.gz MAPT_b2299.vcf.gz MAPT_b2300.vcf.gz MAPT_b2301.vcf.gz MAPT_b2302.vcf.gz -o MAPT.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"ae52306a-aaa0-4cf9-83c5-7a8b9138317b","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## APOE"]},{"cell_type":"code","execution_count":null,"id":"c4e1c19f-40ba-4fc2-ac9a-83cfc55c8e55","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APOE_b2244.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APOE_b2245.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APOE_b2246.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z APOE_b2244.vcf.gz APOE_b2245.vcf.gz APOE_b2246.vcf.gz -o APOE.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"b0b31a58-f219-4148-9600-3083691c9d14","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## APP"]},{"cell_type":"code","execution_count":null,"id":"a9d8ba21-09ee-4c2c-b8d7-59275a7457cb","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1293.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1294.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1295.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1296.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1297.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1298.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1299.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1300.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1301.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1302.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1303.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1304.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1305.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1306.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1307.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1308.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1309.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/1_pvcf_chunks/APP_b1310.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z APP_b1293.vcf.gz APP_b1294.vcf.gz APP_b1295.vcf.gz APP_b1296.vcf.gz APP_b1297.vcf.gz APP_b1298.vcf.gz APP_b1299.vcf.gz APP_b1300.vcf.gz APP_b1301.vcf.gz APP_b1302.vcf.gz APP_b1303.vcf.gz APP_b1304.vcf.gz APP_b1305.vcf.gz APP_b1306.vcf.gz APP_b1307.vcf.gz APP_b1308.vcf.gz APP_b1309.vcf.gz APP_b1310.vcf.gz -o APP.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/2_pvcf_genes\"\n"]},{"cell_type":"markdown","id":"36ed5836-2a94-44ec-9b79-29c0635cd0ab","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Concatenate genes together"]},{"cell_type":"code","execution_count":null,"id":"1a223967-179f-496e-b935-445c3c0c8344","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/GBA.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/PSEN2.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/SNCA.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/TREM2.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/PSEN1.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/GRN.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/MAPT.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/APOE.vcf.gz\" \\\n","-iin=\"/wgs_analysis/results/2_pvcf_genes/APP.vcf.gz\" \\\n","-icmd=\"bcftools concat -O z GBA.vcf.gz PSEN2.vcf.gz SNCA.vcf.gz TREM2.vcf.gz PSEN1.vcf.gz GRN.vcf.gz MAPT.vcf.gz APOE.vcf.gz APP.vcf.gz -o combined.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/3_pvcf_combined\"\n"]},{"cell_type":"markdown","id":"434524be-1789-4c78-8e57-99f2e6de3b0e","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Normalize VCFs before annotation"]},{"cell_type":"markdown","id":"cb462409-8441-4154-b27f-af2f460fa2fc","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["#### Split multiallelic sites into biallelic records"]},{"cell_type":"code","execution_count":null,"id":"c0e635cb-8334-428d-800b-d535ab7c9641","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":null,"id":"56415203-db4b-495b-a085-06ebcbf58349","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/3_pvcf_combined/combined.vcf.gz\" \\\n","-icmd=\"bcftools norm -m-both -o biallelic.vcf combined.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/4_normalized\"\n"]},{"cell_type":"markdown","id":"2b8f43a9-7b0c-4622-8651-5329845ec66c","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["#### Left-align and normalize"]},{"cell_type":"code","execution_count":null,"id":"a960869a-67e1-464d-8ce8-b5b8cb666411","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":null,"id":"cf1c6e99-309b-4ae0-9590-fa8191aef782","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/4_normalized/biallelic.vcf\" \\\n","-iin=\"/wgs_analysis/data/Homo_sapiens_assembly38.fasta\" \\\n","-icmd=\"bcftools norm -f Homo_sapiens_assembly38.fasta -o normalized.vcf biallelic.vcf\" \\\n","--instance-type mem2_ssd1_v2_x64 \\\n","--destination \"${projectid}:/wgs_analysis/results/4_normalized\"\n"]},{"cell_type":"markdown","id":"6a381f28-e318-45d9-a167-3fd40c98dc5b","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Annotation"]},{"cell_type":"markdown","id":"301fd58e-661c-49cc-858e-0f118d15395c","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Filter VCFs to only include a few participants"]},{"cell_type":"markdown","id":"c05d57f5-e6e5-49f3-9b88-174c218ad8e8","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["#### Get subset of participant IDs"]},{"cell_type":"code","execution_count":null,"id":"57e3f4ed-60b7-4fa7-a773-9517ec5c0dd1","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! dx download wgs_analysis/results/ad_ids.txt --overwrite\n","! dx download wgs_analysis/results/rd_ids.txt --overwrite\n","! dx download wgs_analysis/results/control_ids.txt --overwrite\n"]},{"cell_type":"code","execution_count":null,"id":"864b79a5-23df-4b79-81f0-65ea717e84e2","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ad_ids = \"ad_ids.txt\"\n","rd_ids = \"rd_ids.txt\"\n","control_ids = \"control_ids.txt\"\n","output_file = \"annot_ids.txt\"\n","\n","with open(ad_ids, \"r\") as f1, open(rd_ids, \"r\") as f2, open(control_ids, \"r\") as f3, open(output_file, \"w\") as out:\n","    out.write(f1.readline().strip() + \"\\n\")\n","    out.write(f2.readline().strip() + \"\\n\")\n","    out.write(f3.readline().strip() + \"\\n\")\n"]},{"cell_type":"code","execution_count":null,"id":"1ba1919c-109f-43c6-b62e-76ce64d3fcdb","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! dx upload annot_ids.txt --path wgs_analysis/results/annot_ids.txt\n"]},{"cell_type":"markdown","id":"7ef15586-0c90-4fe6-bb56-f678c181e26f","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["#### Get filtered VCFs"]},{"cell_type":"code","execution_count":null,"id":"ad459daf-e093-4841-82f9-98d1acd0862a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","-iin=\"/wgs_analysis/results/annot_ids.txt\" \\\n","-icmd=\"bcftools view -O z -S annot_ids.txt normalized.vcf -o filtered.vcf.gz\" \\\n","--instance-type mem2_ssd1_v2_x32 \\\n","--destination \"${projectid}:/wgs_analysis/results/5_annotated\"\n"]},{"cell_type":"markdown","id":"50c8eaf9-1da4-49b8-ae56-890259085319","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Fetch Annovar libraries and reference genome data"]},{"cell_type":"code","execution_count":null,"id":"1a6d97d7-30e5-4aa4-b113-8a1a14129b8f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%capture\n","\n","! wget http://www.openbioinformatics.org/annovar/download/0wgxR2rIVP/annovar.latest.tar.gz\n","! tar -xzf annovar.latest.tar.gz\n","! chmod a+x ./annovar/*.pl\n","! annovar/annotate_variation.pl -downdb -buildver hg38 -webfrom annovar refGene annovar/humandb/\n","! annovar/annotate_variation.pl -downdb -buildver hg38 -webfrom annovar avsnp150 annovar/humandb/\n","! annovar/annotate_variation.pl -downdb -buildver hg38 -webfrom annovar clinvar_20221231 annovar/humandb/\n","! annovar/annotate_variation.pl -downdb -buildver hg38 -webfrom annovar dbnsfp30a annovar/humandb/\n","! annovar/annotate_variation.pl -downdb -buildver hg38 -webfrom annovar gnomad40_genome annovar/humandb/\n","! dx download wgs_analysis/data/Homo_sapiens_assembly38.fasta --overwrite\n","! dx download wgs_analysis/data/Homo_sapiens_assembly38.fasta.fai --overwrite\n","! dx download wgs_analysis/data/Homo_sapiens_assembly38.dict --overwrite\n"]},{"cell_type":"markdown","id":"96fadfc5-73c5-4f79-ac67-b019a29f8322","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Perform annotation"]},{"cell_type":"code","execution_count":null,"id":"c499f4d3-5cd9-460c-93fc-37c22ffc2079","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","annovar/table_annovar.pl ../../mnt/project/wgs_analysis/results/5_annotated/filtered.vcf.gz annovar/humandb/ \\\n","--buildver hg38 \\\n","--thread 96 \\\n","--remove \\\n","--protocol refGene,avsnp150,clinvar_20221231,dbnsfp30a,gnomad40_genome \\\n","--operation g,f,f,f,f \\\n","--nopolish \\\n","--nastring . \\\n","--out var_calling.annovar \\\n","--vcfinput\n"]},{"cell_type":"code","execution_count":null,"id":"7fa254ae-c13a-4c43-b0a5-5f5a013de50a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["annot_df = pd.read_csv(f\"var_calling.annovar.hg38_multianno.txt\", sep = '\\t')\n","annot_df.to_csv(f\"annotated.csv\", index=False)\n","! dx upload annotated.csv --path wgs_analysis/results/5_annotated/annotated.csv\n"]},{"cell_type":"markdown","id":"8d2e5906-4c04-4cde-afd0-59c0b0be3a69","metadata":{"jp-MarkdownHeadingCollapsed":true,"jupyter":{"outputs_hidden":true},"tags":[]},"source":["# Calculate allele frequencies"]},{"cell_type":"markdown","id":"8ac61a09-1cc2-4687-948f-b82bc3fae0ea","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Subset IDs for all cohort-ancestry combinations"]},{"cell_type":"code","execution_count":null,"id":"4ea7d3b5-5967-43ce-bd4b-fb887e23b958","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ad_df = pd.read_csv(\"../../mnt/project/wgs_analysis/results/AD_cases.csv\")\n","rd_df = pd.read_csv(\"../../mnt/project/wgs_analysis/results/RD_cases.csv\")\n","control_df = pd.read_csv(\"../../mnt/project/wgs_analysis/results/Controls.csv\")\n"]},{"cell_type":"code","execution_count":null,"id":"c2a16b01-0f26-4a37-b998-b2f7de03920c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ad_df = ad_df[[\"ID\",\"label\"]]\n","rd_df = rd_df[[\"ID\",\"label\"]]\n","control_df = control_df[[\"ID\",\"label\"]]\n"]},{"cell_type":"code","execution_count":null,"id":"86fa392b-0444-4994-b9f8-9f0fa521f064","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for ancestry in ad_df[\"label\"].unique():\n","    ids = ad_df[ad_df[\"label\"] == ancestry][\"ID\"]\n","    with open(f\"AD_{ancestry}.txt\", 'w') as file:\n","        for iid in ids:\n","            file.write(f\"{iid}\\n\")\n","    !dx upload AD_{ancestry}.txt --path wgs_analysis/data/ID_Files/AD_{ancestry}.txt\n","\n","for ancestry in rd_df[\"label\"].unique():\n","    ids = rd_df[rd_df[\"label\"] == ancestry][\"ID\"]\n","    with open(f\"RD_{ancestry}.txt\", 'w') as file:\n","        for iid in ids:\n","            file.write(f\"{iid}\\n\")\n","    !dx upload RD_{ancestry}.txt --path wgs_analysis/data/ID_Files/RD_{ancestry}.txt\n","\n","for ancestry in control_df[\"label\"].unique():\n","    ids = control_df[control_df[\"label\"] == ancestry][\"ID\"]\n","    with open(f\"Control_{ancestry}.txt\", 'w') as file:\n","        for iid in ids:\n","            file.write(f\"{iid}\\n\")\n","    !dx upload Control_{ancestry}.txt --path wgs_analysis/data/ID_Files/Control_{ancestry}.txt\n"]},{"cell_type":"markdown","id":"b29f10fe-a17c-4a16-8129-348af38fbfff","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Get frequencies for each gene-cohort-ancestry combination"]},{"cell_type":"code","execution_count":null,"id":"a569c22d-aab0-4e73-a432-ea2c25ddcda2","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","for cohort in {\"AD\",\"RD\",\"Control\"};\n","do\n","    for ancestry in {\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"};\n","    do\n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","        -iin=\"/wgs_analysis/data/ID_Files/${cohort}_${ancestry}.txt\" \\\n","        -icmd=\"plink2 --vcf normalized.vcf --set-all-var-ids 'chr@:#:\\$r:\\$a' --new-id-max-allele-len 999 --keep ${cohort}_${ancestry}.txt --freq --out ${cohort}_${ancestry}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/6_frequencies\"\n","    done\n","done\n"]},{"cell_type":"markdown","id":"fdf813fc-0f30-4a33-9d16-889cf89e92cf","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Get zygosity info"]},{"cell_type":"code","execution_count":null,"id":"2aba3ad2-bd40-40da-a9c2-f3dabd0ab902","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","for cohort in {\"AD\",\"RD\",\"Control\"};\n","do\n","    for ancestry in {\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"};\n","    do\n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","        -iin=\"/wgs_analysis/data/ID_Files/${cohort}_${ancestry}.txt\" \\\n","        -iin=\"/wgs_analysis/results/6_frequencies/${cohort}_${ancestry}.afreq\" \\\n","        -icmd=\"plink2 --vcf normalized.vcf --set-all-var-ids 'chr@:#:\\$r:\\$a' --new-id-max-allele-len 999 --keep ${cohort}_${ancestry}.txt --read-freq ${cohort}_${ancestry}.afreq --export A --het --out ${cohort}_${ancestry}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/6_frequencies\"\n","    done\n","done\n"]},{"cell_type":"markdown","id":"fd7d93a4-0409-4efc-9c57-2869c2f58709","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Find homozygous/heterozygous counts"]},{"cell_type":"code","execution_count":null,"id":"b05f9783-778c-46c3-b7ac-0be1b235f7e1","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        ! dx download /wgs_analysis/results/6_frequencies/{cohort}_{ancestry}.raw\n"]},{"cell_type":"code","execution_count":null,"id":"34d02dd5-2b2a-4ea2-a769-1d387758f4ab","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","for cohort in {\"AD\",\"RD\",\"Control\"};\n","do\n","    for ancestry in {\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"};\n","    do\n","        output_file=\"counts_${cohort}_${ancestry}.tsv\"\n","        temp_output=\"temp_${cohort}_${ancestry}.tsv\"\n","\n","        > $output_file\n","\n","        input_file=\"cut.raw\"\n","\n","        grep \"FID\" ${cohort}_${ancestry}.raw | cut -d$'\\t' -f7- > header.tmp\n","\n","        cut -d$'\\t' -f7- \"${cohort}_${ancestry}.raw\" > $input_file\n","\n","        declare -A count_2\n","        declare -A count_1\n","        declare -A count_0\n","\n","        awk -F'\\t' '\n","        {\n","            for (i=1; i<=NF; i++) {\n","                if ($i == 2) count_2[i]++;\n","                else if ($i == 1) count_1[i]++;\n","                else if ($i == 0) count_0[i]++;\n","            }\n","        }\n","        END {\n","            for (i=1; i<=NF; i++) {\n","                printf(\"%d\", count_2[i]);\n","                if (i<NF) printf(\"\\t\");\n","            }\n","            print \"\";\n","\n","            for (i=1; i<=NF; i++) {\n","                printf(\"%d\", count_1[i]);\n","                if (i<NF) printf(\"\\t\");\n","            }\n","            print \"\";\n","\n","            for (i=1; i<=NF; i++) {\n","                printf(\"%d\", count_0[i]);\n","                if (i<NF) printf(\"\\t\");\n","            }\n","            print \"\";\n","        }' $input_file > count.tmp\n","\n","        cat header.tmp count.tmp > $output_file\n","\n","        rm $input_file count.tmp header.tmp\n","\n","        echo \"Counts have been appended to $output_file\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":null,"id":"d5f679f3-dca4-46cd-bbfe-89783b728ad7","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        ! dx upload counts_{cohort}_{ancestry}.tsv --path wgs_analysis/results/7_zygosity/counts_{cohort}_{ancestry}.tsv\n"]},{"cell_type":"code","execution_count":null,"id":"1c484af5-5d65-4d57-9432-a8020c4ea5b5","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        ! dx download wgs_analysis/results/7_zygosity/counts_{cohort}_{ancestry}.tsv\n"]},{"cell_type":"code","execution_count":null,"id":"ebd4f920-921e-4669-a5e5-06fd199da02d","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    zyg_cohort = []\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        try:\n","            df = pd.read_csv(f\"counts_{cohort}_{ancestry}.tsv\", sep=\"\\t\")\n","            variant_ids = df.columns.values\n","            rename_dict = {}\n","            for vid in variant_ids:\n","                rename_dict[vid] = vid.split(\"_\")[0]\n","            df.rename(rename_dict, axis=1, inplace=True)\n","            zyg_cohort.append(df)\n","        except:\n","            print(f\"No data found at counts_{cohort}_{ancestry}.tsv\")\n","    result_zyg = reduce(lambda x, y: x + y, zyg_cohort)\n","    result_zyg.to_csv(f\"{cohort}_Final.csv\", index=False)\n","    ! dx upload {cohort}_Final.csv --path wgs_analysis/results/7_zygosity/{cohort}_Final.csv\n"]},{"cell_type":"code","execution_count":null,"id":"658c9859-1990-4de1-8769-3aaaf431f1e5","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ad_zyg = pd.read_csv(f\"AD_Final.csv\")\n","rd_zyg = pd.read_csv(f\"RD_Final.csv\")\n","control_zyg = pd.read_csv(f\"Control_Final.csv\")\n","\n","ad_zyg = ad_zyg.T\n","rd_zyg = rd_zyg.T\n","control_zyg = control_zyg.T\n","\n","ad_zyg = ad_zyg.reset_index()\n","rd_zyg = rd_zyg.reset_index()\n","control_zyg = control_zyg.reset_index()\n","\n","ad_zyg.rename(columns={'index': 'ID', 0: 'AD_Homozygous_Ref', 1: 'AD_Heterozygous', 2: 'AD_Homozygous_Alt'}, inplace=True)\n","rd_zyg.rename(columns={'index': 'ID', 0: 'RD_Homozygous_Ref', 1: 'RD_Heterozygous', 2: 'RD_Homozygous_Alt'}, inplace=True)\n","control_zyg.rename(columns={'index': 'ID', 0: 'Control_Homozygous_Ref', 1: 'Control_Heterozygous', 2: 'Control_Homozygous_Alt'}, inplace=True)\n","\n","final_zyg = ad_zyg.merge(rd_zyg, on=\"ID\")\n","final_zyg = final_zyg.merge(control_zyg, on=\"ID\")\n","\n","display(ad_zyg.head())\n","display(final_zyg.head())\n","\n","final_zyg.to_csv(\"final_zygosity.csv\", index=False)\n","! dx upload final_zygosity.csv --path wgs_analysis/results/7_zygosity/final_zygosity.csv\n"]},{"cell_type":"markdown","id":"5be5d449-7d1c-4f60-87a1-78ab4274d979","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Merge annotations with allele frequency outputs"]},{"cell_type":"markdown","id":"1b86f48b-d56f-4092-ba54-49518d311cd6","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Merge frequencies for all ancestries across each cohort"]},{"cell_type":"code","execution_count":null,"id":"49655d4b-1ce1-4583-9bad-b685fff7ccd4","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    freq_files = glob.glob(f\"../../mnt/project/wgs_analysis/results/6_frequencies/{cohort}*.afreq\")\n","\n","    df = pd.read_csv(freq_files[0], sep=\"\\t\")\n","    df = df[[\"ID\",\"ALT_FREQS\",\"OBS_CT\"]]\n","\n","    ancestry = freq_files[0].split(\"_\")[-1].split(\".\")[0]\n","    df.rename({\"ALT_FREQS\":f\"ALT_FREQS_{ancestry}_{cohort}\", \"OBS_CT\":f\"OBS_CT_{ancestry}_{cohort}\"}, inplace=True, axis=1)\n","\n","    for i in range(1, len(freq_files)):\n","        df_merge = pd.read_csv(freq_files[i], sep=\"\\t\")\n","        df_merge = df_merge[[\"ID\",\"ALT_FREQS\",\"OBS_CT\"]]\n","\n","        ancestry = freq_files[i].split(\"_\")[-1].split(\".\")[0]\n","        df_merge.rename({\"ALT_FREQS\":f\"ALT_FREQS_{ancestry}_{cohort}\", \"OBS_CT\":f\"OBS_CT_{ancestry}_{cohort}\"}, inplace=True, axis=1)\n","\n","        df = df.merge(df_merge, on=\"ID\")\n","\n","    df.to_csv(f\"{cohort}.csv\", index=False)\n"]},{"cell_type":"markdown","id":"22556c5e-9f4a-4e0f-8b66-0a89064de79d","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Convert back to vcf format for external CADD calculation"]},{"cell_type":"code","execution_count":null,"id":"c8d5f463-75a0-448c-afbe-b16bd25e0d5c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ids_list = list(pd.read_csv(\"AD.csv\")[\"ID\"])\n","df_for_cadd = pd.DataFrame({\"ID\":ids_list})\n"]},{"cell_type":"code","execution_count":null,"id":"e14203bd-f077-49f8-9dcd-a65b6fa27e96","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["df_for_cadd[[\"#CHROM\",\"POS\",\"REF\",\"ALT\"]] = df_for_cadd[\"ID\"].str.split(':', expand=True)\n","df_for_cadd.drop(\"ID\", axis=1, inplace=True)\n","df_for_cadd.insert(2, 'ID', '.')\n","df_for_cadd.to_csv(\"for_CADD.vcf\", index=False, sep=\"\\t\")\n"]},{"cell_type":"code","execution_count":null,"id":"c57626b2-cc92-439d-8823-3c1c8208038b","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! gzip -c for_CADD.vcf > for_CADD.vcf.gz\n"]},{"cell_type":"code","execution_count":null,"id":"b708a70f-b257-42ba-903a-9fecd5e13f82","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","---------------- DOWNLOAD OUTPUT FILE AND PASS THROUGH EXTERNAL CADD SCORE CALCULATOR ----------------\n","------------------------------ (https://cadd.gs.washington.edu/upload) -------------------------------\n","--------------------------- RENAME RESULT TO \"CADD.tsv.gz\" AND UPLOAD HERE ---------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":null,"id":"8f892214-216e-4dc6-9fcd-f9a19f2a35b4","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! gzip -d CADD.tsv.gz\n","! grep -v \"##\" CADD.tsv > CADD_prelim.tsv\n","cadd_df = pd.read_csv(\"CADD_prelim.tsv\", sep=\"\\t\")\n","cadd_df[\"#Chrom\"] = \"chr\" + cadd_df[\"#Chrom\"].astype(str)\n","cadd_df[\"ID\"] = cadd_df[\"#Chrom\"] + \":\" + cadd_df[\"Pos\"].astype(str) + \":\" + cadd_df[\"Ref\"] + \":\" + cadd_df[\"Alt\"]\n","cadd_df = cadd_df[[\"ID\",\"PHRED\"]]\n","cadd_df.rename({\"PHRED\":\"CADD\"}, axis=1, inplace=True)\n","cadd_df.to_csv(\"CADD_final.csv\", index=False)\n"]},{"cell_type":"markdown","id":"ed853ec9-f492-4cae-87ef-afe2414b3d1a","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Merge CADD scores, frequency files, and annotations"]},{"cell_type":"code","execution_count":null,"id":"bdd57627-5694-4933-9ac5-e18ffe36232e","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["annot_df = pd.read_csv(f\"../../mnt/project/wgs_analysis/results/5_annotated/annotated.csv\", low_memory=False)\n","annot_df.insert(1, \"ID\", annot_df[[\"Chr\",\"Start\",\"Ref\",\"Alt\"]].astype(str).agg(':'.join, axis=1))\n","filtered_columns = [col for col in annot_df.columns if \"Otherinfo\" not in col]\n","annot_df = annot_df[filtered_columns]\n","\n","ad_freq_df = pd.read_csv(\"AD.csv\")\n","ad_freq_df.drop(columns=\"ID\", inplace=True)\n","rd_freq_df = pd.read_csv(\"RD.csv\")\n","rd_freq_df.drop(columns=\"ID\", inplace=True)\n","control_freq_df = pd.read_csv(\"Control.csv\")\n","display(control_freq_df.head())\n","control_freq_df = control_freq_df.merge(final_zyg, on=\"ID\")\n","display(control_freq_df.head())\n","control_freq_df = control_freq_df.merge(cadd_df, on=\"ID\", how=\"left\")\n","display(control_freq_df.head())\n","control_freq_df.drop(columns=\"ID\", inplace=True)\n","\n","merged_df = pd.concat([annot_df,ad_freq_df], axis=1)\n","merged_df = pd.concat([merged_df,rd_freq_df], axis=1)\n","merged_df = pd.concat([merged_df,control_freq_df], axis=1)\n","\n","merged_df.to_csv(\"merged.csv\", index=False)\n","!dx upload merged.csv --path wgs_analysis/results/8_merged/merged.csv\n"]},{"cell_type":"code","execution_count":null,"id":"6656e3a9-b03c-40ee-8614-f85389db8552","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["gene_info_dict = fetch_gene_info_ensembl(gene_names=gene_names, species='human', genome_version='GRCh38')\n","for gene in gene_info_dict:\n","    print(gene_info_dict[gene])\n"]},{"cell_type":"code","execution_count":null,"id":"5bbd5028-da53-4bcb-8cf6-695329b0d622","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["criteria_list = []\n","for gene in [\"APOE\", \"APP\", \"GBA\", \"GRN\", \"MAPT\", \"PSEN1\", \"PSEN2\", \"SNCA\", \"TREM2\"]:\n","    if gene == \"GBA\":\n","        gene = \"GBA1\"\n","    chrnum = gene_info_dict[gene][\"chromosome\"]\n","    start = gene_info_dict[gene][\"start\"]\n","    end = gene_info_dict[gene][\"end\"]\n","    criteria_list.append((merged_df[\"Chr\"] == chrnum) \n","                         & (merged_df[\"Start\"] > start) \n","                         & (merged_df[\"Start\"] < end)\n","                         & (merged_df[\"Func.refGene\"].isin([\"exonic\", \"splicing\"]))\n","                        )\n","\n","filter_criteria = criteria_list[0]\n","for criterion in criteria_list[1:]:\n","    filter_criteria |= criterion\n","\n","filtered_df = merged_df[filter_criteria]\n","filtered_df = filtered_df[filtered_df[\"Func.refGene\"].isin([\"exonic\", \"splicing\"])]\n","\n","print(filtered_df.shape)\n","display(filtered_df)\n"]},{"cell_type":"code","execution_count":null,"id":"05d217ad-2574-4bb6-87b6-b5b7c406cc3c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["ancestries = [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]\n","ctrl_col_names = [f\"ALT_FREQS_{ancestry}_Control\" for ancestry in ancestries if f\"ALT_FREQS_{ancestry}_Control\" in filtered_df.columns.values]\n","ad_col_names = [f\"ALT_FREQS_{ancestry}_AD\" for ancestry in ancestries if f\"ALT_FREQS_{ancestry}_AD\" in filtered_df.columns.values]\n","rd_col_names = [f\"ALT_FREQS_{ancestry}_RD\" for ancestry in ancestries if f\"ALT_FREQS_{ancestry}_RD\" in filtered_df.columns.values]\n","\n","final_filtered_df = filtered_df[(filtered_df[ad_col_names + rd_col_names]>0).any(axis=1)]\n","\n","final_filtered_df[\"Disease\"] = \"\"\n","final_filtered_df[\"Disease\"][(final_filtered_df[ad_col_names]>0).any(axis=1)] = \"AD\"\n","final_filtered_df[\"Disease\"][(final_filtered_df[rd_col_names]>0).any(axis=1)] = \"RD\"\n","final_filtered_df[\"Disease\"][(final_filtered_df[ad_col_names]>0).any(axis=1) & (final_filtered_df[rd_col_names]>0).any(axis=1)] = \"Both\"\n","display(final_filtered_df.head())\n","final_filtered_df.to_csv(\"final_filtered.csv\", index=False)\n"]},{"cell_type":"code","execution_count":null,"id":"2b8758d4-5ff3-41d9-a42d-8460c7340c7c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! dx upload final_filtered.csv --path wgs_analysis/results/8_merged/final_filtered.csv\n"]},{"cell_type":"markdown","id":"f00b768e-f877-4f5e-bfbd-89cfac58f27d","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Remove variants expressed in controls"]},{"cell_type":"code","execution_count":null,"id":"62b093b0-a4e1-4475-9c92-d975b7d0cf35","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["final_filtered_df = final_filtered_df[(final_filtered_df[\"Control_Heterozygous\"] == 0) & (final_filtered_df[\"Control_Homozygous_Alt\"] == 0)]\n","final_filtered_df.to_csv(\"final_filtered_onlycases.csv\", index=False)\n"]},{"cell_type":"code","execution_count":null,"id":"559d41e2-c741-46ff-9360-098bd98f1afd","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! dx upload final_filtered_onlycases.csv --path wgs_analysis/results/8_merged/final_filtered_onlycases.csv\n"]},{"cell_type":"markdown","id":"56bbd159-3c9f-420e-bca9-496a13e5f935","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# APOE Genotyping"]},{"cell_type":"markdown","id":"0f21e371-a148-4206-85de-b7a3fea1c249","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Calculate APOE Genotypes"]},{"cell_type":"code","execution_count":null,"id":"8d3637d0-b547-477e-b5c0-764c535bb2d0","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","for cohort in {\"AD\",\"RD\",\"Control\"};\n","do\n","    for ancestry in {\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"};\n","    do\n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","        -iin=\"/wgs_analysis/data/ID_Files/${cohort}_${ancestry}.txt\" \\\n","        -icmd=\"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep ${cohort}_${ancestry}.txt --set-all-var-ids 'chr@:#:\\$r:\\$a' --new-id-max-allele-len 999 --make-bed --out ${cohort}_${ancestry}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/9_apoe_genotyping\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":null,"id":"49b83674-c4ed-42b2-9933-b9d13c4eec24","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":null,"id":"cfc3c8c9-85a0-4f4c-a3bb-8a878954e0e4","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","for cohort in {\"AD\",\"RD\",\"Control\"};\n","do\n","    for ancestry in {\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"};\n","    do\n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/results/9_apoe_genotyping/${cohort}_${ancestry}.bim\" \\\n","        -iin=\"/wgs_analysis/results/9_apoe_genotyping/${cohort}_${ancestry}.bed\" \\\n","        -iin=\"/wgs_analysis/results/9_apoe_genotyping/${cohort}_${ancestry}.fam\" \\\n","        -iin=\"/wgs_analysis/data/apoe_variants.txt\" \\\n","        -icmd=\"plink --bfile ${cohort}_${ancestry} --extract apoe_variants.txt --make-bed --out apoe_snps_${cohort}_${ancestry}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/9_apoe_genotyping\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":null,"id":"aaf00eea-554a-4147-970e-c40f0f544d80","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":null,"id":"6848833d-e61d-4ba9-81ca-a44da9c3539a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","for cohort in {\"AD\",\"RD\",\"Control\"};\n","do\n","    for ancestry in {\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"};\n","    do\n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/results/9_apoe_genotyping/apoe_snps_${cohort}_${ancestry}.bim\" \\\n","        -iin=\"/wgs_analysis/results/9_apoe_genotyping/apoe_snps_${cohort}_${ancestry}.bed\" \\\n","        -iin=\"/wgs_analysis/results/9_apoe_genotyping/apoe_snps_${cohort}_${ancestry}.fam\" \\\n","        -iin=\"/wgs_analysis/data/apoe_variants.txt\" \\\n","        -icmd=\"plink --bfile apoe_snps_${cohort}_${ancestry} --recode compound-genotypes --out apoe_snps_${cohort}_${ancestry}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/9_apoe_genotyping\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":null,"id":"7f0521bc-98ac-4cc7-839b-b8d1c0de3fe1","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":null,"id":"bda6222a-b673-474c-8958-da4d667cd2ba","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! dx download wgs_analysis/data/APOE_genotypes_PLINK_ped.py\n","for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        ! dx download wgs_analysis/results/9_apoe_genotyping/apoe_snps_{cohort}_{ancestry}.ped\n","        "]},{"cell_type":"code","execution_count":null,"id":"96a6cd4c-70bd-47d3-a0bc-6c7650825415","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        if os.path.exists(f\"apoe_snps_{cohort}_{ancestry}.ped\"):\n","            ! python APOE_genotypes_PLINK_ped.py -i apoe_snps_{cohort}_{ancestry}.ped -o apoe_final_{cohort}_{ancestry}\n","            ! dx upload apoe_final_{cohort}_{ancestry}.APOE_GENOTYPES.csv --path wgs_analysis/results/9_apoe_genotyping/apoe_final_{cohort}_{ancestry}.APOE_GENOTYPES.csv\n","        "]},{"cell_type":"markdown","id":"6cafc9f2-94d8-4dc4-a69b-39e1ec05d2e9","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Combine these results into a table"]},{"cell_type":"code","execution_count":null,"id":"c3d13a32-b0d6-4832-8027-a0caa35d945f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    cohort_counts = []\n","    included_ancestries = []\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        if os.path.exists(f\"apoe_final_{cohort}_{ancestry}.APOE_GENOTYPES.csv\"):\n","            apoe_geno = pd.read_csv(f\"apoe_final_{cohort}_{ancestry}.APOE_GENOTYPES.csv\")\n","            counts = apoe_geno['APOE_GENOTYPE'].value_counts()\n","            counts = counts.reindex([\"e1/e1\", \"e1/e2\", \"e1/e4\", \"e2/e2\", \"e2/e3\", \"e2/e4 or e1/e3\", \"e3/e3\", \"e3/e4\", \"e4/e4\"], fill_value=0)\n","            counts = counts.rename(ancestry)\n","            counts = pd.concat([counts, pd.Series([counts.sum()], index=['total'])])\n","            cohort_counts.append(counts)\n","            included_ancestries.append(ancestry)\n","    cohort_counts = pd.concat(cohort_counts, axis=1)\n","    cohort_counts.columns = included_ancestries\n","    display(cohort_counts)\n","    cohort_percentages = cohort_counts.div(cohort_counts.loc[\"total\"], axis=1) * 100\n","    cohort_percentages.loc[\"total\"] = cohort_counts.loc[\"total\"]\n","    display(cohort_percentages)\n","    cohort_combined = cohort_counts.applymap(str) + ' (' + cohort_percentages.applymap(lambda x: f'{x:.2f}%') + ')'\n","    cohort_combined.loc[\"total\"] = cohort_counts.loc[\"total\"]\n","    display(cohort_combined)\n","    cohort_counts.to_csv(f\"{cohort}_apoe_genotype_counts.csv\")\n","    cohort_percentages.to_csv(f\"{cohort}_apoe_genotype_percentages.csv\")\n","    cohort_combined.to_csv(f\"{cohort}_apoe_genotype_combined.csv\")\n","    ! dx upload {cohort}_apoe_genotype_counts.csv --path wgs_analysis/results/9_apoe_genotyping/{cohort}_apoe_genotype_counts.csv\n","    ! dx upload {cohort}_apoe_genotype_percentages.csv --path wgs_analysis/results/9_apoe_genotyping/{cohort}_apoe_genotype_percentages.csv\n","    ! dx upload {cohort}_apoe_genotype_combined.csv --path wgs_analysis/results/9_apoe_genotyping/{cohort}_apoe_genotype_combined.csv\n","    "]},{"cell_type":"markdown","id":"5cd28104-ad75-4628-aca0-dc5794c3e262","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Find number of controls in pathogenic variants"]},{"cell_type":"code","execution_count":null,"id":"be45b744-0cbe-4d73-8678-dac9fd79ce87","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! dx download wgs_analysis/results/7_zygosity/counts_Control_AAC.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_AFR.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_AJ.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_AMR.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_CAH.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_CAS.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_EAS.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_EUR.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_FIN.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_MDE.tsv\n","! dx download wgs_analysis/results/7_zygosity/counts_Control_SAS.tsv\n"]},{"cell_type":"code","execution_count":null,"id":"2aa98ca7-d2e1-4c8b-ad2d-96fce6e078b5","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["control_zyg = []\n","for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","    df = pd.read_csv(f\"counts_Control_{ancestry}.tsv\", sep=\"\\t\")\n","    variant_ids = df.columns.values\n","    rename_dict = {}\n","    for vid in variant_ids:\n","        rename_dict[vid] = vid.split(\"_\")[0]\n","    df.rename(rename_dict, axis=1, inplace=True)\n","    df = df.T\n","    df = df.reset_index()\n","    df.rename(columns={'index': 'ID', 0: f'{ancestry}_Homozygous_Ref', 1: f'{ancestry}_Heterozygous', 2: f'{ancestry}_Homozygous_Alt'}, inplace=True)\n","    df[ancestry] = df[f'{ancestry}_Heterozygous'] + df[f'{ancestry}_Homozygous_Alt']\n","    df = df[[\"ID\",ancestry]]\n","    control_zyg.append(df)\n"]},{"cell_type":"code","execution_count":null,"id":"7c1de2ef-5e1e-44c7-87d5-d40746b79d22","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["control_zyg_merged = reduce(lambda left, right: pd.merge(left, right, on=\"ID\"), control_zyg)\n"]},{"cell_type":"code","execution_count":null,"id":"2f91eddb-2eda-4f8e-9356-63b73964b7fc","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["pathogenic_vars = [\n","    \"chr1:155235196\",\n","    \"chr1:155235217\",\n","    \"chr1:155235252\",\n","    \"chr1:155235727\",\n","    \"chr1:155235790\",\n","    \"chr1:155235823\",\n","    \"chr1:155235843\",\n","    \"chr1:155236277\",\n","    \"chr1:155237453\",\n","    \"chr1:155238174\",\n","    \"chr1:155238214\",\n","    \"chr1:155238215\",\n","    \"chr1:155238260\",\n","    \"chr1:155238630\",\n","    \"chr1:155240629\",\n","    \"chr17:44350262\",\n","    \"chr17:44350800\",\n","    \"chr17:44351409\",\n","    \"chr4:89828156\",\n","]\n"]},{"cell_type":"code","execution_count":null,"id":"4e462420-1774-43e3-a3b2-5432ade106ea","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["filtered_control_var_counts = control_zyg_merged[control_zyg_merged['ID'].str.startswith(tuple(pathogenic_vars))]\n","filtered_control_var_counts.to_csv(\"filtered_control_var_counts.csv\", index=False)\n"]},{"cell_type":"code","execution_count":null,"id":"f82d41a5-c3e1-4c6b-859e-beab33f7a351","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["! dx upload filtered_control_var_counts.csv --path wgs_analysis/results/10_pathogenic_variants/filtered_control_var_counts.csv\n"]},{"cell_type":"markdown","id":"10edb5f8-45c3-4232-9bdc-fcb80ec988b5","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["# Resilience/protective variants"]},{"cell_type":"code","execution_count":34,"id":"e82d0d96-17fd-459e-b68d-7dc683c9b7a9","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["Error: path \"/opt/notebooks/final_filtered_onlycases.csv\" already exists but\n","-f/--overwrite was not set\n","Error: path \"/opt/notebooks/merged.csv\" already exists but -f/--overwrite was\n","not set\n","\u001b[2K[===========================================================>] Completed 2,921 of 2,921 bytes (100%) /opt/notebooks/protein_var_map.csvsv\n"]}],"source":["! dx download wgs_analysis/results/8_merged/final_filtered_onlycases.csv\n","! dx download wgs_analysis/results/8_merged/merged.csv\n","! dx download wgs_analysis/data/protein_var_map.csv --overwrite\n"]},{"cell_type":"code","execution_count":50,"id":"ac64aa7f-d33f-4fbd-a2e7-71d52cb90c76","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["filtered_var_counts = pd.read_csv(\"final_filtered_onlycases.csv\")\n","filtered_var_counts = filtered_var_counts[filtered_var_counts[\"CADD\"] >= 20]\n","filtered_var_ids = list(filtered_var_counts[\"ID\"])\n"]},{"cell_type":"code","execution_count":52,"id":"e6fde9c5-37d4-461a-a196-23b3f0e66dcb","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[2K[===========================================================>] Completed 21,387,879 of 21,387,879 bytes (100%) /opt/notebooks/Control_EUR.afreqeq\n"]}],"source":["! dx download wgs_analysis/results/6_frequencies/Control_EUR.afreq --overwrite\n","df_freq = pd.read_csv(\"Control_EUR.afreq\", sep=\"\\t\")\n","df_merged = pd.read_csv(\"merged.csv\")\n","df_merged.insert(2, \"VCF_ID\", list(df_freq[\"ID\"]))\n"]},{"cell_type":"code","execution_count":53,"id":"aa3aec76-607c-47ae-a3d9-cda240ad2a0f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["df_merged_filtered = df_merged[df_merged[\"ID\"].isin(filtered_var_ids)]\n","filtered_vcf_ids = list(df_merged_filtered[\"VCF_ID\"])\n"]},{"cell_type":"code","execution_count":54,"id":"a62c7e40-6950-49f4-a40b-4af649672975","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 1,844 of 1,844 bytes (100%) variants_to_keep.txt\n","ID                                file-GpkGGXjJYBPjzXVJg5z7xfqf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/11_phenotypic_data\n","Name                              variant_ids.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug  7 00:56:43 2024\n","Created by                        spencermg3\n"," via the job                      job-GpkFP58JYBPf9gjx0J49ZvPG\n","Last modified                     Wed Aug  7 00:56:44 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n"]}],"source":["file_path = 'variants_to_keep.txt'\n","with open(file_path, 'w') as file:\n","    for variant_id in filtered_vcf_ids:\n","        file.write(f\"{variant_id}\\n\")\n","! dx upload variants_to_keep.txt --path wgs_analysis/results/11_phenotypic_data/variant_ids.txt\n","        "]},{"cell_type":"code","execution_count":null,"id":"bb1683e5-9f5b-431d-950e-6f8f0b663d20","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash\n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","-iin=\"/wgs_analysis/results/11_phenotypic_data/variant_ids.txt\" \\\n","-icmd=\"plink2 --vcf normalized.vcf --set-all-var-ids 'chr@:#:\\$r:\\$a' --new-id-max-allele-len 999 --extract variant_ids.txt --freq --out full_cohort\" \\\n","--instance-type mem1_hdd1_v2_x16 \\\n","--destination \"${projectid}:/wgs_analysis/results/11_phenotypic_data\"\n"]},{"cell_type":"code","execution_count":null,"id":"9271aad0-ad50-409a-a3fb-f5bbae3b45ec","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["%%bash \n","\n","dx run swiss-army-knife \\\n","-iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","-iin=\"/wgs_analysis/results/11_phenotypic_data/full_cohort.afreq\" \\\n","-iin=\"/wgs_analysis/results/11_phenotypic_data/variant_ids.txt\" \\\n","-icmd=\"plink2 --vcf normalized.vcf --set-all-var-ids 'chr@:#:\\$r:\\$a' --new-id-max-allele-len 999 --extract variant_ids.txt --read-freq full_cohort.afreq --export A --het --out full_cohort\" \\\n","--instance-type mem1_hdd1_v2_x16 \\\n","--destination \"${projectid}:/wgs_analysis/results/11_phenotypic_data\"\n"]},{"cell_type":"code","execution_count":55,"id":"534c3f94-5225-429b-bf2a-8226268ab15a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[2K[===========================================================>] Completed 13,852,933 of 13,852,933 bytes (100%) /opt/notebooks/full_cohort.rawaw\n"]}],"source":["! dx download wgs_analysis/results/11_phenotypic_data/full_cohort.raw --overwrite\n"]},{"cell_type":"code","execution_count":56,"id":"d594e248-a96a-44cc-9b1a-a335f510e603","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["df_raw = pd.read_csv(\"full_cohort.raw\", sep=\"\\t\")\n","variant_ids = df_raw.columns.values\n","rename_dict = {}\n","for vid in variant_ids:\n","    rename_dict[vid] = vid.split(\"_\")[0]\n","df_raw.rename(rename_dict, axis=1, inplace=True)\n","df_raw.drop(columns=[\"FID\",\"PAT\",\"MAT\",\"SEX\",\"PHENOTYPE\"], inplace=True)\n","df_raw.fillna(2, inplace=True)\n"]},{"cell_type":"code","execution_count":57,"id":"fb3902bd-b11a-4ff9-9a56-47e9aa5650b0","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["list_vars = []\n","list_vars_chrpos = []\n","list_ids = []\n","for variant in df_raw.columns[1:]:\n","    var_ids = df_raw[df_raw[variant] != 2]['IID'].tolist()\n","    for iid in var_ids:\n","        list_vars.append(variant)\n","        var_chrpos = \":\".join(variant.split(\":\")[:2])\n","        list_vars_chrpos.append(var_chrpos)\n","        list_ids.append(iid)\n","    \n","#id_var_mapper = {}    \n","#for index, row in df_raw.iterrows():\n","#    iid = int(row['IID'])\n","#    \n","#    list_vars = [col for col in df_raw.columns[1:] if row[col] != 2]\n","#    if len(list_vars) > 0:\n","#        id_var_mapper[iid] = list_vars\n","#    "]},{"cell_type":"code","execution_count":58,"id":"a3126f57-477e-43d0-b22e-fdcdd3dfc526","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["df_pheno = pd.DataFrame({\n","    \"Variant_ID_Full\":list_vars,\n","    \"Variant_ID\":list_vars_chrpos,\n","    \"Participant_ID\":list_ids,\n","})\n"]},{"cell_type":"code","execution_count":59,"id":"c05b1a9e-b587-4443-b4f7-35b6cf4c32be","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["protein_var_map = pd.read_csv(\"protein_var_map.csv\")\n","df_pheno = df_pheno.merge(protein_var_map, on=\"Variant_ID\", how=\"left\")\n"]},{"cell_type":"code","execution_count":60,"id":"b11a30d3-4ff4-4995-834d-d464831a6eaf","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>Variant_ID_Full</th>\n","      <th>Variant_ID</th>\n","      <th>Participant_ID</th>\n","      <th>Protein_change</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>chr1:155235769:G:A</td>\n","      <td>chr1:155235769</td>\n","      <td>3520686</td>\n","      <td>p.R347C</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>chr1:155236249:A:G</td>\n","      <td>chr1:155236249</td>\n","      <td>3646248</td>\n","      <td>p.I320T</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>chr1:155236249:A:G</td>\n","      <td>chr1:155236249</td>\n","      <td>2703400</td>\n","      <td>p.I320T</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>chr1:155236262:T:G</td>\n","      <td>chr1:155236262</td>\n","      <td>2114999</td>\n","      <td>p.S316R</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>chr1:155236262:T:G</td>\n","      <td>chr1:155236262</td>\n","      <td>5365927</td>\n","      <td>p.S316R</td>\n","    </tr>\n","    <tr>\n","      <th>...</th>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","    </tr>\n","    <tr>\n","      <th>125</th>\n","      <td>chr21:25982424:G:A</td>\n","      <td>chr21:25982424</td>\n","      <td>1682723</td>\n","      <td>p.P251S</td>\n","    </tr>\n","    <tr>\n","      <th>126</th>\n","      <td>chr21:26000138:T:C</td>\n","      <td>chr21:26000138</td>\n","      <td>2354670</td>\n","      <td>p.I248V</td>\n","    </tr>\n","    <tr>\n","      <th>127</th>\n","      <td>chr21:26021858:C:G</td>\n","      <td>chr21:26021858</td>\n","      <td>3620157</td>\n","      <td>p.V227L</td>\n","    </tr>\n","    <tr>\n","      <th>128</th>\n","      <td>chr21:26021912:C:T</td>\n","      <td>chr21:26021912</td>\n","      <td>4998524</td>\n","      <td>p.A209T</td>\n","    </tr>\n","    <tr>\n","      <th>129</th>\n","      <td>chr21:26051100:G:A</td>\n","      <td>chr21:26051100</td>\n","      <td>2202257</td>\n","      <td>p.P132S</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>130 rows  4 columns</p>\n","</div>"],"text/plain":["        Variant_ID_Full      Variant_ID  Participant_ID Protein_change\n","0    chr1:155235769:G:A  chr1:155235769         3520686        p.R347C\n","1    chr1:155236249:A:G  chr1:155236249         3646248        p.I320T\n","2    chr1:155236249:A:G  chr1:155236249         2703400        p.I320T\n","3    chr1:155236262:T:G  chr1:155236262         2114999        p.S316R\n","4    chr1:155236262:T:G  chr1:155236262         5365927        p.S316R\n","..                  ...             ...             ...            ...\n","125  chr21:25982424:G:A  chr21:25982424         1682723        p.P251S\n","126  chr21:26000138:T:C  chr21:26000138         2354670        p.I248V\n","127  chr21:26021858:C:G  chr21:26021858         3620157        p.V227L\n","128  chr21:26021912:C:T  chr21:26021912         4998524        p.A209T\n","129  chr21:26051100:G:A  chr21:26051100         2202257        p.P132S\n","\n","[130 rows x 4 columns]"]},"metadata":{},"output_type":"display_data"}],"source":["display(df_pheno)"]},{"cell_type":"code","execution_count":45,"id":"a3df2b53-b78d-4628-a821-a0aab67cb301","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["Error: path \"/opt/notebooks/AD_cases.csv\" already exists but -f/--overwrite\n","was not set\n","Error: path \"/opt/notebooks/RD_cases.csv\" already exists but -f/--overwrite\n","was not set\n"]}],"source":["! dx download wgs_analysis/results/AD_cases.csv\n","! dx download wgs_analysis/results/RD_cases.csv\n"]},{"cell_type":"code","execution_count":61,"id":"5d58abf3-e3e2-4049-9b32-535ae8a58637","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["df_ad = pd.read_csv(\"AD_cases.csv\")\n","df_rd = pd.read_csv(\"RD_cases.csv\")\n","\n","df_ad.rename({\"AD_DATE\":\"DATE_OF_ONSET\", \"label\":\"ANCESTRY\", \"ID\":\"Participant_ID\"}, inplace=True, axis=1)\n","df_rd.rename({\"DEM_DATE\":\"DATE_OF_ONSET\", \"label\":\"ANCESTRY\", \"ID\":\"Participant_ID\"}, inplace=True, axis=1)\n","\n","df_dem = pd.concat([df_ad, df_rd], axis=0)\n","df_dem.reset_index(inplace=True, drop=True)\n"]},{"cell_type":"code","execution_count":62,"id":"dd313b85-d5a0-4da8-b670-1896372d0c4f","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[],"source":["df_dem[\"AGE_AT_ONSET\"] = (pd.to_datetime(df_dem['DATE_OF_ONSET']) - pd.to_datetime(df_dem[\"BIRTH_YEAR\"], format='%Y')).dt.days // 365.242374\n","df_dem[\"DAYS_SINCE_ONSET\"] = (pd.to_datetime(df_dem['DATE_OF_DEATH'].fillna(datetime.today().strftime('%Y-%m-%d'))) - pd.to_datetime(df_dem['DATE_OF_ONSET'])).dt.days\n","df_dem[\"AGE\"] = (pd.to_datetime(df_dem['DATE_OF_DEATH'].fillna(datetime.today().strftime('%Y-%m-%d'))) - pd.to_datetime(df_dem['BIRTH_YEAR'], format='%Y')).dt.days // 365.242374\n"]},{"cell_type":"code","execution_count":63,"id":"84fb9cca-c1f6-443c-a7d2-7930848a2a18","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["(130, 4)\n"]}],"source":["print(df_pheno.shape)\n","\n","df_pheno_1 = df_pheno.merge(df_dem[[\n","    \"Participant_ID\",\"GENETIC_SEX\",\"AGE\",\"DATE_OF_ONSET\",\"DATE_OF_DEATH\",\"AGE_AT_ONSET\",\n","    \"DAYS_SINCE_ONSET\",\"ANCESTRY\",\"COGNITIVE_SYMPTOMS_SEVERITY_PAST_WEEK\", \n","    \"SPECIFIC_COGNITIVE_ABILITY_2014\",\"SPECIFIC_COGNITIVE_ABILITY_2019\", \n","    \"TOUCHSCREEN_COGNITIVE_DURATION_2014\",\"TOUCHSCREEN_COGNITIVE_DURATION_2019\", \n","    \"COGNITIVE_TEST_WILLINGNESS\",\"COGNITIVE_TEST_WILLINGNESS\", \"COGNITIVE_TEST_WILLINGNESS\",\n","]], on=\"Participant_ID\", how=\"inner\")\n"]},{"cell_type":"code","execution_count":64,"id":"14846d1c-3e15-4ea7-81a7-17370c6e1a2c","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>Variant_ID_Full</th>\n","      <th>Variant_ID</th>\n","      <th>Participant_ID</th>\n","      <th>Protein_change</th>\n","      <th>GENETIC_SEX</th>\n","      <th>AGE</th>\n","      <th>DATE_OF_ONSET</th>\n","      <th>DATE_OF_DEATH</th>\n","      <th>AGE_AT_ONSET</th>\n","      <th>DAYS_SINCE_ONSET</th>\n","      <th>ANCESTRY</th>\n","      <th>COGNITIVE_SYMPTOMS_SEVERITY_PAST_WEEK</th>\n","      <th>SPECIFIC_COGNITIVE_ABILITY_2014</th>\n","      <th>SPECIFIC_COGNITIVE_ABILITY_2019</th>\n","      <th>TOUCHSCREEN_COGNITIVE_DURATION_2014</th>\n","      <th>TOUCHSCREEN_COGNITIVE_DURATION_2019</th>\n","      <th>COGNITIVE_TEST_WILLINGNESS</th>\n","      <th>COGNITIVE_TEST_WILLINGNESS</th>\n","      <th>COGNITIVE_TEST_WILLINGNESS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>chr1:155235769:G:A</td>\n","      <td>chr1:155235769</td>\n","      <td>3520686</td>\n","      <td>p.R347C</td>\n","      <td>Male</td>\n","      <td>78.0</td>\n","      <td>2017-03-23</td>\n","      <td>2017-08-11</td>\n","      <td>78.0</td>\n","      <td>141</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>chr1:155236249:A:G</td>\n","      <td>chr1:155236249</td>\n","      <td>3646248</td>\n","      <td>p.I320T</td>\n","      <td>Male</td>\n","      <td>77.0</td>\n","      <td>2021-09-12</td>\n","      <td>2021-10-06</td>\n","      <td>77.0</td>\n","      <td>24</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>chr1:155236249:A:G</td>\n","      <td>chr1:155236249</td>\n","      <td>2703400</td>\n","      <td>p.I320T</td>\n","      <td>Male</td>\n","      <td>67.0</td>\n","      <td>2013-05-04</td>\n","      <td>2013-05-07</td>\n","      <td>67.0</td>\n","      <td>3</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>chr1:155236262:T:G</td>\n","      <td>chr1:155236262</td>\n","      <td>2114999</td>\n","      <td>p.S316R</td>\n","      <td>Female</td>\n","      <td>73.0</td>\n","      <td>2017-12-01</td>\n","      <td>2018-05-24</td>\n","      <td>72.0</td>\n","      <td>174</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>chr1:155236262:T:G</td>\n","      <td>chr1:155236262</td>\n","      <td>5365927</td>\n","      <td>p.S316R</td>\n","      <td>Male</td>\n","      <td>63.0</td>\n","      <td>2014-12-22</td>\n","      <td>2015-11-15</td>\n","      <td>62.0</td>\n","      <td>328</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>...</th>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","      <td>...</td>\n","    </tr>\n","    <tr>\n","      <th>125</th>\n","      <td>chr21:25982424:G:A</td>\n","      <td>chr21:25982424</td>\n","      <td>1682723</td>\n","      <td>p.P251S</td>\n","      <td>Male</td>\n","      <td>83.0</td>\n","      <td>2022-07-19</td>\n","      <td>2022-07-19</td>\n","      <td>83.0</td>\n","      <td>0</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>126</th>\n","      <td>chr21:26000138:T:C</td>\n","      <td>chr21:26000138</td>\n","      <td>2354670</td>\n","      <td>p.I248V</td>\n","      <td>Female</td>\n","      <td>82.0</td>\n","      <td>2022-07-26</td>\n","      <td>NaN</td>\n","      <td>80.0</td>\n","      <td>743</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>127</th>\n","      <td>chr21:26021858:C:G</td>\n","      <td>chr21:26021858</td>\n","      <td>3620157</td>\n","      <td>p.V227L</td>\n","      <td>Female</td>\n","      <td>59.0</td>\n","      <td>2017-10-30</td>\n","      <td>2017-12-07</td>\n","      <td>59.0</td>\n","      <td>38</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>128</th>\n","      <td>chr21:26021912:C:T</td>\n","      <td>chr21:26021912</td>\n","      <td>4998524</td>\n","      <td>p.A209T</td>\n","      <td>Male</td>\n","      <td>65.0</td>\n","      <td>2020-10-06</td>\n","      <td>2020-11-26</td>\n","      <td>65.0</td>\n","      <td>51</td>\n","      <td>EUR</td>\n","      <td>No problem</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","    <tr>\n","      <th>129</th>\n","      <td>chr21:26051100:G:A</td>\n","      <td>chr21:26051100</td>\n","      <td>2202257</td>\n","      <td>p.P132S</td>\n","      <td>Female</td>\n","      <td>82.0</td>\n","      <td>2021-03-02</td>\n","      <td>NaN</td>\n","      <td>79.0</td>\n","      <td>1254</td>\n","      <td>EUR</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>NaN</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","      <td>Begin games</td>\n","    </tr>\n","  </tbody>\n","</table>\n","<p>130 rows  19 columns</p>\n","</div>"],"text/plain":["        Variant_ID_Full      Variant_ID  Participant_ID Protein_change  \\\n","0    chr1:155235769:G:A  chr1:155235769         3520686        p.R347C   \n","1    chr1:155236249:A:G  chr1:155236249         3646248        p.I320T   \n","2    chr1:155236249:A:G  chr1:155236249         2703400        p.I320T   \n","3    chr1:155236262:T:G  chr1:155236262         2114999        p.S316R   \n","4    chr1:155236262:T:G  chr1:155236262         5365927        p.S316R   \n","..                  ...             ...             ...            ...   \n","125  chr21:25982424:G:A  chr21:25982424         1682723        p.P251S   \n","126  chr21:26000138:T:C  chr21:26000138         2354670        p.I248V   \n","127  chr21:26021858:C:G  chr21:26021858         3620157        p.V227L   \n","128  chr21:26021912:C:T  chr21:26021912         4998524        p.A209T   \n","129  chr21:26051100:G:A  chr21:26051100         2202257        p.P132S   \n","\n","    GENETIC_SEX   AGE DATE_OF_ONSET DATE_OF_DEATH  AGE_AT_ONSET  \\\n","0          Male  78.0    2017-03-23    2017-08-11          78.0   \n","1          Male  77.0    2021-09-12    2021-10-06          77.0   \n","2          Male  67.0    2013-05-04    2013-05-07          67.0   \n","3        Female  73.0    2017-12-01    2018-05-24          72.0   \n","4          Male  63.0    2014-12-22    2015-11-15          62.0   \n","..          ...   ...           ...           ...           ...   \n","125        Male  83.0    2022-07-19    2022-07-19          83.0   \n","126      Female  82.0    2022-07-26           NaN          80.0   \n","127      Female  59.0    2017-10-30    2017-12-07          59.0   \n","128        Male  65.0    2020-10-06    2020-11-26          65.0   \n","129      Female  82.0    2021-03-02           NaN          79.0   \n","\n","     DAYS_SINCE_ONSET ANCESTRY COGNITIVE_SYMPTOMS_SEVERITY_PAST_WEEK  \\\n","0                 141      EUR                                   NaN   \n","1                  24      EUR                                   NaN   \n","2                   3      EUR                                   NaN   \n","3                 174      EUR                                   NaN   \n","4                 328      EUR                                   NaN   \n","..                ...      ...                                   ...   \n","125                 0      EUR                                   NaN   \n","126               743      EUR                                   NaN   \n","127                38      EUR                                   NaN   \n","128                51      EUR                            No problem   \n","129              1254      EUR                                   NaN   \n","\n","     SPECIFIC_COGNITIVE_ABILITY_2014  SPECIFIC_COGNITIVE_ABILITY_2019  \\\n","0                                NaN                              NaN   \n","1                                NaN                              NaN   \n","2                                NaN                              NaN   \n","3                                NaN                              NaN   \n","4                                NaN                              NaN   \n","..                               ...                              ...   \n","125                              NaN                              NaN   \n","126                              NaN                              NaN   \n","127                              NaN                              NaN   \n","128                              NaN                              NaN   \n","129                              NaN                              NaN   \n","\n","     TOUCHSCREEN_COGNITIVE_DURATION_2014  TOUCHSCREEN_COGNITIVE_DURATION_2019  \\\n","0                                    NaN                                  NaN   \n","1                                    NaN                                  NaN   \n","2                                    NaN                                  NaN   \n","3                                    NaN                                  NaN   \n","4                                    NaN                                  NaN   \n","..                                   ...                                  ...   \n","125                                  NaN                                  NaN   \n","126                                  NaN                                  NaN   \n","127                                  NaN                                  NaN   \n","128                                  NaN                                  NaN   \n","129                                  NaN                                  NaN   \n","\n","    COGNITIVE_TEST_WILLINGNESS COGNITIVE_TEST_WILLINGNESS  \\\n","0                  Begin games                Begin games   \n","1                  Begin games                Begin games   \n","2                  Begin games                Begin games   \n","3                  Begin games                Begin games   \n","4                  Begin games                Begin games   \n","..                         ...                        ...   \n","125                Begin games                Begin games   \n","126                Begin games                Begin games   \n","127                Begin games                Begin games   \n","128                Begin games                Begin games   \n","129                Begin games                Begin games   \n","\n","    COGNITIVE_TEST_WILLINGNESS  \n","0                  Begin games  \n","1                  Begin games  \n","2                  Begin games  \n","3                  Begin games  \n","4                  Begin games  \n","..                         ...  \n","125                Begin games  \n","126                Begin games  \n","127                Begin games  \n","128                Begin games  \n","129                Begin games  \n","\n","[130 rows x 19 columns]"]},"metadata":{},"output_type":"display_data"}],"source":["display(df_pheno_1)\n","df_pheno_1.to_csv(\"pheno.csv\")\n"]},{"cell_type":"markdown","id":"a3b8ffe0-08a0-4e4d-9c65-cfd381f45772","metadata":{"jp-MarkdownHeadingCollapsed":true,"tags":[]},"source":["## Find variants expressed by anyone in the dataset"]},{"cell_type":"code","execution_count":null,"id":"7946f075-a0bd-4eb5-815f-fb7f5c4ee362","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["%%bash\n","\n","for cohort in {\"AD\",\"RD\",\"Control\"};\n","do\n","    for ancestry in {\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"};\n","    do\n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/data/protective_variants.txt\" \\\n","        -iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","        -iin=\"/wgs_analysis/data/ID_Files/${cohort}_${ancestry}.txt\" \\\n","        -iin=\"/wgs_analysis/results/6_frequencies/${cohort}_${ancestry}.afreq\" \\\n","        -icmd=\"plink2 --vcf normalized.vcf --set-all-var-ids 'chr@:#:\\$r:\\$a' --new-id-max-allele-len 999 --extract protective_variants.txt --keep ${cohort}_${ancestry}.txt --read-freq ${cohort}_${ancestry}.afreq --export A --het --out ${cohort}_${ancestry}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/12_protective_variants\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":18,"id":"3874c516-e4da-4aa0-b3eb-736241bd74a6","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[2K[===========================================================>] Completed 573 of 573 bytes (100%) /opt/notebooks/AD_AAC.rawaw\n","chr19_44892887_C_T\n","[===========================================================>] Uploaded 32 of 32 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zVg8JYBPv69029zV67Pjj\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:01:01 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:01:01 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr19_44905307_A_T\n","[===========================================================>] Uploaded 72 of 72 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zVgQJYBPYFf7xPX00QG8k\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:01:02 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:01:03 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr19_44908756_C_A\n","chr19_44908756_C_T\n","chr21_25897620_C_T\n","chr21_26171645_A_G\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zVj0JYBPgG398X3y9KYkv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:01:04 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:01:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr21_26171723_T_C\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zVjQJYBPgG398X3y9KYp0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:01:06 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:01:07 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>IID</th>\n","      <th>chr19:44892887:C:T</th>\n","      <th>chr19:44905307:A:T</th>\n","      <th>chr19:44908756:C:A</th>\n","      <th>chr19:44908756:C:T</th>\n","      <th>chr21:25897620:C:T</th>\n","      <th>chr21:26171645:A:G</th>\n","      <th>chr21:26171723:T:C</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>2592890</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>3219025</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>5343805</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>5405583</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>1831922</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>5</th>\n","      <td>1294579</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>6</th>\n","      <td>3043640</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>7</th>\n","      <td>4976837</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>8</th>\n","      <td>4595192</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>9</th>\n","      <td>5255423</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>10</th>\n","      <td>3350038</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>11</th>\n","      <td>4360655</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>1</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["        IID  chr19:44892887:C:T  chr19:44905307:A:T  chr19:44908756:C:A  \\\n","0   2592890                   1                   2                   2   \n","1   3219025                   1                   2                   2   \n","2   5343805                   2                   1                   2   \n","3   5405583                   2                   1                   2   \n","4   1831922                   1                   1                   2   \n","5   1294579                   2                   0                   2   \n","6   3043640                   2                   1                   2   \n","7   4976837                   2                   1                   2   \n","8   4595192                   2                   2                   2   \n","9   5255423                   1                   1                   2   \n","10  3350038                   2                   0                   2   \n","11  4360655                   2                   0                   2   \n","\n","    chr19:44908756:C:T  chr21:25897620:C:T  chr21:26171645:A:G  \\\n","0                    2                   2                   2   \n","1                    2                   2                   2   \n","2                    2                   2                   2   \n","3                    2                   2                   2   \n","4                    2                   2                   2   \n","5                    2                   2                   2   \n","6                    2                   2                   1   \n","7                    2                   2                   2   \n","8                    2                   2                   2   \n","9                    2                   2                   2   \n","10                   2                   2                   2   \n","11                   2                   2                   1   \n","\n","    chr21:26171723:T:C  \n","0                    2  \n","1                    2  \n","2                    2  \n","3                    2  \n","4                    2  \n","5                    2  \n","6                    1  \n","7                    2  \n","8                    2  \n","9                    2  \n","10                   2  \n","11                   1  "]},"metadata":{},"output_type":"display_data"}],"source":["cohort = \"AD\"\n","ancestry = \"AAC\"\n","! dx download wgs_analysis/results/12_protective_variants/{cohort}_{ancestry}.raw --overwrite\n","df = pd.read_csv(f\"{cohort}_{ancestry}.raw\", sep=\"\\t\")\n","df.drop([\"FID\",\"PAT\",\"MAT\",\"SEX\",\"PHENOTYPE\"], axis=1, inplace=True)\n","df.columns = df.columns.str.split('_').str[0]\n","for var_id in df.columns[1:]:\n","    participant_ids = list(df[\"IID\"][df[var_id] < 2])\n","    if len(participant_ids) > 0:\n","        with open(f\"{cohort}_{ancestry}.txt\", \"w\") as file:\n","            for item in participant_ids:\n","                file.write(f\"{item}\\n\")\n","        ! dx upload {cohort}_{ancestry}.txt --path wgs_analysis/results/12_protective_variants/{var_id.replace(\":\",\"_\")}/{cohort}_{ancestry}.txt\n","display(df)\n"]},{"cell_type":"code","execution_count":null,"id":"b24655cb-3a70-42ce-8910-52e21f00f221","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[2K[===========================================================>] Completed 573 of 573 bytes (100%) /opt/notebooks/AD_AAC.rawaw\n","[===========================================================>] Uploaded 32 of 32 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zb2jJYBPYFf7xPX00QGV2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:08:11 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:08:48 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 72 of 72 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zbG8JYBPq7J1ygzFB6vpv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:08:49 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:08:51 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zbJ0JYBPQ7b3k77P9Q2ZQ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:08:52 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:08:53 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_AAC.txt\n","ID                                file-Gq1zbJQJYBPYFf7xPX00QGX3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:08:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:08:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 1,893 of 1,893 bytes (100%) /opt/notebooks/AD_AFR.rawaw\n","[===========================================================>] Uploaded 72 of 72 bytes (100%) AD_AFR.txt\n","ID                                file-Gq1zbK8JYBPQ7b3k77P9Q2ZX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:08:57 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:09:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 256 of 256 bytes (100%) AD_AFR.txt\n","ID                                file-Gq1zbfjJYBPq7J1ygzFB6vq6\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:09:31 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:09:32 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 120 of 120 bytes (100%) AD_AFR.txt\n","ID                                file-Gq1zbg8JYBPQ7b3k77P9Q2b0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:09:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:09:33 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 120 of 120 bytes (100%) AD_AFR.txt\n","ID                                file-Gq1zbgQJYBPX0q2K61g175pZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:09:34 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:10:04 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 1,068 of 1,068 bytes (100%) /opt/notebooks/AD_AJ.rawaw\n","[===========================================================>] Uploaded 112 of 112 bytes (100%) AD_AJ.txt\n","ID                                file-Gq1zf0jJYBPkjKf7jXj47F90\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:10:11 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:10:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 64 of 64 bytes (100%) AD_AJ.txt\n","ID                                file-Gq1zf1QJYBPX0q2K61g175qJ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:10:14 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:10:56 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) AD_AJ.txt\n","ID                                file-Gq1zfG8JYBPz5ZKbFJ888Y0v\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:10:57 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) AD_AJ.txt\n","ID                                file-Gq1zfJ8JYBPq7J1ygzFB6vvZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:01 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:02 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","dxpy.utils.resolver.ResolutionError: Unable to resolve \"AD_AMR.raw\" to a data object or folder name in '/wgs_analysis/results/12_protective_variants'\n","\u001b[2K[===========================================================>] Completed 441 of 441 bytes (100%) /opt/notebooks/AD_CAH.rawaw\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) AD_CAH.txt\n","ID                                file-Gq1zfK8JYBPq7J1ygzFB6vvy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:05 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:06 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 24 of 24 bytes (100%) AD_CAH.txt\n","ID                                file-Gq1zfKjJYBPbb4F2QJkP4B77\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:07 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_CAH.txt\n","ID                                file-Gq1zfP0JYBPbb4F2QJkP4B7F\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:08 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:09 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_CAH.txt\n","ID                                file-Gq1zfPQJYBPYY1B11py26bXG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:11 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 507 of 507 bytes (100%) /opt/notebooks/AD_CAS.rawaw\n","[===========================================================>] Uploaded 24 of 24 bytes (100%) AD_CAS.txt\n","ID                                file-Gq1zfQ8JYBPYY1B11py26bXX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:14 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 32 of 32 bytes (100%) AD_CAS.txt\n","ID                                file-Gq1zfQQJYBPQKKGYPXPJzZkz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:14 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) AD_CAS.txt\n","ID                                file-Gq1zfV0JYBPYFf7xPX00QGYx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:17 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) AD_CAS.txt\n","ID                                file-Gq1zfVQJYBPq7J1ygzFB6vxB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:18 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:11:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 408 of 408 bytes (100%) /opt/notebooks/AD_EAS.rawaw\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_EAS.txt\n","ID                                file-Gq1zfYjJYBPgG398X3y9KZx4\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_EAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:11:27 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:12:06 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) AD_EAS.txt\n","ID                                file-Gq1zfxjJYBPz5ZKbFJ888Y20\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_EAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:12:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:12:07 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 24 of 24 bytes (100%) AD_EAS.txt\n","ID                                file-Gq1zfy0JYBPYFf7xPX00QGZg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_EAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:12:08 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:12:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 24 of 24 bytes (100%) AD_EAS.txt\n","ID                                file-Gq1zg4jJYBPv69029zV67Qx3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_EAS.txt\n","State                             \u001b[32mclosed\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:12:35 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:12:37 2024\n","Media type                        text/plain\n","archivalState                     \"live\"\n","Size                              24 bytes\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 133,860 of 133,860 bytes (100%) /opt/notebooks/AD_EUR.rawaw\n","[===========================================================>] Uploaded 17,160 of 17,160 bytes (100%) AD_EUR.txt\n","ID                                file-Gq1zgJQJYBPX0q2K61g175y0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:13:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:13:11 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 7,032 of 7,032 bytes (100%) AD_EUR.txt\n","ID                                file-Gq1zgK0JYBPQ7b3k77P9Q2x0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:13:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:13:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,416 of 1,416 bytes (100%) AD_EUR.txt\n","ID                                file-Gq1zgPjJYBPv69029zV67Qxv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:13:19 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:13:20 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,432 of 1,432 bytes (100%) AD_EUR.txt\n","ID                                file-Gq1zgQ8JYBPv69029zV67Qy0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:13:21 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:14:07 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","dxpy.utils.resolver.ResolutionError: Unable to resolve \"AD_FIN.raw\" to a data object or folder name in '/wgs_analysis/results/12_protective_variants'\n","\u001b[2K[===========================================================>] Completed 474 of 474 bytes (100%) /opt/notebooks/AD_MDE.rawaw\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) AD_MDE.txt\n","ID                                file-Gq1zgvQJYBPq7J1ygzFB6x0q\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_MDE.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:14:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:14:31 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 48 of 48 bytes (100%) AD_MDE.txt\n","ID                                file-Gq1zj4QJYBPkyVQ6kkf43bg2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_MDE.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:14:42 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:14:43 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 1,794 of 1,794 bytes (100%) /opt/notebooks/AD_SAS.rawaw\n","[===========================================================>] Uploaded 200 of 200 bytes (100%) AD_SAS.txt\n","ID                                file-Gq1zj58JYBPbb4F2QJkP4BJz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_SAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:14:45 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:14:46 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 104 of 104 bytes (100%) AD_SAS.txt\n","ID                                file-Gq1zj80JYBPQJG2225P66yqF\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_SAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:14:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 24 of 24 bytes (100%) AD_SAS.txt\n","ID                                file-Gq1zj98JYBPvK1VB7ZZ36xy3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_SAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:01 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:02 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 24 of 24 bytes (100%) AD_SAS.txt\n","ID                                file-Gq1zj9jJYBPYY1B11py26bkf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_SAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:03 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:04 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 969 of 969 bytes (100%) /opt/notebooks/RD_AAC.rawaw\n","[===========================================================>] Uploaded 64 of 64 bytes (100%) RD_AAC.txt\n","ID                                file-Gq1zjBQJYBPQJG2225P66yqg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:06 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:06 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 128 of 128 bytes (100%) RD_AAC.txt\n","ID                                file-Gq1zjBjJYBPYY1B11py26bkx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 64 of 64 bytes (100%) RD_AAC.txt\n","ID                                file-Gq1zjF8JYBPbb4F2QJkP4BKx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:09 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:09 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 72 of 72 bytes (100%) RD_AAC.txt\n","ID                                file-Gq1zjFQJYBPYY1B11py26bp3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_AAC.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:11 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 2,289 of 2,289 bytes (100%) /opt/notebooks/RD_AFR.rawaw\n","[===========================================================>] Uploaded 104 of 104 bytes (100%) RD_AFR.txt\n","ID                                file-Gq1zjG8JYBPbb4F2QJkP4BP1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:14 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 296 of 296 bytes (100%) RD_AFR.txt\n","ID                                file-Gq1zjGjJYBPxx1p208XJqkfz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:15 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 192 of 192 bytes (100%) RD_AFR.txt\n","ID                                file-Gq1zjJ0JYBPkyVQ6kkf43bgv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:17 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 200 of 200 bytes (100%) RD_AFR.txt\n","ID                                file-Gq1zjJQJYBPQKKGYPXPJzZz5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_AFR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:18 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:19 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 1,596 of 1,596 bytes (100%) /opt/notebooks/RD_AJ.rawaw\n","[===========================================================>] Uploaded 88 of 88 bytes (100%) RD_AJ.txt\n","ID                                file-Gq1zjK8JYBPQJG2225P66yv5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:21 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 144 of 144 bytes (100%) RD_AJ.txt\n","ID                                file-Gq1zjKQJYBPkyVQ6kkf43bj0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:22 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:54 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) RD_AJ.txt\n","ID                                file-Gq1zjfQJYBPYY1B11py26bpk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) RD_AJ.txt\n","ID                                file-Gq1zjg0JYBPzP0XpV6QGxb0k\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_AJ.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:15:57 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 276 of 276 bytes (100%) /opt/notebooks/RD_AMR.rawaw\n","[===========================================================>] Uploaded 16 of 16 bytes (100%) RD_AMR.txt\n","ID                                file-Gq1zjgjJYBPxx1p208XJqkgp\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_AMR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:15:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 1,068 of 1,068 bytes (100%) /opt/notebooks/RD_CAH.rawaw\n","[===========================================================>] Uploaded 96 of 96 bytes (100%) RD_CAH.txt\n","ID                                file-Gq1zjjQJYBPkyVQ6kkf43bk9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:02 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:03 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 48 of 48 bytes (100%) RD_CAH.txt\n","ID                                file-Gq1zjjjJYBPQJG2225P66yvq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:03 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:04 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 56 of 56 bytes (100%) RD_CAH.txt\n","ID                                file-Gq1zjk8JYBPYY1B11py26bq5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:05 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:06 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 56 of 56 bytes (100%) RD_CAH.txt\n","ID                                file-Gq1zjkjJYBPkyVQ6kkf43bkJ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_CAH.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:07 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 936 of 936 bytes (100%) /opt/notebooks/RD_CAS.rawaw\n","[===========================================================>] Uploaded 56 of 56 bytes (100%) RD_CAS.txt\n","ID                                file-Gq1zjp8JYBPzP0XpV6QGxb1F\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:09 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:10 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 80 of 80 bytes (100%) RD_CAS.txt\n","ID                                file-Gq1zjpjJYBPbb4F2QJkP4BQ3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:11 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:12 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 48 of 48 bytes (100%) RD_CAS.txt\n","ID                                file-Gq1zjq8JYBPvK1VB7ZZ36xzY\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 48 of 48 bytes (100%) RD_CAS.txt\n","ID                                file-Gq1zjqQJYBPYY1B11py26bqZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_CAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:14 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 507 of 507 bytes (100%) /opt/notebooks/RD_EAS.rawaw\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) RD_EAS.txt\n","ID                                file-Gq1zjv8JYBPQJG2225P66yx2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_EAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:17 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) RD_EAS.txt\n","ID                                file-Gq1zjvjJYBPbb4F2QJkP4BQX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_EAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:19 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:19 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 32 of 32 bytes (100%) RD_EAS.txt\n","ID                                file-Gq1zjx0JYBPvK1VB7ZZ36xzq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_EAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 32 of 32 bytes (100%) RD_EAS.txt\n","ID                                file-Gq1zjxQJYBPvK1VB7ZZ36xzx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_EAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:22 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:23 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 166,101 of 166,101 bytes (100%) /opt/notebooks/RD_EUR.rawaw\n","[===========================================================>] Uploaded 16,496 of 16,496 bytes (100%) RD_EUR.txt\n","ID                                file-Gq1zjy8JYBPkyVQ6kkf43bkq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:25 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 10,984 of 10,984 bytes (100%) RD_EUR.txt\n","ID                                file-Gq1zjyQJYBPQJG2225P66yxG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,968 of 1,968 bytes (100%) RD_EUR.txt\n","ID                                file-Gq1zjz0JYBPQKKGYPXPJzb0Y\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:28 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:29 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,976 of 1,976 bytes (100%) RD_EUR.txt\n","ID                                file-Gq1zjzQJYBPYY1B11py26bv2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_EUR.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:30 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 210 of 210 bytes (100%) /opt/notebooks/RD_FIN.rawaw\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) RD_FIN.txt\n","ID                                file-Gq1zk00JYBPYY1B11py26bv8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_FIN.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:32 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:33 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 606 of 606 bytes (100%) /opt/notebooks/RD_MDE.rawaw\n","[===========================================================>] Uploaded 8 of 8 bytes (100%) RD_MDE.txt\n","ID                                file-Gq1zk0jJYBPkyVQ6kkf43bqJ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_MDE.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:35 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:36 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 64 of 64 bytes (100%) RD_MDE.txt\n","ID                                file-Gq1zk10JYBPQJG2225P66yxg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_MDE.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:36 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:37 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","\u001b[2K[===========================================================>] Completed 2,487 of 2,487 bytes (100%) /opt/notebooks/RD_SAS.rawaw\n","[===========================================================>] Uploaded 160 of 160 bytes (100%) RD_SAS.txt\n","ID                                file-Gq1zk1jJYBPxx1p208XJqkjf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_SAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:39 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:40 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 160 of 160 bytes (100%) RD_SAS.txt\n","ID                                file-Gq1zk28JYBPbb4F2QJkP4BV4\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_SAS.txt\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Tue Aug 20 02:16:41 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq1v4KjJYBPx11zB9b3F0BF3\n","Last modified                     Tue Aug 20 02:16:42 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n"]}],"source":["for cohort in [\"AD\",\"RD\",\"Control\"]:\n","    for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","        ! dx download wgs_analysis/results/12_protective_variants/{cohort}_{ancestry}.raw --overwrite\n","        if os.path.exists(f\"{cohort}_{ancestry}.raw\"):\n","            df = pd.read_csv(f\"{cohort}_{ancestry}.raw\", sep=\"\\t\")\n","            df.drop([\"FID\",\"PAT\",\"MAT\",\"SEX\",\"PHENOTYPE\"], axis=1, inplace=True)\n","            df.columns = df.columns.str.split('_').str[0]\n","            for var_id in df.columns[1:]:\n","                participant_ids = list(df[\"IID\"][df[var_id] < 2])\n","                if len(participant_ids) > 0:\n","                    with open(f\"{cohort}_{ancestry}.txt\", \"w\") as file:\n","                        for item in participant_ids:\n","                            file.write(f\"{item}\\n\")\n","                    ! dx upload {cohort}_{ancestry}.txt --path wgs_analysis/results/12_protective_variants/{var_id.replace(\":\",\"_\")}/{cohort}_{ancestry}.txt\n"]},{"cell_type":"markdown","id":"61fede82-8e3c-4766-9ae0-a734eb0e576c","metadata":{},"source":["## APOE genotyping for each variant"]},{"cell_type":"code","execution_count":9,"id":"36582ee0-99b0-4acc-b538-efdb1fa4c73b","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zbG8JYBPq7J1ygzFB6vpv\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2Pp98JYBPbkJb00PzFjBpp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zbfjJYBPq7J1ygzFB6vq6\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2Pp9QJYBPQBz39GXK5jPKz\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zf1QJYBPX0q2K61g175qJ\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2Pp9jJYBPQBz39GXK5jPP1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfKjJYBPbb4F2QJkP4B77\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpB0JYBPQBz39GXK5jPP3\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfQQJYBPQKKGYPXPJzZkz\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpB8JYBPbkJb00PzFjBq2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfxjJYBPz5ZKbFJ888Y20\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpBQJYBPq12Z4BzpFk4g5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zgK0JYBPQ7b3k77P9Q2x0\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpBQJYBPbkJb00PzFjBq5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_MDE.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zj4QJYBPkyVQ6kkf43bg2\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpBjJYBPq12Z4BzpFk4g7\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zj80JYBPQJG2225P66yqF\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpF0JYBPfFgXqPKf492JF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zkQ8JYBPzP0XpV6QGxb3B\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpF8JYBPv715V7jbBQzVQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zky0JYBPYY1B11py26bz3\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpFQJYBPbkJb00PzFjBqF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zp80JYBPvK1VB7ZZ36y2Y\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpFjJYBPxKBqQvQ89y13V\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AMR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zpx0JYBPzP0XpV6QGxb8G\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpG0JYBPkqX37y556ggK7\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zq88JYBPxx1p208XJqkqp\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpG8JYBPkqX37y556ggK9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zqB0JYBPbb4F2QJkP4Bf1\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpGQJYBPYFf7xPX00fk72\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zqj0JYBPX6f98Qxf7K1b9\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpGjJYBPq7J1ygzFBGgB0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zqyjJYBPQKKGYPXPJzb7j\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpJ0JYBPq7J1ygzFBGgB4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_FIN.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvGjJYBPQJG2225P66z5q\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpJ8JYBPz5ZKbFJ88Jbv0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_MDE.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvKjJYBPkyVQ6kkf43fBb\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpJ8JYBPq7J1ygzFBGgB6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvQQJYBPxx1p208XJqp22\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpJQJYBPgG398X3y9bPXp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjBjJYBPYY1B11py26bkx\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpJjJYBPz5ZKbFJ88Jbv6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjGjJYBPxx1p208XJqkfz\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpK0JYBPq7J1ygzFBGgBJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjKQJYBPkyVQ6kkf43bj0\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpK8JYBPkqX37y556ggKp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AMR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjgjJYBPxx1p208XJqkgp\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpKQJYBPbz119b8G455Zk\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjjjJYBPQJG2225P66yvq\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpKjJYBPxKBqQvQ89y1Bq\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjpjJYBPbb4F2QJkP4BQ3\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpP0JYBPfFgXqPKf492JV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjvjJYBPbb4F2QJkP4BQX\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpP8JYBPz5ZKbFJ88JbvG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjyQJYBPQJG2225P66yxG\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpPQJYBPv69029zV6VX7V\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_MDE.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zk10JYBPQJG2225P66yxg\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpPjJYBPgG398X3y9bPY2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zk28JYBPbb4F2QJkP4BV4\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2PpQ0JYBPz5ZKbFJ88JbvQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvBQJYBPxx1p208XJqp1B\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44908756_\n","  C_A\n","\n","Job ID: job-Gq2PpQ0JYBPX0q2K61g1PBjV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvF0JYBPxx1p208XJqp1K\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_25897620_\n","  C_T\n","\n","Job ID: job-Gq2PpQ8JYBPz5ZKbFJ88JbvZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zbJ0JYBPQ7b3k77P9Q2ZQ\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpQQJYBPkjKf7jXj4FzFK\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zbg8JYBPQ7b3k77P9Q2b0\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpQjJYBPz5ZKbFJ88Jbvf\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfG8JYBPz5ZKbFJ888Y0v\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpV0JYBPq7J1ygzFBGgBz\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfP0JYBPbb4F2QJkP4B7F\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpV8JYBPQ7b3k77P9g4k5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfV0JYBPYFf7xPX00QGYx\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpVQJYBPq7J1ygzFBGgF1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfy0JYBPYFf7xPX00QGZg\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpVjJYBPQ7b3k77P9g4p0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zgPjJYBPv69029zV67Qxv\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpX0JYBPQ7b3k77P9g4p4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zj98JYBPvK1VB7ZZ36xy3\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpX8JYBPv69029zV6VX7z\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zkQjJYBPbb4F2QJkP4BVq\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpXQJYBPq7J1ygzFBGgF5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zp5jJYBPQKKGYPXPJzb3q\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpXjJYBPX0q2K61g1PBjx\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zp8QJYBPkyVQ6kkf43f4F\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpY0JYBPz5ZKbFJ88Jbx9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AMR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zq4jJYBPbb4F2QJkP4BZx\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpY8JYBPgG398X3y9bPYf\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zq8QJYBPkyVQ6kkf43f7g\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpYQJYBPX0q2K61g1PBk1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zqX8JYBPbb4F2QJkP4Bf5\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpYjJYBPgG398X3y9bPYp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zqjQJYBPfjkjXxbBKvZp3\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpZ0JYBPX0q2K61g1PBk3\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvF8JYBPkyVQ6kkf43fB6\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpZ8JYBPv69029zV6VX8B\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_FIN.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvJ8JYBPbb4F2QJkP4Bgg\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpZ8JYBPYFf7xPX00fk8y\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_MDE.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvP8JYBPkyVQ6kkf43fBg\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpkjJYBPq12Z4BzpFk4jB\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvV0JYBPQJG2225P66z64\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2Ppp8JYBPkqX37y556ggQ9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjF8JYBPbb4F2QJkP4BKx\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PppQJYBPq12Z4BzpFk4jQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjJ0JYBPkyVQ6kkf43bgv\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PppjJYBPv715V7jbBQzYq\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjfQJYBPYY1B11py26bpk\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PppjJYBPbkJb00PzFjBx9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjk8JYBPYY1B11py26bq5\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2Ppq0JYBPfFgXqPKf492Kj\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjq8JYBPvK1VB7ZZ36xzY\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2Ppq8JYBPbkJb00PzFjBxJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjx0JYBPvK1VB7ZZ36xzq\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpqQJYBPq12Z4BzpFk4jj\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjz0JYBPQKKGYPXPJzb0Y\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2PpqjJYBPQBz39GXK5jPVb\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zk2jJYBPQKKGYPXPJzb13\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2Ppv0JYBPQBz39GXK5jPVg\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zbJQJYBPYFf7xPX00QGX3\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppv8JYBPxKBqQvQ89y1G8\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zbgQJYBPX0q2K61g175pZ\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpvQJYBPq12Z4BzpFk4k1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfJ8JYBPq7J1ygzFB6vvZ\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpvQJYBPkqX37y556ggV9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfPQJYBPYY1B11py26bXG\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpvjJYBPbz119b8G455fY\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zfVQJYBPq7J1ygzFB6vxB\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppx0JYBPq12Z4BzpFk4k3\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zg4jJYBPv69029zV67Qx3\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppx8JYBPxKBqQvQ89y1GQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zgQ8JYBPv69029zV67Qy0\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpxQJYBPbkJb00PzFjBxg\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep AD_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zj9jJYBPYY1B11py26bkf\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpxjJYBPkqX37y556ggVV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zkvjJYBPbb4F2QJkP4BXP\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppy0JYBPv715V7jbBQzZ3\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zp60JYBPQKKGYPXPJzb3x\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppy8JYBPv715V7jbBQzZ6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zpV8JYBPvK1VB7ZZ36y3p\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppy8JYBPq12Z4BzpFk4kG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_AMR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zq58JYBPvK1VB7ZZ36y5J\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpyQJYBPfFgXqPKf492PF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zq90JYBPvK1VB7ZZ36y64\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpyjJYBPv715V7jbBQzZG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zqXQJYBPkyVQ6kkf43f8F\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppz0JYBPfFgXqPKf492Q2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zqjjJYBPZ1ykBk7F2gKXv\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Ppz8JYBPfFgXqPKf492Q4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvFjJYBPQJG2225P66z5g\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpzQJYBPQBz39GXK5jPX2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_FIN.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvJjJYBPbb4F2QJkP4Bgk\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpzQJYBPfFgXqPKf492Q6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_MDE.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvPQJYBPkyVQ6kkf43fBk\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2PpzjJYBPv715V7jbBQzZZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep Control_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zvVQJYBPQKKGYPXPJzbvF\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq00JYBPq12Z4BzpFk4p0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AAC.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjFQJYBPYY1B11py26bp3\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq08JYBPkqX37y556ggVk\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AFR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjJQJYBPQKKGYPXPJzZz5\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq0QJYBPfFgXqPKf492QB\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_AJ.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjg0JYBPzP0XpV6QGxb0k\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq0jJYBPfFgXqPKf492QG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_CAH.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjkjJYBPkyVQ6kkf43bkJ\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq10JYBPbkJb00PzFjBy0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_CAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjqQJYBPYY1B11py26bqZ\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq10JYBPbz119b8G455gV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_EAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjxQJYBPvK1VB7ZZ36xzx\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq18JYBPxKBqQvQ89y1GZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_EUR.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zjzQJYBPYY1B11py26bv2\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq1QJYBPbkJb00PzFjBy2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep RD_SAS.txt --set-all-var-ids 'chr@:#:$r:$a' --new-id-max-allele-len 999 --make-bed --out RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gp9X55QJ8z7kk4qBFkKZv54p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq1zkGjJYBPkyVQ6kkf43bv5\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2Pq1jJYBPv715V7jbBQzb8\n"]}],"source":["%%bash\n","\n","#for var_id in {\"chr19_44892887_C_T\",\"chr19_44905307_A_T\",\"chr19_44908756_C_A\",\"chr21_25897620_C_T\",\"chr21_26171645_A_G\",\"chr21_26171723_T_C\"};\n","for var_id in {\"chr19_44905307_A_T\",\"chr19_44908756_C_A\",\"chr21_25897620_C_T\",\"chr21_26171645_A_G\",\"chr21_26171723_T_C\"};\n","do\n","    files=($(ls ../../mnt/project/wgs_analysis/results/12_protective_variants/${var_id}/*.txt 2>/dev/null))\n","    \n","    for file in \"${files[@]}\";\n","    do\n","        filename=$(basename \"$file\" .txt)\n","        \n","        dx run swiss-army-knife \\\n","            -iin=\"/wgs_analysis/results/4_normalized/normalized.vcf\" \\\n","            -iin=\"/wgs_analysis/results/12_protective_variants/${var_id}/${filename}.txt\" \\\n","            -icmd=\"plink2 --vcf normalized.vcf --chr 19 --from-bp 44905791 --to-bp 44909393 --keep ${filename}.txt --set-all-var-ids 'chr@:#:\\$r:\\$a' --new-id-max-allele-len 999 --make-bed --out ${filename}\" \\\n","            --instance-type mem1_hdd1_v2_x16 \\\n","            --destination \"${projectid}:/wgs_analysis/results/12_protective_variants/${var_id}\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":null,"id":"ace204bf-1d49-4363-852a-d45e1003a72a","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[]},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":7,"id":"e1598296-13b8-4fc3-9d61-1220a97f13e5","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_SAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2K608JpbqXZkqZBz0vxBYb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2K608JpbqxyPxGQyvQk75K\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2K608JpbqxKBqQvQ8kxk1Z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3P0JYBPp8Vj8gzy55F0Z\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx00Jk8ZxPjY9gBpXBX9y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx00Jk8ZqKKbXp14zK9Pg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx00Jk8ZfX032b7qk2Yv0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3P8JYBPq06KvB1Y3y42f\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GybQJfPkpGfgGxy4ZXvj2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GyXjJfPkz5jX3ZJjj4f8V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GyXjJfPkvpYgkXZ6pbGp5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3PQJYBPgY8yBj32KYxp3\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxK0JF39PkZb7PvpYPpXV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxK0JF3930pB041BV9JJk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxV0JF395kJb00PzqgVQ9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3PjJYBPQB71Z0898zgPF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J230Jv62856B2b4vj4Gb6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J230Jv62FKKbXp14zKFbg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J230Jv62Ggj78gY1YVFPJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3PjJYBPVv59GyF51vJ1Q\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gk5jJ6B9GK1VB7ZZY9Kj8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gk5jJ6B90KKGYPXPy29K0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gk5QJ6B9GpYgkXZ6pbBvX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3Q0JYBPQqZp25bbGGZ6P\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gp0jJvxVVXx007FvZ3238\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gp58JvxVxyPxGQyvQjjkf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gp38JvxVby0B5gvky65zx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3Q8JYBPp8Vj8gzy55F0j\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_AD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gp2jJkGX5990qB88g8z2p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gp2jJkGXPFGfVXj9f1Kj3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gp2jJkGX0kJx5pyyYXy8F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3QQJYBPjq6275PV5FPXJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GyvjJV4gbQz1FQfKj16vQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GyvjJV4gyyBvb5Z5bQ022\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GyvjJV4gQz25PyBXXfyvy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3QjJYBPYyQ74Yf3Fq1xG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx8QJ665K2x28QPqy6yx3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx8QJ665P5ZKbFJ8jGy9y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx8QJ665BGfgGxy4ZXq7b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3V0JYBPgY8yBj32KYxp5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J0q0JYJv0p2gZvzPgg86b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J0q0JYJv5y0B5gvky68Zy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J0q0JYJvP5jX3ZJjj4g82\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3V8JYBPjq6275PV5FPXY\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqP0Jx0F30pB041BV9Gpg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqP0Jx0F3Ff7xPX0Qf2pk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqP0Jx0F856B2b4vj498f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3V8JYBPp8Vj8gzy55F0y\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AMR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GzXQJVF9zY7gjvfFV9X6q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GzXQJVF9qKKbXp14zKB6z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GzXQJVF9b02Y7F5Vgy2FV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3VQJYBPq06KvB1Y3y432\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gq98J33xGK1VB7ZZY9PBJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gq98J33x4yGJ5Zp8jj2JX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gq98J33x8x783QXkYPFVx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3VjJYBPQB71Z0898zgPb\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J36QJ4Gy6g7pBk1ZqXK4f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J36QJ4GyFQqkP42zZq7Py\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J36QJ4GyJ8Pfq3Q3pj9BP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3X0JYBPgY8yBj32KYxpF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GybjJJxqG715V7jbpQGYp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GybjJJxq9YGG02fpQ7Kvk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GybjJJxqJyPxGQyvQjpbx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3X8JYBPjq6275PV5FPXb\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J320JgkBY5Pg2x8bZ1F4g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J320JgkBVQY8Fkg7p0xP0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J320JgkBp8b9KpY2jPzfG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3XQJYBPYyQ74Yf3Fq1xQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_FIN --extract apoe_variants.txt --make-bed --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J8v8JpPGfX032b7qk2jyG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J8v8JpPGX8fGZ2vVzzxX4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J8v8JpPGv8YpBV8FX805y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3XjJYBPgY8yBj32KYxpJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J820J4Bv5zf5kzgYyjVVk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J820J4Bv94GzVY4Ygj7ZY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J820J4Bv3P0GpvykxjF9q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3Y0JYBPYyQ74Yf3Fq1xZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gy78J4kxJYkv1B4gXz43X\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gy70J4kxBZ2qPG9vQ6QQX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gy70J4kxFvby033qg7J3F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3Y0JYBPQqZp25bbGGZ6q\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxyQJjXj1QY8Fkg7p0qKb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxqQJjXjGK1VB7ZZY9Q85\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxqQJjXj856B2b4vj4B56\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3Y8JYBPQqZp25bbGGZ6x\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GvjQJ51vKfqfg8QvbBbQx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GvjQJ51vKyBvb5Z5bPzFq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GvjQJ51v6X032b7qk2Yp6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3YQJYBPQqZp25bbGGZ6z\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxgQJv8F6xqfz0f2z6yvx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxgQJv8F856B2b4vj4B32\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxgQJv8FBGfgGxy4ZXqXf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3YjJYBPQB71Z0898zgPx\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx0jJ7gxfX032b7qk2Yv4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx0jJ7gxxVy5vFz8k07V0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gx0jJ7gxyQ3xVK3YvKV00\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3Z0JYBPp8Vj8gzy55F1B\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqzjJqFBG715V7jbpQBjJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqzjJqFBJVy5vFz8k0776\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqzjJqFB3Ff7xPX0Qf2z9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3Z8JYBPYZ982v2vBV2P2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxfQJVK5Qp2gZvzPgg6Xp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxfQJVK5YKgffPFvyxp4b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GxfQJVK5p5vbVY7FV5BzZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3ZQJYBPp8Vj8gzy55F1G\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J0g0J2P63pf88k2jX1Jj8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J0g0J2P641ykBk7FXxq64\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2J0g0J2P60XVfQ8GgxVzx2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3ZQJYBPjq6275PV5FPXx\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_FIN --extract apoe_variants.txt --make-bed --out apoe_snps_RD_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gv48Jj60xx1p208Xxxp2k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gv48Jj60ZbVqfq0fg004X\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Gv48Jj60pzGzV4FJVB4F5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3ZjJYBPYyQ74Yf3Fq1xp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_RD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GpgQJJjkGpYgkXZ6pbFJX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GpgQJJjk3PPff05GZF5KQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GpgQJJjk3Y1B11pyXB483\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3b8JYBPYyQ74Yf3Fq1xv\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqB8J76kJgJ0Q4F2bj955\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqB8J76kGK1VB7ZZY9PBY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2GqBjJ76k3pf88k2jX1FXP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2z3bQJYBPq06KvB1Y3y444\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJpjJ69xp2Q641z0Q2vB4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJpjJ69xjjKj5Pj1gXq7J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJpjJ69xfP0264BgQ5Y6q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3bjJYBPjq6275PV5FPY4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZb8JjvGBvJk8k19yK1bk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZb8JjvGKvbJkVK8gV764\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZb8JjvG4QjFQX8PZpgpg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3bjJYBPgY8yBj32KYxpz\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJVQJY5z7gz68F4gyVpYG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJKjJY5zG7PPVb8GjX9JY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJKjJY5zFjz4z629QF4g6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3f0JYBPgY8yBj32KYxq1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKFQJY7FfVp7b89QvzYk7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKFQJY7FgFP5V8gyvYf77\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKK0JY7FyvbJkVK8gV6Jj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3f8JYBPQB71Z0898zgQ5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXp0JBVgv2Z68p7kpJg85\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXp0JBVgggz68F4gyVqJf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXp0JBVgvZ518xbZX5kYb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3fQJYBPp8Vj8gzy55F1b\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKY8J76bpyV38637pBkjy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKY8J76bjgpbb8y1qzpxg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKY8J76bYGp7GG9Pb3B7x\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3fjJYBPQB71Z0898zgQ7\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGf0J95g09X6XJJJX1F5p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGf0J95gJZ2P2zV7YvKP9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGf0J95gPv4k1f1pY6vV9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3g0JYBPYyQ74Yf3Fq1y8\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AAC.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_AD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXXjJkzz9QgBbZjbyJK3X\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXXjJkzz2b7G635ZjF1pk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXXjJkzz1pX90JQkxGV20\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3g8JYBPYyQ74Yf3Fq1yB\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AFR.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Vg30JZGV3kqZqgG1z6Vy9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Vg30JZGV98xZ7G59yG4X5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Vg30JZGV342XyZF6pp7gJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3gQJYBPgY4k75Q84k8Bv\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AJ.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VG98JYzq3Z982v2vp66qg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VG98JYzqFJ2vbvj5pQypJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VG98JYzq82XFpb3zZ2jB9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3gjJYBPgY8yBj32KYxq9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK40J6XVGPg373K5zqFVX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK40J6XV98xZ7G59yBvY8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK40J6XV80zFFJqjjb542\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3gjJYBPgY4k75Q84k8By\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AMR.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VjQjJygZKvbJkVK8gV7fG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VjQjJygZGfXb0Gkxb67XB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VjQjJygZ0qZp25bbqqGPf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3j0JYBPp8Vj8gzy55F1q\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_CAH.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AMR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKqQJpB2bPq62qfxQjFk3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKqQJpB2VJFY2f4jx69gJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKqQJpB2kb93K3fJZX9p5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3j8JYBPqJ2vbvj5BqyGG\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_CAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VjfjJz0P0YyBJFf7QQpfG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VjjQJz0P2jQFqbfzb0Ybp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VjfjJz0P7YbkxY88ZqJ8K\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3jQJYBPzpx2fX0688v7k\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_EAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGfQJK24f0x1yQ0BYjGkP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGfQJK24VpX90JQkxGQPK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGfQJK24qqJzZZVYv7bgZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3jjJYBPfpPyJF1bB09bq\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_EUR.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ1QJGpYqk6ZPPJpypPfz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ1QJGpYk614q5k5Xkxq3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ1QJGpYp2Q641z0Q2v8v\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3k0JYBPv2Z68p7kBy2gz\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_MDE.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ6QJ28Zqx5Fy5gfQg958\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ6QJ28Zp2Q641z0Q2v93\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ6QJ28ZVfY2YYYBv7vPP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3k8JYBPfpPyJF1bB09bx\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_SAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_FIN --extract apoe_variants.txt --make-bed --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VP00JY4fB2Q641z0Q2vPv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VP20JY4f2Y98Bf9kpPF5z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VP00JY4fJ6k0vvzJy72B3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3kQJYBPgY4k75Q84k8F1\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_EUR.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJpjJQb4q06KvB1YYP23p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJpjJQb4Q0JGvG9bYJV6P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJpjJQb4bVg15VpYzxjpZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3kjJYBPkqz9367f8xq48\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_EUR.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXx0JYB6ggz68F4gyVqJq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXx0JYB6YKxfQzZXfvv0b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXx0JYB6k4jXpj9YjzJZ7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3kjJYBPv2Z68p7kBy2j7\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_AAC.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQFjJKQjX79pf2KzkGp7v\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQFjJKQjp8Vj8gzyZZg69\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQFjJKQjVBYZ4Gj8VPQY6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3p0JYBPkqz9367f8xq4B\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_AFR.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ00J2F5v5vpXXfxxvYFZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ00J2F5v5vpXXfxxvYFX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGzQJ2F5Z1f0VBZpgG241\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3p8JYBPqJ2vbvj5BqyGQ\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_AJ.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKPjJ04yX7kVjq5Kykj57\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKX8J04ypvJk8k19yK0QJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKPjJ04yzJ0b43Z9kYk4V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3pQJYBPzpx2fX0688v83\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_CAH.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AMR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKVjJ7z6qG0JVbz0Z7152\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKVjJ7z6jjKj5Pj1gXq96\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKVjJ7z6pFPj9f05zvFQ7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3pjJYBPVv59GyF51vJ29\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_CAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXjjJXKZ7gz68F4gyVqJV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXjjJXKZ9yjqx4VGZkzqZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXjjJXKZFQ80vyVYbxK22\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3q0JYBPVv59GyF51vJ2K\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV28J6610J20bxyZg1vb0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV28J6612vGZpPykZPgb0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV28J66139jV3VVFz7kgJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3q8JYBPxGKKKYZ43p4x0\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_EAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGg8J91JkYK3k1Y7qpG9q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGg8J91JY42XyZF6pkY4V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGg8J91Jg2Y7YQfBqvf5B\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3qQJYBPq06KvB1Y3y47f\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKg0JY12FBv7bGBGgk9QB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKg0JY122jQFqbfzb0XBP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKg0JY121BYZ4Gj8VPQ99\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3qjJYBPYyQ74Yf3Fq1z4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_RD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZpQJJgPj75gb4f3YYBqj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZpQJJgPf0x1yQ0BYjJxz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZp8JJgPqjGQ794yp546F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3v0JYBPVv59GyF51vJ2Z\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZ6jJZ7JjYZPgg23g8FBy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZ6jJZ7Jjq6275PVZxfQF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZ6jJZ7Jk6P6YFQzv5KjB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2z3v8JYBPjq6275PV5FPbV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPVjJG0K000VQ2QJf7F3g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPVjJG0KGZqf0P4ByGyf7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPVjJG0K1p3BJz3byjbZJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44908756_\n","  C_A\n","\n","Job ID: job-Gq2z3v8JYBPjq6275PV5FPbY\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKBjJbY45Zfz4zvFz87vF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKBjJbY42vGZpPykZPgKp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKBjJbY4GB3QZqfxpg04P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_25897620_\n","  C_T\n","\n","Job ID: job-Gq2z3vQJYBPVv59GyF51vJ2p\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV5QJK9b2f024083q7pQ1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV5QJK9bGqkgzzG9v5zbV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV5QJK9b8ykp61qykg6ZZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3vjJYBPYyQ74Yf3Fq1zQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKg0Jk0jY1Z34B54Qq3yQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKg0Jk0jpY6Px7g6yK23z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKg0Jk0jk6P6YFQzv5K13\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3x0JYBPVv59GyF51vJ30\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJG0J9047jZj95jGZbJB7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJG0J9049yjqx4VGZkz1F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJG0J9040B71Z089jPB76\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3x8JYBPQB71Z0898zgV5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQKQJ3qX2pGg0V52XVyKB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQBjJ3qXJGKKKYZ4Y7v0p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQBjJ3qXGZqf0P4ByGyjV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3xQJYBPYyQ74Yf3Fq1zX\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPf8Jqy2pvJk8k19yK0Xz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPf8Jqy2xGKKKYZ4Y7qz9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPf8Jqy2v4FK0kqjppzkG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3xjJYBPjq6275PV5FPbk\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbvjJbxXgZ5k7QkkjYZy9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbvjJbxXXf024083q7q4p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbvjJbxXjp8Z6qvZQKgbJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3y0JYBPq06KvB1Y3y49P\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGvjJ3bQZ36Z3VJppk52v\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGvjJ3bQXvGZpPykZPgFX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGvjJ3bQX6JX5yk3QVBkP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3y8JYBPVv59GyF51vJ32\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VfkQJ1PjYKxfQzZXfxQ7G\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VfkQJ1PjYFBq8B3xX08j5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VfkQJ1PjpY6Px7g6yK36Y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3y8JYBPp8Vj8gzy55yYZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGjjJFp8pFPj9f05zvFF6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGjjJFp8XY98Bf9kpPByf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGjjJFp8k6P6YFQzv5Jkb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3yQJYBPVv59GyF51vJ36\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKxjJY758q6275PVZx11x\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKxjJY757FyJK5zZvgZ9y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKxjJY751fY2YYYBv7vYX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3yjJYBPQB71Z0898zgVJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKXQJZkggJBGp39FQBpg9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKXQJZkgj2XFpb3zZ2jYx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKXQJZkgg2Y7YQfBqx9vK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3z0JYBPp8Vj8gzy55yYp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AMR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY1jJBQBfz7QQ3pVb57Q4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY1jJBQBk5vf21VgXYvxq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY1jJBQBVpX90JQkxGV3p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3z8JYBPYyQ74Yf3Fq204\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_EUR.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYj0J83PQZ5kqBjvYVBpf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYj0J83PvqkgzzG9v609F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYj0J83PbPq62qfxQjGG8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z3zQJYBPkqz9367f8xq61\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_SAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AAC.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AFR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AJ.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AMR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_CAH.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_CAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_EAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_EUR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_FIN.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_MDE.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_SAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AAC.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AFR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AJ.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ80Jfvzv5vpXXfxxvZ3k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ80Jfvzbxbx6kqYV3x27\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ80JfvzpyV38637pBkzV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z430JYBPgY4k75Q84k8F6\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_CAH.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJyjJzyK39jV3VVFz7k6B\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJyjJzyK3yQ74Yf3q9yfJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJyjJzyK7FyJK5zZvgZ7k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z438JYBPxGKKKYZ43p4xB\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VFz8JV336pPyJF1bkP551\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VFz8JV336Vp7b89QvzXzJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VFz8JV33FVg8qK36q6fXX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z43QJYBPQqZp25bbGGZ92\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_FIN --extract apoe_variants.txt --make-bed --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPv8J1g524gZVpbbf1BxJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPv8J1g5FjGQ794yp5340\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPv8J1g52yFQVqbyxXbp4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z43jJYBPVv59GyF51vJ46\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_CAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VpJ8JXq6VK4YKgB3k4bbp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VpJ8JXq6xGKKKYZ4Y7xVJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VpJ8JXq6YvV86yz4zPZyy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z440JYBPp8Vj8gzy55yZq\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_EAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbfQJy3QG2Z68p7kpJgf8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbfQJy3QF0pKv7Y3ZjZV5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbfQJy3QPbjFQ20Bp7ZqJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z440JYBPYyQ74Yf3Fq218\n"]},{"name":"stderr","output_type":"stream","text":["dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_EUR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_SAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_AAC.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_AFR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_AJ.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_CAH.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_CAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_EAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_EUR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/AD_SAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AAC.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AFR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AJ.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_AMR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_CAH.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_CAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_EAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_EUR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_FIN.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_MDE.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/Control_SAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AAC.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AFR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_AJ.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_CAH.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_CAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_EAS.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_EUR.bim\" to a name or ID\n","dxpy.exceptions.DXCLIError: Value provided for input field \"in\" could not be parsed as file: could not resolve \"/wgs_analysis/results/12_protective_variants/RD_SAS.bim\" to a name or ID\n"]},{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPyQJ3Y21BYZ4Gj8VPQQ3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPyQJ3Y2B63Pxq87jvJjp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPyQJ3Y2PbjFQ20Bp7Z2x\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4F8JYBPxGKKKYZ43p53K\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGbQJvGVGVGz5KVGVVVjx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGbQJvGVF0pKv7Y3ZjYZ0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VGbQJvGVG93Gj7gGqxzg6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4FQJYBPfpPyJF1bB0BKp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKyjJ6g7X7kVjq5Kykj6Y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKyjJ6g7jbBZJ7p3gbGQF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKyjJ6g7VpX90JQkxGQfk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4FjJYBPqJ2vbvj5BqyGy\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV18JPJxkQgBbZjbyJJxk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV18JPJxqVg8qK36q6g05\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VV18JPJxYkqZqgG1z6VBp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4G0JYBPqJ2vbvj5BqyJ0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJQ8JK30436V9XYXfV6Vj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJQ8JK30B9QpfZ7yfk0Fk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJQ8JK303yQ74Yf3q9ybf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4G8JYBPv2Z68p7kBy2jq\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Vq20JB017gz68F4gyVvYv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Vq20JB016fZpFQy6y7Zq8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2Vq30JB016pPyJF1bkP704\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4GQJYBPv2Z68p7kBy2jx\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZ88JzpzzV2Pgf4qfQF4Z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZ88JzpzfpbbpQyqyJk4q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZ88JzpzbQG3QbP4qxYb5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4GjJYBPkqz9367f8xq6b\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VVyQJ28F27kf98X4kKvp0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VVyQJ28F0Jzp1zg9p62JJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VVyQJ28F2vGZpPykZPgfk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2z4GjJYBPkqz9367f8xq6g\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJJ0JJJVXf024083q7p2V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJJ0JJJVvZ518xbZX5k1k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJJ0JJJVffZpFQy6y7Y3P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4J0JYBPfpPyJF1bB0BP0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZQ8J7xg0Z5kqBjvYVBvy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZJjJ7xg41f0VBZpgG2z1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZJjJ7xg0J20bxyZg1x1B\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4J8JYBPqJ2vbvj5BqyJ6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKb0JQGFzJ0b43Z9kYk4g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKXQJQGFk5vf21VgXYvGg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKXQJQGFVBYZ4Gj8VPQ95\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4JQJYBPgYbkxY885BYK2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK4QJ67fV4fvZyJGZj7gj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK4QJ67fXY98Bf9kpPF2q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK4QJ67fZg783b1qkZ6vQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4JjJYBPzpx2fX0688v8q\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VVX8JGP8YGp7GG9Pb3BK4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VVPjJGP8jbBZJ7p3gbGkx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VVVQJGP8xV7Yp7f1yJbyk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4K8JYBPgY4k75Q84k8Ff\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKB0Jy5jfpbbpQyqyJjGJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKB0Jy5jqVg8qK36q6fgq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKB0Jy5jgY4k75Q8Z564P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4KQJYBPv2Z68p7kBy3V1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPf0JzbYpxG95XQ7q40GY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPf0JzbYkqz9367fjZqjZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPf0JzbYQ0JGvG9bYJVBV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4Z8JYBPgY4k75Q84kqvy\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile AD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ6jJzKJZF7bZPXkb5YJ2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ6jJzKJzv7642Vpp6Yqv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ6jJzKJYyQ74Yf3q9yyJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4ZQJYBPzpx2fX0688v95\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKjQJ6V3j0zFFJqjjb566\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKjQJ6V3ggz68F4gyVpg2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKjQJ6V3YyQ74Yf3q9yjF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4ZjJYBPv2Z68p7kBy3V8\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPJ0JZ5VByV38637pBkqZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPB8JZ5VB7KzJyQ9qBQvx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPB8JZ5V80zFFJqjjb57g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4b0JYBPqJ2vbvj5BqyJG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJqQJp8z5QG3QbP4qxXpV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJqQJp8z8ykp61qykg6B6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJqQJp8zGZqf0P4ByGx0Q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4xQJYBPkqz9367f8xq7G\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_AMR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXfjJ0bzjYZPgg23g8F2y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXfjJ0bzq35Bg42gYv60J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VXfjJ0bzYkqZqgG1z6VYj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4y0JYBPfpPyJF1bB0BPz\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY20Jy13PYF8k9B6yKYfg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY20Jy139XPffj21bjp0Q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY20Jy135Vg15VpYzxkk6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z4y0JYBPv2Z68p7kBy3VX\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZy8JZ931fY2YYYBv7xF6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZy8JZ938jKj5Pj1gXv7z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VZy8JZ93B63Pxq87jvKV7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z570JYBPqJ2vbvj5BqyKZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK20JQGxF48KJ8PYvVkqF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK20JQGx7gz68F4gyVpZ6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VK20JQGx7FP5V8gyvYf6k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z578JYBPfpPyJF1bB0BQ6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbV8Jg7qXyFQVqbyxXfbb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbZ0Jg7qvQkKGVvVxgkX1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VbYjJg7qX79pf2KzkGq1k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z57QJYBPqJ2vbvj5BqyKp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_FIN --extract apoe_variants.txt --make-bed --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKvQJFvqGQkKGVvVxgjx1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKvQJFvq7q9V65z3VFz5F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKvQJFvq96P6YFQzv5K19\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z57jJYBPfpPyJF1bB0BQ8\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_MDE --extract apoe_variants.txt --make-bed --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ1QJBq7000VQ2QJf7BpK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ1QJBq71pX90JQkxGQVB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VJ1QJBq77Y4k75Q8Z55y6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z580JYBPqJ2vbvj5BqyP0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile Control_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYy8Jg22fVp7b89QvzZY9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYy8Jg22QG472YY8YQzvg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYy8Jg22Y1Z34B54Qq4yB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z588JYBPxGKKKYZ43p555\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AAC --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKjjJjJpXpGg0V52XVy8q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKjjJjJpzv7642Vpp6Yj1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKjjJjJpq057zf96p03vb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z58QJYBPzpx2fX0688v9v\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AFR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPK8JPPyQB71Z089jPBQQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPK8JPPyq06KvB1YYP289\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPK8JPPyjykp61qykg6Vb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z58jJYBPgYbkxY885BYKj\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_AJ --extract apoe_variants.txt --make-bed --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VkqjJgfkj2XFpb3zZ2y8j\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VkqjJgfkkvfzqY6VfbVyj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VkqjJgfkQZ5kqBjvYVFF3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z590JYBPzpx2fX0688v9z\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAH --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ6QJ1bK3Gp7GG9Pb3BBg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ6QJ1bKGPg373K5zqzGG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VQ6QJ1bK9qz9367fjZqq1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z598JYBPgYbkxY885BYKp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_CAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY5jJXqkGbbbyzpqzqPPz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY5jJXqkJZ2P2zV7YvPQP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VY5jJXqkGVGz5KVGVYJ31\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z59QJYBPfpPyJF1bB0BQB\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKxQJq16vqkgzzG9v5zPQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKxQJq16ZF7bZPXkb5Y9g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VKxQJq16X5ZXGk12qQ69f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z59QJYBPgYbkxY885BYKy\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_EUR --extract apoe_variants.txt --make-bed --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPXQJ96yqjGQ794yp532f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPXQJ96ygp85fYz9Y3F74\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VPXQJ96yfz7QQ3pVb576k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z59jJYBPkqz9367f8xq7x\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile RD_SAS --extract apoe_variants.txt --make-bed --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYG8JPb6zJ0b43Z9kYkp1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYG8JPb6VfY2YYYBv7x7J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2VYG8JPb6ZKyxgJbjgq62k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2z5B0JYBPqJ2vbvj5BqyPB\n"]}],"source":["%%bash\n","\n","for var_id in {\"chr19_44892887_C_T\",\"chr19_44905307_A_T\",\"chr19_44908756_C_A\",\"chr21_25897620_C_T\",\"chr21_26171645_A_G\",\"chr21_26171723_T_C\"};\n","do\n","    files=($(ls ../../mnt/project/wgs_analysis/results/12_protective_variants/${var_id}/*.bim 2>/dev/null))\n","    \n","    for file in \"${files[@]}\";\n","    do\n","        filename=$(basename \"$file\" .bim)\n","        \n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/results/12_protective_variants/${var_id}/${filename}.bim\" \\\n","        -iin=\"/wgs_analysis/results/12_protective_variants/${var_id}/${filename}.bed\" \\\n","        -iin=\"/wgs_analysis/results/12_protective_variants/${var_id}/${filename}.fam\" \\\n","        -iin=\"/wgs_analysis/data/apoe_variants.txt\" \\\n","        -icmd=\"plink --bfile ${filename} --extract apoe_variants.txt --make-bed --out apoe_snps_${filename}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/12_protective_variants/${var_id}\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":null,"id":"7369a34f-5daf-4ba9-bdd1-eb344629c3f3","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":9,"id":"da93f6a4-b298-412d-9331-8685bd2e94f3","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AAC --recode compound-genotypes --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z68QJY6B8qY4VYKxv0Jzx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z68QJY6B8Yxjxz4gZJkxB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z68QJY6BPpx2fX06j8vK2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVkjJYBPk6P6YFQzGP9J1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AFR --recode compound-genotypes --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zjJ0498Yxjxz4gZJkqV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zQJ0498qY4VYKxv0Jz4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5x0J0496pPyJF1bp0BX4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVp0JYBPYZ982v2vBV57k\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AJ --recode compound-genotypes --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5k0J4x4ZYV5FVxXqz9k1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5k0J4x4zpx2fX06j8vF7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5k0J4x4qk6ZPPJpz7Q8p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVp8JYBPXJfFXKqK9B4b9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAH --recode compound-genotypes --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zjJBPP0kpgXbjxxFG3y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zjJBPP60x1yQ0BZ42bJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zjJBPPJ9yYJ9G4YBk2Z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVpQJYBPjYxjxz4g4Jx21\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAS --recode compound-genotypes --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6x8JZYyFBv7bGBGj4qxb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6x8JZYy98xZ7G59ykZvQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6x8JZYyBpGk7FX5y731g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVpjJYBPq0pKv7Y35448p\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EAS --recode compound-genotypes --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z640JF44FYJg18XXZqGZg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z640JF448q6275PVbFQ02\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z640JF443KxfQzZXg82BK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVq0JYBPgjZj95jG533jf\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EUR --recode compound-genotypes --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6B0J9vPqx5Fy5gfV5KJq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6B0J9vPZF7bZPXkbYvBJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6B0J9vPZ36V9XYXfpKGY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVq8JYBPq0pKv7Y35448v\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_MDE --recode compound-genotypes --out apoe_snps_AD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6b8Jf3zjq6275PVbFQ1x\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6b8Jf3zQkpgXbjxxFG6g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6b8Jf3zvqkgzzG9vZ5kx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVqQJYBPqjGQ794yBX3bv\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_SAS --recode compound-genotypes --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5XjJx2v0Jzp1zg9pk7qg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5XjJx2vB9QpfZ7ygBz3J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5XjJx2v5KpyXJJxx1y3V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVqjJYBPk6P6YFQzGP9Jf\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AAC --recode compound-genotypes --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z648J4yYV4fvZyJGbBPPx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z648J4yYXyFQVqbyxqyj0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z648J4yYqBv7bGBGj4899\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVv0JYBPXJfFXKqK9B4bQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AFR --recode compound-genotypes --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJgkPY9jV3VVFzbx57\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64jJgkPzv4k1f1pYbKZx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJgkPpG5YqfyYk22Jy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVv8JYBPXJfFXKqK9B4bZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AJ --recode compound-genotypes --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5x8JbJz8jKj5Pj1j404p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5x8JbJz4YV5FVxXqz9kP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5x8JbJz2pGg0V52Xp51y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVvQJYBPYZ982v2vBV586\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AMR --recode compound-genotypes --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6KjJBB11p3BJz3bz4q0g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6KjJBB1PJ0b43Z9p34Yk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6V8JBB1B2Q641z0QK4VF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVvjJYBPpyV38637Bq543\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAH --recode compound-genotypes --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6X0JzQFpG5YqfyYk22KG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6X0JzQFk4jXpj9YkFx4X\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6X0JzQFxgBV8P2gykJQx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVx0JYBPq0pKv7Y354493\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAS --recode compound-genotypes --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6Y0Jfpp2b7G635ZjXqq3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6Y8JfppGB3QZqfxq2qvX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6Y0JfppBpGk7FX5y730X\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVx8JYBPXJfFXKqK9B4f4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EAS --recode compound-genotypes --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJKvK60x1yQ0BZ42bj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJKvK60x1yQ0BZ42bp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJKvK98xZ7G59ykZpX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVxQJYBPp3vZ9vzj3gxx6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EUR --recode compound-genotypes --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6bQJf7Z96P6YFQzvP5P1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6bQJf7ZGPg373K5Q74x6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6bQJf7ZB2Q641z0QK4Vg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVxQJYBPxgBV8P2gKkPyK\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_FIN --recode compound-genotypes --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z670J2PGYGp7GG9PbQkFZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z670J2PGZ235Z576vjXkg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z670J2PGxGKKKYZ4Yp58q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVxjJYBPQ20QFgPGP1qjP\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_MDE --recode compound-genotypes --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5g0Jj53qx5Fy5gfV5KFk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5g0Jj53k4jXpj9YkFvyy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5fjJj53YZ982v2vpV3Gz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVy0JYBPQ20QFgPGP1qjV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_SAS --recode compound-genotypes --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5g0J494vQkKGVvVy5KP3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5g0J494g5BkFzbFY4zyY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5g0J494v5ybjQPxpkBQb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVy8JYBPZK5383fXPzZJQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AAC --recode compound-genotypes --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5YQJ7p9Fpf3XKgZby4Xx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5YQJ7p92Y98Bf9kpkvj5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5YQJ7p980zFFJqjk2X82\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVyQJYBPZK5383fXPzZJX\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AFR --recode compound-genotypes --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6XQJGyY5xbx6kqYVXy3F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6XjJGyYBY6Px7g6yk5f3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6XQJGyYJX3QpyV9fpjv6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVyjJYBPp3vZ9vzj3gxxB\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AJ --recode compound-genotypes --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z68QJFv4X5ZXGk12qkQ5j\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z68QJFv4Vv59GyF5VvJ9Y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6B8JFv4pvJk8k19ykVqf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVz0JYBPxgBV8P2gKkPyZ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAH --recode compound-genotypes --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6FjJ01Bv7PPVb8Gk5pgK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z698J01BjVkqQfJYV84bz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z698J01Bg5BkFzbFY5007\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVz8JYBPbKpyXJJxJ21Gj\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAS --recode compound-genotypes --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6j8JGKxJy4V8GG1gY2KY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6j8JGKxFjGQ794ypVzVV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6j8JGKxJZ2P2zV7Z63vx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVzQJYBPZ235Z576Gk97Z\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EAS --recode compound-genotypes --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5pQJ1Jgjp8Z6qvZV1QxG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5pQJ1Jgp8Vj8gzyb6X8b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5pQJ1JggzY48vQxYjg8P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zVzjJYBPp3vZ9vzj3gxxV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EUR --recode compound-genotypes --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5jQJb1gGB3QZqfxq2qp7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5jjJb1g436V9XYXfpJVx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5kQJb1g1K4YKgB3kV52q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zXQQJYBPbKpyXJJxJ21J3\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_FIN --recode compound-genotypes --out apoe_snps_RD_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6B0JVq7vZ518xbZXf61k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6B0JVq7pyV38637pq382\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6B0JVq7XpGg0V52Xp534\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zXQjJYBPp3vZ9vzj3gxxv\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_MDE --recode compound-genotypes --out apoe_snps_RD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6q8JkV5pY6Px7g6yk5fZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6qQJkV5gk6zfK99b53vJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6q8JkV5xX3QpyV9fpjy9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zXV0JYBPpG5YqfyY924Bg\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_SAS --recode compound-genotypes --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5fQJQbXx9yYJ9G4YBk21\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5fQJQbXqx5Fy5gfV5KFZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5fQJQbXgZ5k7Qkkjx4Qf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44892887_\n","  C_T\n","\n","Job ID: job-Gq2zXV8JYBPpG5YqfyY924Bk\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AAC --recode compound-genotypes --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5j8JQbVVK4YKgB3kV52k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5bQJQbVppGk7FX5y72yX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5bQJQbVQ20QFgPGz16BQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXVQJYBPbKpyXJJxJ21J7\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AFR --recode compound-genotypes --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJg78533Bj6qFgGVg2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJg789GJXBK5FxGbBj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJg787jZj95jGb30KF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXVjJYBPp3vZ9vzj3gy2Y\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AJ --recode compound-genotypes --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5Y0JVKf7gz68F4gz5F0y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5b0JVKfFk6ZPPJpz7Q78\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5Y0JVKf9GJXBK5FxGb80\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXX0JYBPxgBV8P2gKkPyx\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAH --recode compound-genotypes --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5ZjJGB7Q00VQ2QJfyx3q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5ZjJGB7XyP5022vQVQKg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5ZjJGB7xy4V8GG1gY2FY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXX0JYBPxgBV8P2gKkPyz\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAS --recode compound-genotypes --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6V0JYpqp8Vj8gzyb6XGg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6V0JYpqjYxjxz4gZJky6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6V0JYpqQB71Z089jzgzj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXX8JYBPZ235Z576Gk988\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EAS --recode compound-genotypes --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJGx23FBq8B3xXGZG0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJGx2GbbbyzpqQ49qx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJGx2ByV38637pq37q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXXQJYBPQ20QFgPGP1qy6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EUR --recode compound-genotypes --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z638JZ63f0x1yQ0BZ42bb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z638JZ63p9QpfZ7ygBz51\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z638JZ63qx5Fy5gfV5KJ7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXXjJYBPbKpyXJJxJ21JF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_MDE --recode compound-genotypes --out apoe_snps_AD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zQJ1QKfpbbpQyqyjZBy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zQJ1QKfpbbpQyqyjZBv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5zQJ1QKYGp7GG9PbQkB5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXY0JYBPbKpyXJJxJ21JJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_SAS --recode compound-genotypes --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z60QJ90pYGp7GG9PbQkBK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z60QJ90pZ235Z576vjXk5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z60QJ90pfKQBpvvFpZKk7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXY8JYBPbKpyXJJxJ21JP\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AAC --recode compound-genotypes --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6KQJz9YXJfFXKqKk9j51\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6KjJz9YQ00VQ2QJfzQg1\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6KQJz9YpyV38637pq38p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXYQJYBPZ235Z576Gk98B\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AFR --recode compound-genotypes --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z66QJYPYG7PPVb8Gk5pg9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJYPY7Z5k7Qkkjx4VK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJYPY25ZXGk12qkPJ3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXYjJYBPZK5383fXPzZXp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AJ --recode compound-genotypes --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6p0JpkB4235Z576vjXpx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6p0JpkB1fY2YYYBvk8g4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6p0JpkB000VQ2QJfzQgJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXZ0JYBPbKpyXJJxJ21JY\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AMR --recode compound-genotypes --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64jJ4zJ7gz68F4gz5F3f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64jJ4zJ2yP5022vQVQQ4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64jJ4zJJFYYY1f3zy41V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXZ8JYBPqpkx2415FjPjJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAH --recode compound-genotypes --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z63QJQ17vZ518xbZXf616\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z62QJQ17Q9X6XJJJXQ4zp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z658JQ17ppGk7FX5y72zX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXZQJYBPZ235Z576Gk98G\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAS --recode compound-genotypes --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z760Jy6GJZ2P2zV7Z63y9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z760Jy6G3kqZqgG1zbXfj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z770Jy6G0B71Z089jzj3P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXZjJYBPZ235Z576Gk98K\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EAS --recode compound-genotypes --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6k0J22ybxbx6kqYVXy3P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6fQJ22yYKxfQzZXg82FB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6fQJ22yk4jXpj9YkFx4p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXb0JYBPZ235Z576Gk98Q\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EUR --recode compound-genotypes --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6pQJY12ffZpFQy6yKpK4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6pQJY12gJBGp39FQQ5bk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6p8JY12V4fvZyJGbBPV2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXb8JYBPjq6275PV5FXXV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_FIN --recode compound-genotypes --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0Jpxf2vGZpPykZzxFB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0Jpxf1p3BJz3bz4pzG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0Jpxf6KQBpvvFpZKjp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXbQJYBPVv59GyF51vPzV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_MDE --recode compound-genotypes --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6yjJ802Fjz4z629QgbJP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6yjJ8027JBGp39FQQk79\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6yjJ8024yF8kGz8ZvqKf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXbjJYBPVv59GyF51vPzY\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_SAS --recode compound-genotypes --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0J3JY020QFgPGz1kxK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0J3JY1fY2YYYBvk8YB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0J3JYFx5Fy5gfV5KJ3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXf0JYBPp8Vj8gzy56ZvP\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AAC --recode compound-genotypes --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6f0J5G914fvZyJGbBPV0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6bjJ5G914fvZyJGbBPQy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6bjJ5G9GPg373K5Q74xB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXf0JYBPVv59GyF51vPzg\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AFR --recode compound-genotypes --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z75jJjp4xFYYY1f3zy46Y\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z75jJjp4Y42XyZF6q7485\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z75jJjp4f0x1yQ0BZ42gY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXf8JYBPgY8yBj32KZ04K\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AJ --recode compound-genotypes --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z8J7pxXyFQVqbyxqyzG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6zQJ7pxX5ZXGk12qkQ6Q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z8J7pxqpkx2415qjKfP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXfQJYBPYyQ74Yf3FqfxV\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AMR --recode compound-genotypes --out apoe_snps_RD_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6BjJVPQXyFQVqbyxqyjP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z688JVPQgzY48vQxYjg9V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z688JVPQpY6Px7g6yk5bV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXfjJYBPp8Vj8gzy56Zvb\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAH --recode compound-genotypes --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6j0JpZv72Y7YQfBvF87Z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6j8JpZvBY6Px7g6yk5fB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6j0JpZvFjGQ794ypVzVP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXg0JYBPVv59GyF51vQ03\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAS --recode compound-genotypes --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z650JY3v6pPyJF1bp0BXZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z650JY3v39jV3VVFzbx5J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z650JY3vGbbbyzpqQ49v5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXg8JYBPgY8yBj32KZ04X\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EAS --recode compound-genotypes --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6f8J03XGQkKGVvVy5KVJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6f8J03XPv4k1f1pYbKbQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6f8J03X1fY2YYYBvk8bV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXgQJYBPq06KvB1Y3y8ZB\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EUR --recode compound-genotypes --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6ZQJ45Q2yFQVqbyxqykv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6ZQJ45Q09X6XJJJXQ50P\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6ZjJ45QB2Q641z0QK4VV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXgjJYBPVv59GyF51vQ0P\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_MDE --recode compound-genotypes --out apoe_snps_RD_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6G8JZ6b7k6zfK99b53qb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6G8JZ6b39jV3VVFzbx5x\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6G8JZ6bPJ0b43Z9p34Yg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXj0JYBPQB71Z0898zpfG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_SAS --recode compound-genotypes --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6x8J8ZJ3FBq8B3xXGZJF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6x8J8ZJ7zY48vQxYjgBk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6x8J8ZJJ9yYJ9G4YFGg3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44905307_\n","  A_T\n","\n","Job ID: job-Gq2zXj8JYBPp8Vj8gzy56Zvv\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EUR --recode compound-genotypes --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6Y8Jq1k39jV3VVFzbx6z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6Y8Jq1k9GJXBK5FxGbFK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6Y8Jq1k0B71Z089jzj04\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr19_44908756_\n","  C_A\n","\n","Job ID: job-Gq2zXjQJYBPQqZp25bbGGfZY\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EUR --recode compound-genotypes --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6JQJ61QpvJk8k19ykVvf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6JQJ61QqBv7bGBGj48BQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6JQJ61QfKQBpvvFpZKp9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_25897620_\n","  C_T\n","\n","Job ID: job-Gq2zXjjJYBPQqZp25bbGGfZb\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AAC --recode compound-genotypes --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z750Jb5p5KpyXJJxx1y7p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z750Jb5p4yF8kGz8ZvqP0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z750Jb5p0Z5kqBjvYx6Qj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXjjJYBPQB71Z0898zpfQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AFR --recode compound-genotypes --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z72jJJK6b33Bj6qFgGVjg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z72jJJK6kGJXBK5FxGbG3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z72jJJK6pkFKG415xvYV6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXk0JYBPgY8yBj32KZ04b\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AJ --recode compound-genotypes --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z8Jjg7gvP5PQVZXgkfv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z8Jjg7gZ5k7Qkkjx4XX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z8Jjg7jp8Z6qvZV1V17\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXk8JYBPp8Vj8gzy56Zvz\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAH --recode compound-genotypes --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0JyQ20Jzp1zg9pk7y0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0JyQ2B8Vj8gzyb6X9Q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z5z0JyQ25KpyXJJxx1y5f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXkQJYBPjq6275PV5FXj9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAS --recode compound-genotypes --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z608JZqJkYK3k1Y7v4vjv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z608JZqJZYV5FVxXqz9kz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z608JZqJqG0JVbz0Zf5BQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXkjJYBPq06KvB1Y3y8ZJ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EAS --recode compound-genotypes --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z778JJjg6fZpFQy6yKpKk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z778JJjgJgBV8P2gykJXP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z778JJjgGqkgzzG9vZ5pq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXp0JYBPQqZp25bbGGfZp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EUR --recode compound-genotypes --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6yQJJbX2JfFXKqKk9j5g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6v0JJbX1p3BJz3bz4q1J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6qjJJbXJFYYY1f3zy42p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXp8JYBPQqZp25bbGGfZv\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_SAS --recode compound-genotypes --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6v8JbQFgvP5PQVZXgkf4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6v8JbQFXb7G635ZjXqqV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6v8JbQFx9yYJ9G4YFGfx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXpQJYBPQB71Z0898zpff\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AAC --recode compound-genotypes --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z67QJYqKf0x1yQ0BZ42f0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z67QJYqKzjK97Q0vkkV73\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z67QJYqKvbbbyzpqQ49xy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXpjJYBPjq6275PV5FXjG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AFR --recode compound-genotypes --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJk4VF057zf96pBX8J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJk4V44z5z9b6z6Qjq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z64QJk4VBG5YqfyYk22Jv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXq0JYBPVv59GyF51vQ0X\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AJ --recode compound-genotypes --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6BQJyFqQB71Z089jzgz2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6BjJyFqpkFKG415xvYP3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6BQJyFqffZpFQy6yKpJ2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXyjJYBPp8Vj8gzy56Zx9\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AMR --recode compound-genotypes --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z75QJgvYQ00VQ2QJfzQj5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z75QJgvYjbBZJ7p3j42K0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z75QJgvYfKQBpvvFpZKvQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zXz0JYBPQB71Z0898zpg5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAH --recode compound-genotypes --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6QjJyxggJBGp39FQQ5Vf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6QjJyxgqpf3XKgZby4ZQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6QjJyxgq0pKv7Y3b42GP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zY28JYBPp8Vj8gzy56ZxG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAS --recode compound-genotypes --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z73QJ4b00Jzp1zg9pk7z8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z73QJ4b02yP5022vQX46j\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z73QJ4b02vGZpPykZzxK2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zY90JYBPQqZp25bbGGfb6\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EAS --recode compound-genotypes --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6GjJ49JjbBZJ7p3j42GQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6PQJ49JQkpgXbjxxFG63\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6GjJ49JkvfzqY6Vg0qkv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zY98JYBPjq6275PV5FXqz\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EUR --recode compound-genotypes --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z710J55KvQkKGVvVy5KX5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z710J55KqqJzZZVYvj73b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z710J55KvB3QZqfxq2qxB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zY98JYBPYyQ74Yf3Fqg4P\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_FIN --recode compound-genotypes --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z71QJy17zv4k1f1pYbKfp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z71QJy17gjZj95jGb30QG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z71QJy17vbbbyzpqQ49zY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zY9QJYBPQqZp25bbGGfbP\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_MDE --recode compound-genotypes --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6YQJ3P1B2Q641z0QK4VJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6V0J3P13FBq8B3xXGZGz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6V0J3P12yFQVqbyxqyk0\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zY9jJYBPq06KvB1Y3y8bK\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_SAS --recode compound-genotypes --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8J15x7zY48vQxYjgFj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8J15x0kpgXbjxxFG8J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8J15x80zFFJqjk2XFQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYB0JYBPQB71Z0898zpk2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AAC --recode compound-genotypes --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z77jJXfvByV38637pq3F3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z77jJXfvBxG95XQ7qV5XZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z79jJXfv7JBGp39FQQk7k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYB8JYBPQB71Z0898zpk5\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AFR --recode compound-genotypes --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z0JJ9QB2Q641z0QK4Vv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z0JJ9QF057zf96pBXBK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6z0JJ9QF06KvB1YYy4XK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYBQJYBPQqZp25bbGGfbp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AJ --recode compound-genotypes --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z74jJQpYj0zFFJqjk2XBy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z74jJQpYYZ982v2vpV3P5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z74jJQpYxgBV8P2gykJX3\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYBjJYBPp3vZ9vzj3gy4p\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAH --recode compound-genotypes --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q8JGX9xgBV8P2gykJYJ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q8JGX9jp8Z6qvZV1V1Z\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q8JGX9v7PPVb8Gk5pjq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYF0JYBPQB71Z0898zpkF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAS --recode compound-genotypes --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8J97g7gz68F4gz63PG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7BQJ97g0B71Z089jzj3q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8J97g9GJXBK5FxGbGb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYF8JYBPQB71Z0898zpy1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EAS --recode compound-genotypes --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6y0J4xF6pbbpQyqyjfJg\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6xjJ4xF09X6XJJJXQ50p\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6xjJ4xFFx5Fy5gfV5KKK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYFQJYBPjq6275PV5FXv7\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EUR --recode compound-genotypes --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z750JfgbQG472YY8Yy7Yf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z750JfgbZYV5FVxXqz9qZ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z750JfgbbQG3QbP4vFGV9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYFjJYBPq06KvB1Y3y8by\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_SAS --recode compound-genotypes --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z730JK1VjjKj5Pj1j406f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z730JK1VxFYYY1f3zy467\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z730JK1Vv5ybjQPxpkBXy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171645_\n","  A_G\n","\n","Job ID: job-Gq2zYG0JYBPq06KvB1Y3y8f0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AAC --recode compound-genotypes --out apoe_snps_AD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z77jJx7PZK5383fXzzX2b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z780Jx7PzYF8k9B6z0Gyk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z77jJx7PQG472YY8Yy7Z9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYG0JYBPYyQ74Yf3Fqg4v\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AFR --recode compound-genotypes --out apoe_snps_AD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6v0J8X1k4jXpj9YkFx5j\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6v0J8X1jjKj5Pj1j405q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6v8J8X1qBv7bGBGj4qxP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYG8JYBPQqZp25bbGGffG\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_AJ --recode compound-genotypes --out apoe_snps_AD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6q0J55ZZyF8kGz8ZvqK4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6q0J55ZY9jV3VVFzbx7g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z6q0J55ZXb7G635ZjXqqP\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYGQJYBPQB71Z0898zpy8\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAH --recode compound-genotypes --out apoe_snps_AD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Z8JBP8FjGQ794ypVzfG\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Z8JBP8BpGk7FX5y7340\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Z8JBP880zFFJqjk2XJ9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYGjJYBPQqZp25bbGGffK\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_CAS --recode compound-genotypes --out apoe_snps_AD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z76QJ5QZ6fZpFQy6yKpKf\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z76QJ5QZJgBV8P2gykJX9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z798J5QZ436V9XYXfpKKz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYJ0JYBPq06KvB1Y3y8f4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EAS --recode compound-genotypes --out apoe_snps_AD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z74jJYxVqBv7bGBGj4qyQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z74jJYxVYGp7GG9PbQkJv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z74jJYxVZ36V9XYXfpKKq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYJ8JYBPgY8yBj32KZ06f\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_EUR --recode compound-genotypes --out apoe_snps_AD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z730JQ5vV4fvZyJGbBPVp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z730JQ5vjq6275PVbFQ28\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z730JQ5vg2Y7YQfBvF890\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYJQJYBPVv59GyF51vQ67\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_AD_SAS --recode compound-genotypes --out apoe_snps_AD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7JjJ2q67jZj95jGb30XF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7K0J2q6PJ0b43Z9p34b8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7JjJ2q639jV3VVFzbx8k\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYJjJYBPQqZp25bbGGffQ\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AAC --recode compound-genotypes --out apoe_snps_Control_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q0J5645KpyXJJxx1y8f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q0J56460x1yQ0BZ42jk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q0J564J9yYJ9G4YFGj7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYY0JYBPp8Vj8gzy56Zyq\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AFR --recode compound-genotypes --out apoe_snps_Control_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z738JZ9XYGp7GG9PbQkJj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z738JZ9Xkqz9367fjxqFp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z760JZ9XgJBGp39FQQk7b\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYY8JYBPgY8yBj32KZ06v\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AJ --recode compound-genotypes --out apoe_snps_Control_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7jjJB3ZGB3QZqfxq2qyj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7jjJB3ZGPg373K5Q7548\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7jjJB3Z25ZXGk12qkQ78\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYbQJYBPq06KvB1Y3y8qf\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_AMR --recode compound-genotypes --out apoe_snps_Control_AMR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q0J3p2v7PPVb8Gk5pjk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q0J3p2ppGk7FX5y7335\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7Q0J3p2jbBZJ7p3j42KQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYbjJYBPQqZp25bbGGfff\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAH --recode compound-genotypes --out apoe_snps_Control_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7XQJKzZv5ybjQPxpkBYv\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7b0JKzZXb7G635ZjXqv9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7XQJKzZzjK97Q0vkkVB2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYf0JYBPVv59GyF51vQ6P\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_CAS --recode compound-genotypes --out apoe_snps_Control_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7j8JY26Yv976JQ9bf1Py\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7j8JY26gYbkxY88bBYYY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7j8JY26g5BkFzbFY502g\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYf8JYBPjq6275PV5FXvg\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EAS --recode compound-genotypes --out apoe_snps_Control_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z73QJb1J9GJXBK5FxGbG5\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z73QJb1J342XyZF6q747q\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z73QJb1J0G472YY8Yy7YQ\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYfQJYBPQqZp25bbGGffy\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_EUR --recode compound-genotypes --out apoe_snps_Control_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z72jJ4461p3BJz3bz4q2J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z72jJ44675BkFzbFY501K\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z72QJ446BxG95XQ7qV5XB\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYx0JYBPp8Vj8gzy56ZzF\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_FIN --recode compound-genotypes --out apoe_snps_Control_FIN\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z860JYz1vfXb0Gkxbb5Vj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z860JYz1p3vZ9vzjYgqzq\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z860JYz1XpGg0V52Xp6pz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYz0JYBPgY8yBj32KZ07B\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_MDE --recode compound-genotypes --out apoe_snps_Control_MDE\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z78QJFVf0qZp25bbvGZXb\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z78QJFVf0B71Z089jzj3f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z78QJFVf3v976JQ9bf1P4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYz8JYBPq06KvB1Y3y8v2\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_Control_SAS --recode compound-genotypes --out apoe_snps_Control_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7JjJzb91pX90JQkxbg3V\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7JjJzb9PjK97Q0vkkV9J\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7JjJzb996P6YFQzvP5Q9\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYzQJYBPjq6275PV5G6Jv\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AAC --recode compound-genotypes --out apoe_snps_RD_AAC\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7f0J9gvqpf3XKgZby4fV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7f0J9gvffZpFQy6yKpV4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7f0J9gvqx5Fy5gfV5Q3F\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYzQJYBPQB71Z0898zq03\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AFR --recode compound-genotypes --out apoe_snps_RD_AFR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8JqV1gY8yBj32yYy55\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8JqV1q06KvB1YYy4Xx\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7B8JqV1YFBq8B3xXGZKF\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zYzjJYBPYyQ74Yf3Fqg5b\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_AJ --recode compound-genotypes --out apoe_snps_RD_AJ\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z81jJZkPzpx2fX06j8vQV\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z81jJZkPXb7G635ZjXqx2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z81jJZkPq057zf96pF5y6\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zZ00JYBPpG5YqfyY924K1\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAH --recode compound-genotypes --out apoe_snps_RD_CAH\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7G8JQxV7k6zfK99b53xp\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7G8JQxV020QFgPGz1p08\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7G8JQxVFx5Fy5gfV5Q2f\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zZ08JYBPp3vZ9vzj3gyF0\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_CAS --recode compound-genotypes --out apoe_snps_RD_CAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7vjJb7Xp3vZ9vzjYgqzX\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7p8Jb7Xk8xZ7G59ykZxj\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7p8Jb7XQJzp1zg9pkqV4\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zZ0QJYBPZ235Z576Gk9Kp\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EAS --recode compound-genotypes --out apoe_snps_RD_EAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7f8JZ3PvfXb0Gkxbb5VK\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7f8JZ3PxGKKKYZ4Yp5F2\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7f8JZ3PvQkKGVvVy5PF8\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zZ0jJYBPQ20QFgPGP1v6J\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_EUR --recode compound-genotypes --out apoe_snps_RD_EUR\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z820J42gVv59GyF5VvJGz\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z820J42gzpx2fX06j8vQY\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z820J42ggp85fYz9YZ6Yk\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zZ10JYBPbKpyXJJxJ21b4\n","\n","Using input JSON:\n","{\n","    \"cmd\": \"plink --bfile apoe_snps_RD_SAS --recode compound-genotypes --out apoe_snps_RD_SAS\",\n","    \"in\": [\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7bQJKZZ2yP5022vQX47G\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7bQJKZZ9vfzqY6Vg0qqy\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gq2z7bQJKZZPYF8k9B6z0Gz7\"\n","            }\n","        },\n","        {\n","            \"$dnanexus_link\": {\n","                \"project\": \"project-Gbgg70jJYBPf3yZ26F3GVFjx\",\n","                \"id\": \"file-Gky57B8JYBPy8ZV1GfFG444b\"\n","            }\n","        }\n","    ]\n","}\n","\n","Calling app-GkvkZK89gxxfJBYgQX76pY8g with output destination project-Gbgg70jJY\n","  BPf3yZ26F3GVFjx:/wgs_analysis/results/12_protective_variants/chr21_26171723_\n","  T_C\n","\n","Job ID: job-Gq2zZ18JYBPZ235Z576Gk9Ky\n"]}],"source":["%%bash\n","\n","for var_id in {\"chr19_44892887_C_T\",\"chr19_44905307_A_T\",\"chr19_44908756_C_A\",\"chr21_25897620_C_T\",\"chr21_26171645_A_G\",\"chr21_26171723_T_C\"};\n","do\n","    files=($(ls ../../mnt/project/wgs_analysis/results/12_protective_variants/${var_id}/*.bim 2>/dev/null))\n","    \n","    for file in \"${files[@]}\";\n","    do\n","        filename=$(basename \"$file\" .bim)\n","        \n","        dx run swiss-army-knife \\\n","        -iin=\"/wgs_analysis/results/12_protective_variants/${var_id}/apoe_snps_${filename}.bim\" \\\n","        -iin=\"/wgs_analysis/results/12_protective_variants/${var_id}/apoe_snps_${filename}.bed\" \\\n","        -iin=\"/wgs_analysis/results/12_protective_variants/${var_id}/apoe_snps_${filename}.fam\" \\\n","        -iin=\"/wgs_analysis/data/apoe_variants.txt\" \\\n","        -icmd=\"plink --bfile apoe_snps_${filename} --recode compound-genotypes --out apoe_snps_${filename}\" \\\n","        --instance-type mem1_hdd1_v2_x16 \\\n","        --destination \"${projectid}:/wgs_analysis/results/12_protective_variants/${var_id}\"\n","    done\n","done\n"]},{"cell_type":"code","execution_count":null,"id":"d29eb5be-e251-44c8-baae-fe3e2d0d2b86","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true}},"outputs":[],"source":["\"\"\"\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------ PAUSE HERE UNTIL PREVIOUS STEP COMPLETES ------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","------------------------------------------------------------------------------------------------------\n","\"\"\""]},{"cell_type":"code","execution_count":4,"id":"562e2fef-af36-40a4-9bcc-6ed3db17eab0","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_RD_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 1,975 of 1,975 bytes (100%) /opt/notebooks/apoe_snps_Control_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 500 of 500 bytes (100%) /opt/notebooks/apoe_snps_RD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 51,550 of 51,550 bytes (100%) /opt/notebooks/apoe_snps_RD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 200 of 200 bytes (100%) /opt/notebooks/apoe_snps_Control_AMR.pedd\n","\u001b[2K[===========================================================>] Completed 175 of 175 bytes (100%) /opt/notebooks/apoe_snps_RD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 53,625 of 53,625 bytes (100%) /opt/notebooks/apoe_snps_AD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_RD_FIN.pedd\n","\u001b[2K[===========================================================>] Completed 300 of 300 bytes (100%) /opt/notebooks/apoe_snps_RD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 725 of 725 bytes (100%) /opt/notebooks/apoe_snps_Control_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_AD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 225 of 225 bytes (100%) /opt/notebooks/apoe_snps_AD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_AD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 275 of 275 bytes (100%) /opt/notebooks/apoe_snps_RD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 200 of 200 bytes (100%) /opt/notebooks/apoe_snps_Control_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 350 of 350 bytes (100%) /opt/notebooks/apoe_snps_AD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 200 of 200 bytes (100%) /opt/notebooks/apoe_snps_RD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 825 of 825 bytes (100%) /opt/notebooks/apoe_snps_Control_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_RD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 332,550 of 332,550 bytes (100%) /opt/notebooks/apoe_snps_Control_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 775 of 775 bytes (100%) /opt/notebooks/apoe_snps_Control_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 3,525 of 3,525 bytes (100%) /opt/notebooks/apoe_snps_Control_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 100 of 100 bytes (100%) /opt/notebooks/apoe_snps_AD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 2,100 of 2,100 bytes (100%) /opt/notebooks/apoe_snps_Control_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 625 of 625 bytes (100%) /opt/notebooks/apoe_snps_AD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 325 of 325 bytes (100%) /opt/notebooks/apoe_snps_RD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_Control_FIN.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_AD_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_AD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 1,100 of 1,100 bytes (100%) /opt/notebooks/apoe_snps_Control_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 250 of 250 bytes (100%) /opt/notebooks/apoe_snps_RD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 421,225 of 421,225 bytes (100%) /opt/notebooks/apoe_snps_Control_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 225 of 225 bytes (100%) /opt/notebooks/apoe_snps_AD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 925 of 925 bytes (100%) /opt/notebooks/apoe_snps_RD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 500 of 500 bytes (100%) /opt/notebooks/apoe_snps_RD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 21,975 of 21,975 bytes (100%) /opt/notebooks/apoe_snps_AD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 34,325 of 34,325 bytes (100%) /opt/notebooks/apoe_snps_RD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 400 of 400 bytes (100%) /opt/notebooks/apoe_snps_RD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 5,550 of 5,550 bytes (100%) /opt/notebooks/apoe_snps_Control_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 300 of 300 bytes (100%) /opt/notebooks/apoe_snps_Control_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 200 of 200 bytes (100%) /opt/notebooks/apoe_snps_AD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 450 of 450 bytes (100%) /opt/notebooks/apoe_snps_Control_AMR.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_AD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 2,125 of 2,125 bytes (100%) /opt/notebooks/apoe_snps_Control_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 200 of 200 bytes (100%) /opt/notebooks/apoe_snps_Control_FIN.pedd\n","\u001b[2K[===========================================================>] Completed 1,425 of 1,425 bytes (100%) /opt/notebooks/apoe_snps_Control_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 150 of 150 bytes (100%) /opt/notebooks/apoe_snps_AD_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 200 of 200 bytes (100%) /opt/notebooks/apoe_snps_RD_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 850 of 850 bytes (100%) /opt/notebooks/apoe_snps_Control_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 5,675 of 5,675 bytes (100%) /opt/notebooks/apoe_snps_Control_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 100 of 100 bytes (100%) /opt/notebooks/apoe_snps_AD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 150 of 150 bytes (100%) /opt/notebooks/apoe_snps_RD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_AD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 325 of 325 bytes (100%) /opt/notebooks/apoe_snps_AD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_RD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_RD_AMR.pedd\n","\u001b[2K[===========================================================>] Completed 800 of 800 bytes (100%) /opt/notebooks/apoe_snps_AD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 450 of 450 bytes (100%) /opt/notebooks/apoe_snps_RD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 4,925 of 4,925 bytes (100%) /opt/notebooks/apoe_snps_Control_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 1,600 of 1,600 bytes (100%) /opt/notebooks/apoe_snps_Control_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 100 of 100 bytes (100%) /opt/notebooks/apoe_snps_Control_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_Control_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 375 of 375 bytes (100%) /opt/notebooks/apoe_snps_AD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 600 of 600 bytes (100%) /opt/notebooks/apoe_snps_RD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_RD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 150 of 150 bytes (100%) /opt/notebooks/apoe_snps_RD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 4,425 of 4,425 bytes (100%) /opt/notebooks/apoe_snps_AD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_Control_AMR.pedd\n","\u001b[2K[===========================================================>] Completed 875 of 875 bytes (100%) /opt/notebooks/apoe_snps_Control_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_Control_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 3,875 of 3,875 bytes (100%) /opt/notebooks/apoe_snps_Control_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 675 of 675 bytes (100%) /opt/notebooks/apoe_snps_Control_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_AD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 275 of 275 bytes (100%) /opt/notebooks/apoe_snps_RD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 62,400 of 62,400 bytes (100%) /opt/notebooks/apoe_snps_Control_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_Control_FIN.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_AD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 100 of 100 bytes (100%) /opt/notebooks/apoe_snps_RD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_Control_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 200 of 200 bytes (100%) /opt/notebooks/apoe_snps_RD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 2,425 of 2,425 bytes (100%) /opt/notebooks/apoe_snps_Control_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 175 of 175 bytes (100%) /opt/notebooks/apoe_snps_RD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_AD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_AD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_AD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 1,375 of 1,375 bytes (100%) /opt/notebooks/apoe_snps_Control_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 650 of 650 bytes (100%) /opt/notebooks/apoe_snps_Control_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 6,150 of 6,150 bytes (100%) /opt/notebooks/apoe_snps_RD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_AD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 2,425 of 2,425 bytes (100%) /opt/notebooks/apoe_snps_Control_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 625 of 625 bytes (100%) /opt/notebooks/apoe_snps_RD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 150 of 150 bytes (100%) /opt/notebooks/apoe_snps_RD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 225 of 225 bytes (100%) /opt/notebooks/apoe_snps_RD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 275 of 275 bytes (100%) /opt/notebooks/apoe_snps_RD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_RD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 4,475 of 4,475 bytes (100%) /opt/notebooks/apoe_snps_AD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 100 of 100 bytes (100%) /opt/notebooks/apoe_snps_RD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_AD_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_Control_MDE.pedd\n","\u001b[2K[===========================================================>] Completed 650 of 650 bytes (100%) /opt/notebooks/apoe_snps_Control_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 6,175 of 6,175 bytes (100%) /opt/notebooks/apoe_snps_RD_EUR.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_AD_CAS.pedd\n","\u001b[2K[===========================================================>] Completed 175 of 175 bytes (100%) /opt/notebooks/apoe_snps_RD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_Control_AMR.pedd\n","\u001b[2K[===========================================================>] Completed 25 of 25 bytes (100%) /opt/notebooks/apoe_snps_Control_FIN.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_AD_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 375 of 375 bytes (100%) /opt/notebooks/apoe_snps_AD_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_Control_AJ.pedd\n","\u001b[2K[===========================================================>] Completed 50 of 50 bytes (100%) /opt/notebooks/apoe_snps_AD_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_AD_SAS.pedd\n","\u001b[2K[===========================================================>] Completed 1,375 of 1,375 bytes (100%) /opt/notebooks/apoe_snps_Control_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 700 of 700 bytes (100%) /opt/notebooks/apoe_snps_Control_CAH.pedd\n","\u001b[2K[===========================================================>] Completed 875 of 875 bytes (100%) /opt/notebooks/apoe_snps_Control_AAC.pedd\n","\u001b[2K[===========================================================>] Completed 3,875 of 3,875 bytes (100%) /opt/notebooks/apoe_snps_Control_AFR.pedd\n","\u001b[2K[===========================================================>] Completed 75 of 75 bytes (100%) /opt/notebooks/apoe_snps_AD_EAS.pedd\n","\u001b[2K[===========================================================>] Completed 62,500 of 62,500 bytes (100%) /opt/notebooks/apoe_snps_Control_EUR.pedd\n"]}],"source":["for var_id in [\"chr19_44892887_C_T\",\"chr19_44905307_A_T\",\"chr19_44908756_C_A\",\"chr21_25897620_C_T\",\"chr21_26171645_A_G\",\"chr21_26171723_T_C\"]:\n","    ! mkdir {var_id}\n","    \n","    ! dx download wgs_analysis/results/12_protective_variants/{var_id}/*.ped --overwrite\n","    ! mv *.ped {var_id}/\n","    "]},{"cell_type":"code","execution_count":8,"id":"8a1beda0-8c99-4e0b-a70a-58a9ff328b6d","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[2K[===========================================================>] Completed 5,721 of 5,721 bytes (100%) /opt/notebooks/APOE_genotypes_PLINK_ped.pypy\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 156 of 156 bytes (100%) chr19_44892887_C_T/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq360BjJYBPk6P6YFQzGfQFq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:19 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:20 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 309 of 309 bytes (100%) chr19_44892887_C_T/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq360F8JYBPk6P6YFQzGfQFz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:21 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:22 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 426 of 426 bytes (100%) chr19_44892887_C_T/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq360G0JYBPk6P6YFQzGfQG1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:24 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr19_44892887_C_T/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq360GjJYBPgjZj95jG5G09p\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:27 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr19_44892887_C_T/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360J8JYBPXJfFXKqK9Q8Q2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:29 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr19_44892887_C_T/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360K0JYBPjYxjxz4g4gJQB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:32 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:33 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 58,665 of 58,665 bytes (100%) chr19_44892887_C_T/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq360KjJYBPgjZj95jG5G0BV\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:35 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:35 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr19_44892887_C_T/apoe_final_AD_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq360P8JYBPqjGQ794yBpJY1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:37 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:38 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 723 of 723 bytes (100%) chr19_44892887_C_T/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360Q0JYBPXJfFXKqK9Q8Y7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:40 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:41 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 867 of 867 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq360QjJYBPq0pKv7Y35PXxF\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:43 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:43 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,379 of 2,379 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq360V8JYBPYZ982v2vBgK7k\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:45 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:46 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,235 of 2,235 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq360X0JYBPjYxjxz4g4gJV5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:48 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:49 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AMR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 264 of 264 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","ID                                file-Gq360XjJYBPgjZj95jG5G0FV\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:51 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,290 of 1,290 bytes (100%) chr19_44892887_C_T/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq360Y8JYBPYZ982v2vBgK85\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:53 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:54 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 966 of 966 bytes (100%) chr19_44892887_C_T/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360Z0JYBPXJfFXKqK9Q8fb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:57 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 903 of 903 bytes (100%) chr19_44892887_C_T/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360ZQJYBPYZ982v2vBgK8Y\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:46:58 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:46:59 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 367,842 of 367,842 bytes (100%) chr19_44892887_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq360b8JYBPk6P6YFQzGfQXk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:01 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:02 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_FIN.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr19_44892887_C_T/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","ID                                file-Gq360f0JYBPq0pKv7Y35PXz3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:04 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 264 of 264 bytes (100%) chr19_44892887_C_T/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq360fjJYBPYZ982v2vBgKFq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:07 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 3,891 of 3,891 bytes (100%) chr19_44892887_C_T/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360g8JYBPk6P6YFQzGfQYB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:09 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:10 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 264 of 264 bytes (100%) chr19_44892887_C_T/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq360j0JYBPXJfFXKqK9Q8gg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 408 of 408 bytes (100%) chr19_44892887_C_T/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq360jQJYBPx9yYJ9G43QqvB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:14 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 354 of 354 bytes (100%) chr19_44892887_C_T/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq360k8JYBPXpGg0V5232j67\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:17 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 390 of 390 bytes (100%) chr19_44892887_C_T/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq360p0JYBPXpGg0V5232j6F\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 237 of 237 bytes (100%) chr19_44892887_C_T/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360pQJYBPx9yYJ9G43QqvX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:22 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:23 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 84 of 84 bytes (100%) chr19_44892887_C_T/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360q8JYBPgJBGp39F0pXbK\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:25 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:26 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 56,775 of 56,775 bytes (100%) chr19_44892887_C_T/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq360v0JYBPqBv7bGBG8G40b\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:28 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:29 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_FIN.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_FIN.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr19_44892887_C_T/apoe_final_RD_FIN.APOE_GENOTYPES.csv\n","ID                                file-Gq360vQJYBPxZ2P2zV74QKPy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_FIN.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:30 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:31 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr19_44892887_C_T/apoe_final_RD_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq360x8JYBPXpGg0V5232j8p\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 588 of 588 bytes (100%) chr19_44892887_C_T/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq360y0JYBPk6P6YFQzGfQb3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:36 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:37 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 414 of 414 bytes (100%) chr19_44892887_C_T/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq360yQJYBPq0pKv7Y35PY3K\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:38 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:39 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 749 of 749 bytes (100%) chr19_44892887_C_T/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq360zjJYBPqjGQ794yBpJyV\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:43 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:44 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,104 of 1,104 bytes (100%) chr19_44892887_C_T/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3610jJYBPqjGQ794yBpJyY\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:47 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:48 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr19_44892887_C_T/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3611jJYBPXJfFXKqK9Q8xG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:52 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr19_44892887_C_T/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36130JYBPXJfFXKqK9Q8xk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:47:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:47:56 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr19_44892887_C_T/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36140JYBPq0pKv7Y35PYz9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:00 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:01 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 147,753 of 147,753 bytes (100%) chr19_44892887_C_T/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36150JYBPgjZj95jG5G0Yj\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:04 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr19_44892887_C_T/apoe_final_AD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36168JYBPk6P6YFQzGfQjF\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:09 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:09 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,863 of 1,863 bytes (100%) chr19_44892887_C_T/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36178JYBPYZ982v2vBgKQX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:14 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_AD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,119 of 2,119 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36188JYBPq0pKv7Y35PZ03\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:17 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 5,899 of 5,899 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3619QJYBPx9yYJ9G43Qv2f\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:22 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:22 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 5,559 of 5,559 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361BQJYBPgJBGp39F0pXyg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 690 of 690 bytes (100%) chr19_44892887_C_T/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361FQJYBPx9yYJ9G43Qv34\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:30 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:31 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_AMR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 3,144 of 3,144 bytes (100%) chr19_44892887_C_T/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361GjJYBPZyF8kGz853fB9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:35 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:35 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,400 of 2,400 bytes (100%) chr19_44892887_C_T/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361JjJYBPx9yYJ9G43Qv5x\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:39 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:40 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,267 of 2,267 bytes (100%) chr19_44892887_C_T/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361KjJYBPZyF8kGz853fBQ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:43 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:44 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 913,176 of 913,176 bytes (100%) chr19_44892887_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361Q0JYBPxZ2P2zV74QKV2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:48 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:48 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr19_44892887_C_T/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361V0JYBPgvP5PQVZ2vXX5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:52 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:53 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_FIN.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 690 of 690 bytes (100%) chr19_44892887_C_T/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361X0JYBPXpGg0V5232jKy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:48:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:48:57 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 9,847 of 9,847 bytes (100%) chr19_44892887_C_T/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361Y8JYBPqBv7bGBG8G41f\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:01 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:02 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_Control_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 690 of 690 bytes (100%) chr19_44892887_C_T/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361Z8JYBPgJBGp39F0pY09\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:05 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:06 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,030 of 1,030 bytes (100%) chr19_44892887_C_T/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361bQJYBPZyF8kGz853fFV\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:10 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 892 of 892 bytes (100%) chr19_44892887_C_T/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361fQJYBPXY98Bf9kBxB30\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:14 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 956 of 956 bytes (100%) chr19_44892887_C_T/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361gQJYBPXJfFXKqK9Q94B\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:18 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:19 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 621 of 621 bytes (100%) chr19_44892887_C_T/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361jQJYBPpyV38637F4Z1J\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:22 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:23 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 202 of 202 bytes (100%) chr19_44892887_C_T/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361kjJYBPgvP5PQVZ2vXYg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:27 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 141,831 of 141,831 bytes (100%) chr19_44892887_C_T/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361pjJYBPZyF8kGz853fG1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:31 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:32 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_FIN.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr19_44892887_C_T/apoe_final_RD_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361v0JYBPxZ2P2zV74QKZ3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:36 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:36 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_FIN.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr19_44892887_C_T/apoe_final_RD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361x0JYBPgvP5PQVZ2vXZ6\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:40 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:41 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44892887_C_T/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44892887_C_T/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,518 of 1,518 bytes (100%) chr19_44892887_C_T/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq361y0JYBPgjZj95jG5G11p\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:44 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:45 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44892887_C_T/apoe_final_RD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 291 of 291 bytes (100%) chr19_44905307_A_T/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq361z0JYBPYZ982v2vBgKvb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:48 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:49 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 939 of 939 bytes (100%) chr19_44905307_A_T/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq361zjJYBPXJfFXKqK9Q96g\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:52 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 264 of 264 bytes (100%) chr19_44905307_A_T/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq3620QJYBPqjGQ794yBpK9F\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:54 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr19_44905307_A_T/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq36210JYBPYZ982v2vBgKx1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:49:57 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 156 of 156 bytes (100%) chr19_44905307_A_T/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq3621jJYBPq0pKv7Y35PZ7X\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:49:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr19_44905307_A_T/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq3622QJYBPqjGQ794yBpK9b\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:02 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:02 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 24,312 of 24,312 bytes (100%) chr19_44905307_A_T/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq36230JYBPqjGQ794yBpKG5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:04 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 210 of 210 bytes (100%) chr19_44905307_A_T/apoe_final_AD_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq3623jJYBPgJBGp39F0pY47\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 399 of 399 bytes (100%) chr19_44905307_A_T/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq3624QJYBPqBv7bGBG8G45F\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:10 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,830 of 1,830 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq36250JYBPgvP5PQVZ2vXgY\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 6,195 of 6,195 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq3625jJYBPx9yYJ9G43QvGj\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:15 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 5,430 of 5,430 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq36268JYBPk6P6YFQzGfV5Y\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:17 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AMR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 534 of 534 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","ID                                file-Gq36270JYBPk6P6YFQzGfV5k\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,605 of 1,605 bytes (100%) chr19_44905307_A_T/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq3627jJYBPjYxjxz4g4gK3P\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:23 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:23 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,361 of 2,361 bytes (100%) chr19_44905307_A_T/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq36288JYBPqjGQ794yBpKGj\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:25 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:26 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 372 of 372 bytes (100%) chr19_44905307_A_T/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq36290JYBPjYxjxz4g4gK3b\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:28 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:29 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 461,703 of 461,703 bytes (100%) chr19_44905307_A_T/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq3629jJYBPxZ2P2zV74QKj1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:31 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:31 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_FIN.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 264 of 264 bytes (100%) chr19_44905307_A_T/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","ID                                file-Gq362B8JYBPZyF8kGz853gGx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 966 of 966 bytes (100%) chr19_44905307_A_T/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq362F0JYBPgJBGp39F0pY9V\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:36 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:36 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 6,204 of 6,204 bytes (100%) chr19_44905307_A_T/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq362FQJYBPgJBGp39F0pYB0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:38 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:39 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 507 of 507 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq362G8JYBPq0pKv7Y35PZBk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:41 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:42 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,074 of 1,074 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq362J0JYBPpyV38637F4ZG4\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:44 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:44 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 543 of 543 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq362JQJYBPpyV38637F4ZG9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:46 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:47 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AMR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AMR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AMR.APOE_GENOTYPES.csv\n","ID                                file-Gq362K8JYBPk6P6YFQzGfV8g\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AMR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:49 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:49 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 210 of 210 bytes (100%) chr19_44905307_A_T/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq362KjJYBPpyV38637F4ZP1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:52 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 318 of 318 bytes (100%) chr19_44905307_A_T/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq362PQJYBPjYxjxz4g4gK48\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr19_44905307_A_T/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq362Q8JYBPjYxjxz4g4gK4F\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:57 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:50:57 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 38,064 of 38,064 bytes (100%) chr19_44905307_A_T/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq362QjJYBPXJfFXKqK9Q9PK\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:50:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 264 of 264 bytes (100%) chr19_44905307_A_T/apoe_final_RD_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq362VQJYBPXY98Bf9kBxB9Q\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:02 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:03 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 588 of 588 bytes (100%) chr19_44905307_A_T/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq362X0JYBPx9yYJ9G43QvJ9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:04 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 759 of 759 bytes (100%) chr19_44905307_A_T/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362XjJYBPXpGg0V5232jfB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,331 of 2,331 bytes (100%) chr19_44905307_A_T/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362YjJYBPYZ982v2vBgP7b\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:11 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:12 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 690 of 690 bytes (100%) chr19_44905307_A_T/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362b0JYBPqBv7bGBG8G4JP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:16 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr19_44905307_A_T/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362f0JYBPgjZj95jG5G17Q\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 414 of 414 bytes (100%) chr19_44905307_A_T/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362g0JYBPq0pKv7Y35PZJx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:24 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr19_44905307_A_T/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362j8JYBPXJfFXKqK9Q9XX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:29 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:29 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 60,494 of 60,494 bytes (100%) chr19_44905307_A_T/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362k8JYBPq0pKv7Y35PZK1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 552 of 552 bytes (100%) chr19_44905307_A_T/apoe_final_AD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362p8JYBPgvP5PQVZ2vXyf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:37 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:38 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,035 of 1,035 bytes (100%) chr19_44905307_A_T/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362q8JYBPx9yYJ9G43QvQ3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:41 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:42 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_AD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 4,524 of 4,524 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362vQJYBPxZ2P2zV74QP7Z\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:46 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:46 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 15,371 of 15,371 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362xQJYBPXY98Bf9kBxBG7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:50 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:51 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 13,696 of 13,696 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362yQJYBPx9yYJ9G43QvQZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,380 of 1,380 bytes (100%) chr19_44905307_A_T/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq362zjJYBPq0pKv7Y35PZP7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:51:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:51:59 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_AMR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 4,061 of 4,061 bytes (100%) chr19_44905307_A_T/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3630jJYBPXJfFXKqK9Q9fB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:03 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:04 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 5,993 of 5,993 bytes (100%) chr19_44905307_A_T/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3631jJYBPgjZj95jG5G187\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 966 of 966 bytes (100%) chr19_44905307_A_T/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36330JYBPYZ982v2vBgPG1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:12 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,158,979 of 1,158,979 bytes (100%) chr19_44905307_A_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36340JYBPqjGQ794yBpKKv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:17 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 690 of 690 bytes (100%) chr19_44905307_A_T/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36350JYBPgvP5PQVZ2vY1Q\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_FIN.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,484 of 2,484 bytes (100%) chr19_44905307_A_T/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36360JYBPx9yYJ9G43QvQq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:24 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 15,786 of 15,786 bytes (100%) chr19_44905307_A_T/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36378JYBPqBv7bGBG8G4KF\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:29 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_Control_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,227 of 1,227 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36388JYBPXpGg0V5232jg2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,676 of 2,676 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3639QJYBPq0pKv7Y35PZQK\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:38 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:38 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,375 of 1,375 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363BQJYBPgjZj95jG5G1FP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:42 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:43 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_AMR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr19_44905307_A_T/apoe_final_RD_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363FQJYBPq0pKv7Y35PZQf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:46 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:47 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_AMR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 552 of 552 bytes (100%) chr19_44905307_A_T/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363GQJYBPqjGQ794yBpKPb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:50 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:51 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 828 of 828 bytes (100%) chr19_44905307_A_T/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363JjJYBPgjZj95jG5G1G7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:55 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:52:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr19_44905307_A_T/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363KjJYBPXY98Bf9kBxBP7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:52:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 94,350 of 94,350 bytes (100%) chr19_44905307_A_T/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363PjJYBPx9yYJ9G43QvV4\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:03 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:04 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 690 of 690 bytes (100%) chr19_44905307_A_T/apoe_final_RD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363V0JYBPZyF8kGz853gyz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:08 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44905307_A_T/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44905307_A_T/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,518 of 1,518 bytes (100%) chr19_44905307_A_T/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363X0JYBPx9yYJ9G43QvV9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44905307_A_T/apoe_final_RD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr19_44908756_C_A/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44908756_C_A/apoe_final_Control_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 156 of 156 bytes (100%) chr19_44908756_C_A/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq363Y0JYBPx9yYJ9G43QvVb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44908756_C_A\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:17 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr19_44908756_C_A/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr19_44908756_C_A/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 414 of 414 bytes (100%) chr19_44908756_C_A/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363YjJYBPZyF8kGz853gz8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44908756_C_A\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:19 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:20 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr19_44908756_C_A/apoe_final_Control_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_25897620_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_25897620_C_T/apoe_final_Control_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_25897620_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq363ZjJYBPqBv7bGBG8G4P9\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_25897620_C_T\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:23 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:24 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_25897620_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_25897620_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_25897620_C_T/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq363bQJYBPxZ2P2zV74QPQ0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_25897620_C_T\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_25897620_C_T/apoe_final_Control_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr21_26171645_A_G/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq363fQJYBPxZ2P2zV74QPQz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:30 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:31 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 480 of 480 bytes (100%) chr21_26171645_A_G/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq363g8JYBPXpGg0V5232jp8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171645_A_G/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq363j0JYBPgJBGp39F0pYjy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:36 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:36 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr21_26171645_A_G/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq363jQJYBPXY98Bf9kBxBYF\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:38 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:39 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171645_A_G/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq363k8JYBPx9yYJ9G43Qvfv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:41 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:41 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171645_A_G/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq363kjJYBPZyF8kGz853gzq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:43 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:44 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 4,854 of 4,854 bytes (100%) chr21_26171645_A_G/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq363pQJYBPgJBGp39F0pYk2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:46 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:47 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171645_A_G/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq363q8JYBPZyF8kGz853gzy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:49 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:50 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,029 of 1,029 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq363qjJYBPXpGg0V5232jpp\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:52 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 4,332 of 4,332 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq363vQJYBPXpGg0V5232jq2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq363x8JYBPxZ2P2zV74QPX0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:57 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:53:58 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AMR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","ID                                file-Gq363xjJYBPXY98Bf9kBxBfG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:53:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 795 of 795 bytes (100%) chr21_26171645_A_G/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq363yQJYBPx9yYJ9G43QvgX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:02 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:03 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 750 of 750 bytes (100%) chr21_26171645_A_G/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq363z8JYBPXpGg0V5232jqv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:05 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,542 of 1,542 bytes (100%) chr21_26171645_A_G/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq363zjJYBPgJBGp39F0pYkb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 67,908 of 67,908 bytes (100%) chr21_26171645_A_G/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq3640QJYBPgvP5PQVZ2vYJ8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:11 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_FIN.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171645_A_G/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","ID                                file-Gq36418JYBPxZ2P2zV74QPXY\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171645_A_G/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq3641jJYBPXY98Bf9kBxBfg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:15 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:16 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,676 of 2,676 bytes (100%) chr21_26171645_A_G/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq3642QJYBPgvP5PQVZ2vYJX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:18 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:19 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 273 of 273 bytes (100%) chr21_26171645_A_G/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq36430JYBPZyF8kGz853j1z\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 714 of 714 bytes (100%) chr21_26171645_A_G/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq3643jJYBPx9yYJ9G43Qvj0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:23 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:24 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171645_A_G/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq3644QJYBPZyF8kGz853j23\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:26 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 246 of 246 bytes (100%) chr21_26171645_A_G/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq36450JYBPZyF8kGz853j25\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:28 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:29 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 210 of 210 bytes (100%) chr21_26171645_A_G/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq3645jJYBPqBv7bGBG8G4V8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:31 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:32 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 156 of 156 bytes (100%) chr21_26171645_A_G/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq3646QJYBPgJBGp39F0pYpP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:34 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 6,744 of 6,744 bytes (100%) chr21_26171645_A_G/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq36470JYBPZyF8kGz853j4Y\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:36 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:37 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171645_A_G/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq3647jJYBPXY98Bf9kBxBg6\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:39 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:40 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr21_26171645_A_G/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3648QJYBPZyF8kGz853j4j\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:42 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:42 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,158 of 1,158 bytes (100%) chr21_26171645_A_G/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3649QJYBPgJBGp39F0pYq5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:46 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:47 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171645_A_G/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364BQJYBPgJBGp39F0pYqf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:50 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:51 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr21_26171645_A_G/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364FjJYBPgvP5PQVZ2vZ6Q\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:55 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:54:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171645_A_G/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364GjJYBPXpGg0V5232jx8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:54:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171645_A_G/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364JjJYBPqBv7bGBG8G4jG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:03 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:04 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 12,336 of 12,336 bytes (100%) chr21_26171645_A_G/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364KjJYBPXpGg0V5232jxB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171645_A_G/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364Q0JYBPx9yYJ9G43QvvZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_AD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,533 of 2,533 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364V0JYBPgJBGp39F0pYzB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:17 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 10,778 of 10,778 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364X0JYBPXpGg0V5232jxj\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364Y8JYBPgvP5PQVZ2vZGy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:25 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171645_A_G/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364Z8JYBPgvP5PQVZ2vZJ7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:29 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_AMR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,991 of 1,991 bytes (100%) chr21_26171645_A_G/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364b8JYBPXpGg0V5232jyQ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:34 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,932 of 1,932 bytes (100%) chr21_26171645_A_G/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364fQJYBPXY98Bf9kBxBz5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:38 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:39 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 3,928 of 3,928 bytes (100%) chr21_26171645_A_G/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364gQJYBPqBv7bGBG8G4pZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:42 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:43 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 172,102 of 172,102 bytes (100%) chr21_26171645_A_G/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364jjJYBPxZ2P2zV74QPjZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:47 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:47 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171645_A_G/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364kjJYBPxZ2P2zV74QPjf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:52 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_FIN.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171645_A_G/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364pjJYBPgvP5PQVZ2vZXk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:55:55 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:55:56 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 6,826 of 6,826 bytes (100%) chr21_26171645_A_G/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364v0JYBPxZ2P2zV74QPjy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:00 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_Control_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 685 of 685 bytes (100%) chr21_26171645_A_G/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364x0JYBPgvP5PQVZ2vZXz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:04 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,784 of 1,784 bytes (100%) chr21_26171645_A_G/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364y0JYBPgvP5PQVZ2vZY1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:08 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:09 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171645_A_G/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq364z0JYBPxZ2P2zV74QPk3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 616 of 616 bytes (100%) chr21_26171645_A_G/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36508JYBPxZ2P2zV74QPk6\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:17 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:17 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 552 of 552 bytes (100%) chr21_26171645_A_G/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36518JYBPx9yYJ9G43Qx3G\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:21 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:22 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 414 of 414 bytes (100%) chr21_26171645_A_G/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3652QJYBPxZ2P2zV74QPkB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:26 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 17,082 of 17,082 bytes (100%) chr21_26171645_A_G/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3653QJYBPgvP5PQVZ2vZfP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:30 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:31 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171645_A_G/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171645_A_G/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 897 of 897 bytes (100%) chr21_26171645_A_G/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3654QJYBPqBv7bGBG8G4zB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:34 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:35 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171645_A_G/apoe_final_RD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr21_26171723_T_C/apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq3655jJYBPgvP5PQVZ2vZk3\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:39 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:39 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 480 of 480 bytes (100%) chr21_26171723_T_C/apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq36568JYBPZyF8kGz853jYG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:41 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:42 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171723_T_C/apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq36570JYBPqBv7bGBG8G4zK\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:44 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:45 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr21_26171723_T_C/apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq3657jJYBPxZ2P2zV74QPkf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:47 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:47 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171723_T_C/apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq36588JYBPgvP5PQVZ2vZkP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:49 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:50 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171723_T_C/apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq36590JYBPgvP5PQVZ2vZkb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:52 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:53 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 4,908 of 4,908 bytes (100%) chr21_26171723_T_C/apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq3659QJYBPx9yYJ9G43Qx69\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171723_T_C/apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq365B8JYBPgvP5PQVZ2vZkk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:56:57 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:56:58 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,029 of 1,029 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq365F0JYBPgJBGp39F0pZ9q\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:00 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 4,332 of 4,332 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq365FQJYBPXpGg0V5232k38\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:02 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:03 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq365G8JYBPXY98Bf9kBxF1v\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:05 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:06 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AMR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 129 of 129 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","ID                                file-Gq365J0JYBPqBv7bGBG8G6Xx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:08 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 822 of 822 bytes (100%) chr21_26171723_T_C/apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq365JQJYBPxZ2P2zV74QPpY\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:11 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 750 of 750 bytes (100%) chr21_26171723_T_C/apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq365K8JYBPgvP5PQVZ2vZpV\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:14 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,542 of 1,542 bytes (100%) chr21_26171723_T_C/apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq365P0JYBPqBv7bGBG8G6Yk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:16 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 68,016 of 68,016 bytes (100%) chr21_26171723_T_C/apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq365PQJYBPZyF8kGz853jb1\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:18 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:19 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_FIN.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171723_T_C/apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","ID                                file-Gq365Q8JYBPgvP5PQVZ2vZqG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:21 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:22 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_MDE.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 102 of 102 bytes (100%) chr21_26171723_T_C/apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","ID                                file-Gq365V0JYBPqBv7bGBG8G6ZZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:24 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,676 of 2,676 bytes (100%) chr21_26171723_T_C/apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq365VQJYBPXpGg0V5232k4K\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_AAC.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 300 of 300 bytes (100%) chr21_26171723_T_C/apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","ID                                file-Gq365X8JYBPQZ5kqBjv42Z0G\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:29 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_AFR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 741 of 741 bytes (100%) chr21_26171723_T_C/apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","ID                                file-Gq365Y0JYBPqBv7bGBG8G6b0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:32 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:33 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_AJ.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 75 of 75 bytes (100%) chr21_26171723_T_C/apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","ID                                file-Gq365YjJYBPXpGg0V5232k53\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:35 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:35 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_CAH.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 246 of 246 bytes (100%) chr21_26171723_T_C/apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","ID                                file-Gq365Z8JYBPgvP5PQVZ2vZv4\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:37 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:38 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_CAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 210 of 210 bytes (100%) chr21_26171723_T_C/apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","ID                                file-Gq365b0JYBPYKxfQzZX7J9fq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:40 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:41 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_EAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 156 of 156 bytes (100%) chr21_26171723_T_C/apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","ID                                file-Gq365bjJYBPqG0JVbz04k998\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:43 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:43 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_EUR.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 6,771 of 6,771 bytes (100%) chr21_26171723_T_C/apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","ID                                file-Gq365f8JYBPbQG3QbP4GX2Z2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:45 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:46 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_SAS.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171723_T_C/apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","ID                                file-Gq365g0JYBPQZ5kqBjv42Z19\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:48 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:49 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr21_26171723_T_C/apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365gQJYBPVp3BJz3bPBJzx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:50 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:51 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,158 of 1,158 bytes (100%) chr21_26171723_T_C/apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365jjJYBPfKQBpvvFBjk5K\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:55 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:57:56 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171723_T_C/apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365kjJYBPqG0JVbz04k99q\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:57:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr21_26171723_T_C/apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365pjJYBPzv4k1f1p3kxYB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:03 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:04 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171723_T_C/apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365v0JYBPbQG3QbP4GX2bB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:08 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:09 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171723_T_C/apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365x0JYBPVp3BJz3bPBK10\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 12,474 of 12,474 bytes (100%) chr21_26171723_T_C/apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365y0JYBPfKQBpvvFBjk6b\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:16 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:17 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171723_T_C/apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq365z8JYBPVp3BJz3bPBK1f\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_AD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:21 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:22 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_AD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,533 of 2,533 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36608JYBPfKQBpvvFBjk9k\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:25 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:26 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 10,778 of 10,778 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36618JYBPzv4k1f1p3kxg0\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:29 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3662QJYBPbQG3QbP4GX2jv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:34 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:35 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 345 of 345 bytes (100%) chr21_26171723_T_C/apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3663QJYBPQZ5kqBjv42g2K\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_AMR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:38 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:39 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_AMR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 2,060 of 2,060 bytes (100%) chr21_26171723_T_C/apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3664jJYBPfKQBpvvFBjkJV\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:43 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:43 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,932 of 1,932 bytes (100%) chr21_26171723_T_C/apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3665jJYBPbxbx6kqY1j4qq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:47 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:48 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 3,928 of 3,928 bytes (100%) chr21_26171723_T_C/apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq3666jJYBPfKQBpvvFBjkQg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:52 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 172,378 of 172,378 bytes (100%) chr21_26171723_T_C/apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36680JYBPbQG3QbP4GX2pz\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:58:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:58:57 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171723_T_C/apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq36690JYBPYKxfQzZX7J9y6\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_FIN.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:00 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:01 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_FIN.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 276 of 276 bytes (100%) chr21_26171723_T_C/apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366B8JYBPYKxfQzZX7J9yK\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_MDE.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:05 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_MDE.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 6,826 of 6,826 bytes (100%) chr21_26171723_T_C/apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366F8JYBPVp3BJz3bPBK4y\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_Control_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:09 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:10 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_Control_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 754 of 754 bytes (100%) chr21_26171723_T_C/apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366G8JYBPzv4k1f1p3kxpG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_AAC.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:14 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_AAC.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 1,853 of 1,853 bytes (100%) chr21_26171723_T_C/apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366JQJYBPQZ5kqBjv437pZ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_AFR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:18 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_AFR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 207 of 207 bytes (100%) chr21_26171723_T_C/apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366KQJYBPqG0JVbz04kx3x\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_AJ.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:22 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:23 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_AJ.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 616 of 616 bytes (100%) chr21_26171723_T_C/apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366PQJYBPYKxfQzZX7J9z8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_CAH.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:27 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_CAH.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 552 of 552 bytes (100%) chr21_26171723_T_C/apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366QjJYBPbxbx6kqY1j4yK\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_CAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:31 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:32 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_CAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 414 of 414 bytes (100%) chr21_26171723_T_C/apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366VjJYBPqG0JVbz04kx4F\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_EAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:35 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:36 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_EAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 17,151 of 17,151 bytes (100%) chr21_26171723_T_C/apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366XjJYBPbxbx6kqY1j4yp\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_EUR.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:39 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:40 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_EUR.APOE_SUMMARY.APOE_GENOTYPES.csv'\n","Your complete genotype file has been saved here: chr21_26171723_T_C/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","The summary counts have been saved here: chr21_26171723_T_C/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_SUMMARY.csv\n","Thanks!\n","[===========================================================>] Uploaded 897 of 897 bytes (100%) chr21_26171723_T_C/apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","ID                                file-Gq366Z0JYBPfKQBpvvFBjkbP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              apoe_final_RD_SAS.APOE_GENOTYPES.APOE_GENOTYPES.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Wed Aug 21 21:59:44 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Wed Aug 21 21:59:44 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Traceback (most recent call last):\n","  File \"/opt/notebooks/APOE_genotypes_PLINK_ped.py\", line 39, in <module>\n","    input_ped_df = pd.read_csv(args.input, sep = \" \", header=None, names=header_text)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 211, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/util/_decorators.py\", line 331, in wrapper\n","    return func(*args, **kwargs)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 950, in read_csv\n","    return _read(filepath_or_buffer, kwds)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 611, in _read\n","    return parser.read(nrows)\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/readers.py\", line 1778, in read\n","    ) = self._engine.read(  # type: ignore[attr-defined]\n","  File \"/opt/conda/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 230, in read\n","    chunks = self._reader.read_low_memory(nrows)\n","  File \"pandas/_libs/parsers.pyx\", line 808, in pandas._libs.parsers.TextReader.read_low_memory\n","  File \"pandas/_libs/parsers.pyx\", line 890, in pandas._libs.parsers.TextReader._read_rows\n","  File \"pandas/_libs/parsers.pyx\", line 952, in pandas._libs.parsers.TextReader._convert_column_data\n","pandas.errors.ParserError: Too many columns specified: expected 8 and found 1\n","FileNotFoundError: [Errno 2] No such file or directory: 'chr21_26171723_T_C/apoe_final_RD_SAS.APOE_SUMMARY.APOE_GENOTYPES.csv'\n"]}],"source":["! dx download wgs_analysis/data/APOE_genotypes_PLINK_ped.py --overwrite\n","for var_id in [\"chr19_44892887_C_T\",\"chr19_44905307_A_T\",\"chr19_44908756_C_A\",\"chr21_25897620_C_T\",\"chr21_26171645_A_G\",\"chr21_26171723_T_C\"]:\n","    for ped_file in glob.glob(f\"{var_id}/*\"):\n","        ! python APOE_genotypes_PLINK_ped.py -i {ped_file} -o {ped_file.replace(\"snps\",\"final\")[:-4]}\n","        ! dx upload {ped_file.replace(\"snps\",\"final\")[:-4]}.APOE_GENOTYPES.csv --path wgs_analysis/results/12_protective_variants/{ped_file.replace(\"snps\",\"final\")[:-4]}.APOE_GENOTYPES.csv\n","        "]},{"cell_type":"code","execution_count":13,"id":"83c6865f-fd6a-4a13-a065-cf4f225753ce","metadata":{"collapsed":true,"jupyter":{"outputs_hidden":true},"tags":[],"trusted":true},"outputs":[{"name":"stdout","output_type":"stream","text":["chr19_44892887_C_T\n","AD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>5</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>78</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>85</td>\n","      <td>0</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>1</td>\n","      <td>4</td>\n","      <td>11</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>1457</td>\n","      <td>1</td>\n","      <td>19</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>3</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>520</td>\n","      <td>1</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>4</td>\n","      <td>9</td>\n","      <td>14</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>2</td>\n","      <td>2145</td>\n","      <td>2</td>\n","      <td>25</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  CAH  CAS  EAS   EUR  MDE  SAS\n","e1/e1             0    0   0    0    0    0     0    0    0\n","e1/e2             0    0   0    0    0    0     0    0    0\n","e1/e4             0    0   0    0    0    0     0    0    0\n","e2/e2             0    0   0    0    0    0     0    0    0\n","e2/e3             0    2   0    0    0    0     5    0    0\n","e2/e4 or e1/e3    0    2   0    0    0    0    78    0    0\n","e3/e3             0    0   1    0    0    0    85    0    3\n","e3/e4             1    4  11    1    2    2  1457    1   19\n","e4/e4             3    1   2    0    1    0   520    1    3\n","total             4    9  14    1    3    2  2145    2   25"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0.0</td>\n","      <td>22.222222</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.233100</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0.0</td>\n","      <td>22.222222</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>3.636364</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>7.142857</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>3.962704</td>\n","      <td>0.0</td>\n","      <td>12.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>25.0</td>\n","      <td>44.444444</td>\n","      <td>78.571429</td>\n","      <td>100.0</td>\n","      <td>66.666667</td>\n","      <td>100.0</td>\n","      <td>67.925408</td>\n","      <td>50.0</td>\n","      <td>76.0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>75.0</td>\n","      <td>11.111111</td>\n","      <td>14.285714</td>\n","      <td>0.0</td>\n","      <td>33.333333</td>\n","      <td>0.0</td>\n","      <td>24.242424</td>\n","      <td>50.0</td>\n","      <td>12.0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>4.0</td>\n","      <td>9.000000</td>\n","      <td>14.000000</td>\n","      <td>1.0</td>\n","      <td>3.000000</td>\n","      <td>2.0</td>\n","      <td>2145.000000</td>\n","      <td>2.0</td>\n","      <td>25.0</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                 AAC        AFR         AJ    CAH        CAS    EAS  \\\n","e1/e1            0.0   0.000000   0.000000    0.0   0.000000    0.0   \n","e1/e2            0.0   0.000000   0.000000    0.0   0.000000    0.0   \n","e1/e4            0.0   0.000000   0.000000    0.0   0.000000    0.0   \n","e2/e2            0.0   0.000000   0.000000    0.0   0.000000    0.0   \n","e2/e3            0.0  22.222222   0.000000    0.0   0.000000    0.0   \n","e2/e4 or e1/e3   0.0  22.222222   0.000000    0.0   0.000000    0.0   \n","e3/e3            0.0   0.000000   7.142857    0.0   0.000000    0.0   \n","e3/e4           25.0  44.444444  78.571429  100.0  66.666667  100.0   \n","e4/e4           75.0  11.111111  14.285714    0.0  33.333333    0.0   \n","total            4.0   9.000000  14.000000    1.0   3.000000    2.0   \n","\n","                        EUR   MDE   SAS  \n","e1/e1              0.000000   0.0   0.0  \n","e1/e2              0.000000   0.0   0.0  \n","e1/e4              0.000000   0.0   0.0  \n","e2/e2              0.000000   0.0   0.0  \n","e2/e3              0.233100   0.0   0.0  \n","e2/e4 or e1/e3     3.636364   0.0   0.0  \n","e3/e3              3.962704   0.0  12.0  \n","e3/e4             67.925408  50.0  76.0  \n","e4/e4             24.242424  50.0  12.0  \n","total           2145.000000   2.0  25.0  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>2 (22.22%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>5 (0.23%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>2 (22.22%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>78 (3.64%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (7.14%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>85 (3.96%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>3 (12.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>1 (25.00%)</td>\n","      <td>4 (44.44%)</td>\n","      <td>11 (78.57%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>2 (66.67%)</td>\n","      <td>2 (100.00%)</td>\n","      <td>1457 (67.93%)</td>\n","      <td>1 (50.00%)</td>\n","      <td>19 (76.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>3 (75.00%)</td>\n","      <td>1 (11.11%)</td>\n","      <td>2 (14.29%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (33.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>520 (24.24%)</td>\n","      <td>1 (50.00%)</td>\n","      <td>3 (12.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>4</td>\n","      <td>9</td>\n","      <td>14</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>2</td>\n","      <td>2145</td>\n","      <td>2</td>\n","      <td>25</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                       AAC         AFR           AJ          CAH         CAS  \\\n","e1/e1            0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   \n","e1/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   \n","e1/e4            0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   \n","e2/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   \n","e2/e3            0 (0.00%)  2 (22.22%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   \n","e2/e4 or e1/e3   0 (0.00%)  2 (22.22%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   \n","e3/e3            0 (0.00%)   0 (0.00%)    1 (7.14%)    0 (0.00%)   0 (0.00%)   \n","e3/e4           1 (25.00%)  4 (44.44%)  11 (78.57%)  1 (100.00%)  2 (66.67%)   \n","e4/e4           3 (75.00%)  1 (11.11%)   2 (14.29%)    0 (0.00%)  1 (33.33%)   \n","total                    4           9           14            1           3   \n","\n","                        EAS            EUR         MDE          SAS  \n","e1/e1             0 (0.00%)      0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e2             0 (0.00%)      0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e4             0 (0.00%)      0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e2/e2             0 (0.00%)      0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e2/e3             0 (0.00%)      5 (0.23%)   0 (0.00%)    0 (0.00%)  \n","e2/e4 or e1/e3    0 (0.00%)     78 (3.64%)   0 (0.00%)    0 (0.00%)  \n","e3/e3             0 (0.00%)     85 (3.96%)   0 (0.00%)   3 (12.00%)  \n","e3/e4           2 (100.00%)  1457 (67.93%)  1 (50.00%)  19 (76.00%)  \n","e4/e4             0 (0.00%)   520 (24.24%)  1 (50.00%)   3 (12.00%)  \n","total                     2           2145           2           25  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 299 of 299 bytes (100%) chr19_44892887_C_T/AD_apoe_genotype_counts.csv\n","ID                                file-Gq38548JYBPY9jV3VVF0111B\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:05 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:06 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 682 of 682 bytes (100%) chr19_44892887_C_T/AD_apoe_genotype_percentages.csv\n","ID                                file-Gq3854jJYBPjp8Z6qvZ1B8q5\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:07 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 968 of 968 bytes (100%) chr19_44892887_C_T/AD_apoe_genotype_combined.csv\n","ID                                file-Gq38558JYBPjp8Z6qvZ1B8q7\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              AD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:09 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:09 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","RD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>14</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>117</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>2</td>\n","      <td>6</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>186</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>2</td>\n","      <td>4</td>\n","      <td>9</td>\n","      <td>7</td>\n","      <td>5</td>\n","      <td>0</td>\n","      <td>1386</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>14</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>359</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>4</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>8</td>\n","      <td>13</td>\n","      <td>11</td>\n","      <td>12</td>\n","      <td>7</td>\n","      <td>1</td>\n","      <td>2062</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>20</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  CAH  CAS  EAS   EUR  FIN  MDE  SAS\n","e1/e1             0    0   0    0    0    0     0    0    0    0\n","e1/e2             0    0   0    0    0    0     0    0    0    0\n","e1/e4             0    0   0    0    0    0     0    0    0    0\n","e2/e2             0    0   0    0    0    0     0    0    0    0\n","e2/e3             1    2   0    0    1    0    14    0    0    1\n","e2/e4 or e1/e3    0    1   1    2    0    1   117    0    0    0\n","e3/e3             2    6   0    2    0    0   186    0    1    1\n","e3/e4             2    4   9    7    5    0  1386    1    0   14\n","e4/e4             3    0   1    1    1    0   359    0    0    4\n","total             8   13  11   12    7    1  2062    1    1   20"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>12.5</td>\n","      <td>15.384615</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>14.285714</td>\n","      <td>0.0</td>\n","      <td>0.678952</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>5.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0.0</td>\n","      <td>7.692308</td>\n","      <td>9.090909</td>\n","      <td>16.666667</td>\n","      <td>0.000000</td>\n","      <td>100.0</td>\n","      <td>5.674103</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>25.0</td>\n","      <td>46.153846</td>\n","      <td>0.000000</td>\n","      <td>16.666667</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>9.020369</td>\n","      <td>0.0</td>\n","      <td>100.0</td>\n","      <td>5.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>25.0</td>\n","      <td>30.769231</td>\n","      <td>81.818182</td>\n","      <td>58.333333</td>\n","      <td>71.428571</td>\n","      <td>0.0</td>\n","      <td>67.216295</td>\n","      <td>100.0</td>\n","      <td>0.0</td>\n","      <td>70.0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>37.5</td>\n","      <td>0.000000</td>\n","      <td>9.090909</td>\n","      <td>8.333333</td>\n","      <td>14.285714</td>\n","      <td>0.0</td>\n","      <td>17.410281</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>20.0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>8.0</td>\n","      <td>13.000000</td>\n","      <td>11.000000</td>\n","      <td>12.000000</td>\n","      <td>7.000000</td>\n","      <td>1.0</td>\n","      <td>2062.000000</td>\n","      <td>1.0</td>\n","      <td>1.0</td>\n","      <td>20.0</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                 AAC        AFR         AJ        CAH        CAS    EAS  \\\n","e1/e1            0.0   0.000000   0.000000   0.000000   0.000000    0.0   \n","e1/e2            0.0   0.000000   0.000000   0.000000   0.000000    0.0   \n","e1/e4            0.0   0.000000   0.000000   0.000000   0.000000    0.0   \n","e2/e2            0.0   0.000000   0.000000   0.000000   0.000000    0.0   \n","e2/e3           12.5  15.384615   0.000000   0.000000  14.285714    0.0   \n","e2/e4 or e1/e3   0.0   7.692308   9.090909  16.666667   0.000000  100.0   \n","e3/e3           25.0  46.153846   0.000000  16.666667   0.000000    0.0   \n","e3/e4           25.0  30.769231  81.818182  58.333333  71.428571    0.0   \n","e4/e4           37.5   0.000000   9.090909   8.333333  14.285714    0.0   \n","total            8.0  13.000000  11.000000  12.000000   7.000000    1.0   \n","\n","                        EUR    FIN    MDE   SAS  \n","e1/e1              0.000000    0.0    0.0   0.0  \n","e1/e2              0.000000    0.0    0.0   0.0  \n","e1/e4              0.000000    0.0    0.0   0.0  \n","e2/e2              0.000000    0.0    0.0   0.0  \n","e2/e3              0.678952    0.0    0.0   5.0  \n","e2/e4 or e1/e3     5.674103    0.0    0.0   0.0  \n","e3/e3              9.020369    0.0  100.0   5.0  \n","e3/e4             67.216295  100.0    0.0  70.0  \n","e4/e4             17.410281    0.0    0.0  20.0  \n","total           2062.000000    1.0    1.0  20.0  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1 (12.50%)</td>\n","      <td>2 (15.38%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (14.29%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>14 (0.68%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (5.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>1 (7.69%)</td>\n","      <td>1 (9.09%)</td>\n","      <td>2 (16.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>117 (5.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>2 (25.00%)</td>\n","      <td>6 (46.15%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (16.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>186 (9.02%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>1 (5.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>2 (25.00%)</td>\n","      <td>4 (30.77%)</td>\n","      <td>9 (81.82%)</td>\n","      <td>7 (58.33%)</td>\n","      <td>5 (71.43%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1386 (67.22%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>14 (70.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>3 (37.50%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (9.09%)</td>\n","      <td>1 (8.33%)</td>\n","      <td>1 (14.29%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>359 (17.41%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>4 (20.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>8</td>\n","      <td>13</td>\n","      <td>11</td>\n","      <td>12</td>\n","      <td>7</td>\n","      <td>1</td>\n","      <td>2062</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>20</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                       AAC         AFR          AJ         CAH         CAS  \\\n","e1/e1            0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e2            0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e4            0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e2            0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e3           1 (12.50%)  2 (15.38%)   0 (0.00%)   0 (0.00%)  1 (14.29%)   \n","e2/e4 or e1/e3   0 (0.00%)   1 (7.69%)   1 (9.09%)  2 (16.67%)   0 (0.00%)   \n","e3/e3           2 (25.00%)  6 (46.15%)   0 (0.00%)  2 (16.67%)   0 (0.00%)   \n","e3/e4           2 (25.00%)  4 (30.77%)  9 (81.82%)  7 (58.33%)  5 (71.43%)   \n","e4/e4           3 (37.50%)   0 (0.00%)   1 (9.09%)   1 (8.33%)  1 (14.29%)   \n","total                    8          13          11          12           7   \n","\n","                        EAS            EUR          FIN          MDE  \\\n","e1/e1             0 (0.00%)      0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e2             0 (0.00%)      0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e4             0 (0.00%)      0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e2/e2             0 (0.00%)      0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e2/e3             0 (0.00%)     14 (0.68%)    0 (0.00%)    0 (0.00%)   \n","e2/e4 or e1/e3  1 (100.00%)    117 (5.67%)    0 (0.00%)    0 (0.00%)   \n","e3/e3             0 (0.00%)    186 (9.02%)    0 (0.00%)  1 (100.00%)   \n","e3/e4             0 (0.00%)  1386 (67.22%)  1 (100.00%)    0 (0.00%)   \n","e4/e4             0 (0.00%)   359 (17.41%)    0 (0.00%)    0 (0.00%)   \n","total                     1           2062            1            1   \n","\n","                        SAS  \n","e1/e1             0 (0.00%)  \n","e1/e2             0 (0.00%)  \n","e1/e4             0 (0.00%)  \n","e2/e2             0 (0.00%)  \n","e2/e3             1 (5.00%)  \n","e2/e4 or e1/e3    0 (0.00%)  \n","e3/e3             1 (5.00%)  \n","e3/e4           14 (70.00%)  \n","e4/e4            4 (20.00%)  \n","total                    20  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 327 of 327 bytes (100%) chr19_44892887_C_T/RD_apoe_genotype_counts.csv\n","ID                                file-Gq3855QJYBPzJ0b43Z9BKJ05\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:10 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:11 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 803 of 803 bytes (100%) chr19_44892887_C_T/RD_apoe_genotype_percentages.csv\n","ID                                file-Gq38560JYBPjp8Z6qvZ1B8qX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:12 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:13 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,071 of 1,071 bytes (100%) chr19_44892887_C_T/RD_apoe_genotype_combined.csv\n","ID                                file-Gq3856QJYBPjp8Z6qvZ1B8qf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              RD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:14 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Control\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>5</td>\n","      <td>12</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>242</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>4</td>\n","      <td>7</td>\n","      <td>6</td>\n","      <td>0</td>\n","      <td>6</td>\n","      <td>3</td>\n","      <td>2</td>\n","      <td>960</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>4</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>7</td>\n","      <td>39</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>12</td>\n","      <td>8</td>\n","      <td>5</td>\n","      <td>2322</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>59</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>11</td>\n","      <td>23</td>\n","      <td>48</td>\n","      <td>8</td>\n","      <td>21</td>\n","      <td>22</td>\n","      <td>23</td>\n","      <td>8913</td>\n","      <td>1</td>\n","      <td>4</td>\n","      <td>69</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>4</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>865</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>9</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>29</td>\n","      <td>84</td>\n","      <td>79</td>\n","      <td>8</td>\n","      <td>44</td>\n","      <td>33</td>\n","      <td>31</td>\n","      <td>13302</td>\n","      <td>2</td>\n","      <td>8</td>\n","      <td>141</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  AMR  CAH  CAS  EAS    EUR  FIN  MDE  SAS\n","e1/e1             0    0   0    0    0    0    0      0    0    0    0\n","e1/e2             0    0   0    0    0    0    0      0    0    0    0\n","e1/e4             0    0   0    0    0    0    0      0    0    0    0\n","e2/e2             0    1   0    0    0    0    0      0    0    0    0\n","e2/e3             5   12   3    0    1    0    0    242    0    0    0\n","e2/e4 or e1/e3    4    7   6    0    6    3    2    960    0    0    4\n","e3/e3             7   39  19    0   12    8    5   2322    1    3   59\n","e3/e4            11   23  48    8   21   22   23   8913    1    4   69\n","e4/e4             2    2   3    0    4    0    1    865    0    1    9\n","total            29   84  79    8   44   33   31  13302    2    8  141"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.000000</td>\n","      <td>1.190476</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>17.241379</td>\n","      <td>14.285714</td>\n","      <td>3.797468</td>\n","      <td>0.0</td>\n","      <td>2.272727</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>1.819275</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>13.793103</td>\n","      <td>8.333333</td>\n","      <td>7.594937</td>\n","      <td>0.0</td>\n","      <td>13.636364</td>\n","      <td>9.090909</td>\n","      <td>6.451613</td>\n","      <td>7.216960</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>2.836879</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>24.137931</td>\n","      <td>46.428571</td>\n","      <td>24.050633</td>\n","      <td>0.0</td>\n","      <td>27.272727</td>\n","      <td>24.242424</td>\n","      <td>16.129032</td>\n","      <td>17.456022</td>\n","      <td>50.0</td>\n","      <td>37.5</td>\n","      <td>41.843972</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>37.931034</td>\n","      <td>27.380952</td>\n","      <td>60.759494</td>\n","      <td>100.0</td>\n","      <td>47.727273</td>\n","      <td>66.666667</td>\n","      <td>74.193548</td>\n","      <td>67.004962</td>\n","      <td>50.0</td>\n","      <td>50.0</td>\n","      <td>48.936170</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>6.896552</td>\n","      <td>2.380952</td>\n","      <td>3.797468</td>\n","      <td>0.0</td>\n","      <td>9.090909</td>\n","      <td>0.000000</td>\n","      <td>3.225806</td>\n","      <td>6.502782</td>\n","      <td>0.0</td>\n","      <td>12.5</td>\n","      <td>6.382979</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>29.000000</td>\n","      <td>84.000000</td>\n","      <td>79.000000</td>\n","      <td>8.0</td>\n","      <td>44.000000</td>\n","      <td>33.000000</td>\n","      <td>31.000000</td>\n","      <td>13302.000000</td>\n","      <td>2.0</td>\n","      <td>8.0</td>\n","      <td>141.000000</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                      AAC        AFR         AJ    AMR        CAH        CAS  \\\n","e1/e1            0.000000   0.000000   0.000000    0.0   0.000000   0.000000   \n","e1/e2            0.000000   0.000000   0.000000    0.0   0.000000   0.000000   \n","e1/e4            0.000000   0.000000   0.000000    0.0   0.000000   0.000000   \n","e2/e2            0.000000   1.190476   0.000000    0.0   0.000000   0.000000   \n","e2/e3           17.241379  14.285714   3.797468    0.0   2.272727   0.000000   \n","e2/e4 or e1/e3  13.793103   8.333333   7.594937    0.0  13.636364   9.090909   \n","e3/e3           24.137931  46.428571  24.050633    0.0  27.272727  24.242424   \n","e3/e4           37.931034  27.380952  60.759494  100.0  47.727273  66.666667   \n","e4/e4            6.896552   2.380952   3.797468    0.0   9.090909   0.000000   \n","total           29.000000  84.000000  79.000000    8.0  44.000000  33.000000   \n","\n","                      EAS           EUR   FIN   MDE         SAS  \n","e1/e1            0.000000      0.000000   0.0   0.0    0.000000  \n","e1/e2            0.000000      0.000000   0.0   0.0    0.000000  \n","e1/e4            0.000000      0.000000   0.0   0.0    0.000000  \n","e2/e2            0.000000      0.000000   0.0   0.0    0.000000  \n","e2/e3            0.000000      1.819275   0.0   0.0    0.000000  \n","e2/e4 or e1/e3   6.451613      7.216960   0.0   0.0    2.836879  \n","e3/e3           16.129032     17.456022  50.0  37.5   41.843972  \n","e3/e4           74.193548     67.004962  50.0  50.0   48.936170  \n","e4/e4            3.225806      6.502782   0.0  12.5    6.382979  \n","total           31.000000  13302.000000   2.0   8.0  141.000000  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>1 (1.19%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>5 (17.24%)</td>\n","      <td>12 (14.29%)</td>\n","      <td>3 (3.80%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (2.27%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>242 (1.82%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>4 (13.79%)</td>\n","      <td>7 (8.33%)</td>\n","      <td>6 (7.59%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>6 (13.64%)</td>\n","      <td>3 (9.09%)</td>\n","      <td>2 (6.45%)</td>\n","      <td>960 (7.22%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>4 (2.84%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>7 (24.14%)</td>\n","      <td>39 (46.43%)</td>\n","      <td>19 (24.05%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>12 (27.27%)</td>\n","      <td>8 (24.24%)</td>\n","      <td>5 (16.13%)</td>\n","      <td>2322 (17.46%)</td>\n","      <td>1 (50.00%)</td>\n","      <td>3 (37.50%)</td>\n","      <td>59 (41.84%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>11 (37.93%)</td>\n","      <td>23 (27.38%)</td>\n","      <td>48 (60.76%)</td>\n","      <td>8 (100.00%)</td>\n","      <td>21 (47.73%)</td>\n","      <td>22 (66.67%)</td>\n","      <td>23 (74.19%)</td>\n","      <td>8913 (67.00%)</td>\n","      <td>1 (50.00%)</td>\n","      <td>4 (50.00%)</td>\n","      <td>69 (48.94%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2 (6.90%)</td>\n","      <td>2 (2.38%)</td>\n","      <td>3 (3.80%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>4 (9.09%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (3.23%)</td>\n","      <td>865 (6.50%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (12.50%)</td>\n","      <td>9 (6.38%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>29</td>\n","      <td>84</td>\n","      <td>79</td>\n","      <td>8</td>\n","      <td>44</td>\n","      <td>33</td>\n","      <td>31</td>\n","      <td>13302</td>\n","      <td>2</td>\n","      <td>8</td>\n","      <td>141</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                        AAC          AFR           AJ          AMR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e2/e2             0 (0.00%)    1 (1.19%)    0 (0.00%)    0 (0.00%)   \n","e2/e3            5 (17.24%)  12 (14.29%)    3 (3.80%)    0 (0.00%)   \n","e2/e4 or e1/e3   4 (13.79%)    7 (8.33%)    6 (7.59%)    0 (0.00%)   \n","e3/e3            7 (24.14%)  39 (46.43%)  19 (24.05%)    0 (0.00%)   \n","e3/e4           11 (37.93%)  23 (27.38%)  48 (60.76%)  8 (100.00%)   \n","e4/e4             2 (6.90%)    2 (2.38%)    3 (3.80%)    0 (0.00%)   \n","total                    29           84           79            8   \n","\n","                        CAH          CAS          EAS            EUR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e2/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e2/e3             1 (2.27%)    0 (0.00%)    0 (0.00%)    242 (1.82%)   \n","e2/e4 or e1/e3   6 (13.64%)    3 (9.09%)    2 (6.45%)    960 (7.22%)   \n","e3/e3           12 (27.27%)   8 (24.24%)   5 (16.13%)  2322 (17.46%)   \n","e3/e4           21 (47.73%)  22 (66.67%)  23 (74.19%)  8913 (67.00%)   \n","e4/e4             4 (9.09%)    0 (0.00%)    1 (3.23%)    865 (6.50%)   \n","total                    44           33           31          13302   \n","\n","                       FIN         MDE          SAS  \n","e1/e1            0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e4            0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e2/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e2/e3            0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e2/e4 or e1/e3   0 (0.00%)   0 (0.00%)    4 (2.84%)  \n","e3/e3           1 (50.00%)  3 (37.50%)  59 (41.84%)  \n","e3/e4           1 (50.00%)  4 (50.00%)  69 (48.94%)  \n","e4/e4            0 (0.00%)  1 (12.50%)    9 (6.38%)  \n","total                    2           8          141  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 369 of 369 bytes (100%) chr19_44892887_C_T/Control_apoe_genotype_counts.csv\n","ID                                file-Gq3856jJYBPgp85fYz93yVBK\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              Control_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:15 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:16 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,100 of 1,100 bytes (100%) chr19_44892887_C_T/Control_apoe_genotype_percentages.csv\n","ID                                file-Gq38578JYBPjp8Z6qvZ1B8qy\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              Control_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:17 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:18 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,188 of 1,188 bytes (100%) chr19_44892887_C_T/Control_apoe_genotype_combined.csv\n","ID                                file-Gq3857jJYBPY9jV3VVF01124\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44892887_C_T\n","Name                              Control_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:19 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:19 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr19_44905307_A_T\n","AD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>8</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>141</td>\n","      <td>0</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>59</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1</td>\n","      <td>7</td>\n","      <td>3</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>367</td>\n","      <td>4</td>\n","      <td>10</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>2</td>\n","      <td>12</td>\n","      <td>5</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>298</td>\n","      <td>1</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>5</td>\n","      <td>10</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>6</td>\n","      <td>1</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>9</td>\n","      <td>32</td>\n","      <td>8</td>\n","      <td>3</td>\n","      <td>4</td>\n","      <td>1</td>\n","      <td>879</td>\n","      <td>6</td>\n","      <td>13</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  CAH  CAS  EAS  EUR  MDE  SAS\n","e1/e1             0    0   0    0    0    0    0    0    0\n","e1/e2             0    0   0    0    0    0    0    0    0\n","e1/e4             0    0   0    0    0    0    0    0    0\n","e2/e2             0    0   0    0    0    0    8    0    0\n","e2/e3             1    0   0    0    0    0  141    0    1\n","e2/e4 or e1/e3    0    3   0    0    0    0   59    0    0\n","e3/e3             1    7   3    2    1    1  367    4   10\n","e3/e4             2   12   5    1    3    0  298    1    2\n","e4/e4             5   10   0    0    0    0    6    1    0\n","total             9   32   8    3    4    1  879    6   13"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.000000</td>\n","      <td>0.000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.000000</td>\n","      <td>0.000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.000000</td>\n","      <td>0.000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.000000</td>\n","      <td>0.000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.910125</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>11.111111</td>\n","      <td>0.000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>16.040956</td>\n","      <td>0.000000</td>\n","      <td>7.692308</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0.000000</td>\n","      <td>9.375</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>6.712173</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>11.111111</td>\n","      <td>21.875</td>\n","      <td>37.5</td>\n","      <td>66.666667</td>\n","      <td>25.0</td>\n","      <td>100.0</td>\n","      <td>41.751991</td>\n","      <td>66.666667</td>\n","      <td>76.923077</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>22.222222</td>\n","      <td>37.500</td>\n","      <td>62.5</td>\n","      <td>33.333333</td>\n","      <td>75.0</td>\n","      <td>0.0</td>\n","      <td>33.902162</td>\n","      <td>16.666667</td>\n","      <td>15.384615</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>55.555556</td>\n","      <td>31.250</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.682594</td>\n","      <td>16.666667</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>9.000000</td>\n","      <td>32.000</td>\n","      <td>8.0</td>\n","      <td>3.000000</td>\n","      <td>4.0</td>\n","      <td>1.0</td>\n","      <td>879.000000</td>\n","      <td>6.000000</td>\n","      <td>13.000000</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                      AAC     AFR    AJ        CAH   CAS    EAS         EUR  \\\n","e1/e1            0.000000   0.000   0.0   0.000000   0.0    0.0    0.000000   \n","e1/e2            0.000000   0.000   0.0   0.000000   0.0    0.0    0.000000   \n","e1/e4            0.000000   0.000   0.0   0.000000   0.0    0.0    0.000000   \n","e2/e2            0.000000   0.000   0.0   0.000000   0.0    0.0    0.910125   \n","e2/e3           11.111111   0.000   0.0   0.000000   0.0    0.0   16.040956   \n","e2/e4 or e1/e3   0.000000   9.375   0.0   0.000000   0.0    0.0    6.712173   \n","e3/e3           11.111111  21.875  37.5  66.666667  25.0  100.0   41.751991   \n","e3/e4           22.222222  37.500  62.5  33.333333  75.0    0.0   33.902162   \n","e4/e4           55.555556  31.250   0.0   0.000000   0.0    0.0    0.682594   \n","total            9.000000  32.000   8.0   3.000000   4.0    1.0  879.000000   \n","\n","                      MDE        SAS  \n","e1/e1            0.000000   0.000000  \n","e1/e2            0.000000   0.000000  \n","e1/e4            0.000000   0.000000  \n","e2/e2            0.000000   0.000000  \n","e2/e3            0.000000   7.692308  \n","e2/e4 or e1/e3   0.000000   0.000000  \n","e3/e3           66.666667  76.923077  \n","e3/e4           16.666667  15.384615  \n","e4/e4           16.666667   0.000000  \n","total            6.000000  13.000000  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>8 (0.91%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1 (11.11%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>141 (16.04%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (7.69%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>3 (9.38%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>59 (6.71%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1 (11.11%)</td>\n","      <td>7 (21.88%)</td>\n","      <td>3 (37.50%)</td>\n","      <td>2 (66.67%)</td>\n","      <td>1 (25.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>367 (41.75%)</td>\n","      <td>4 (66.67%)</td>\n","      <td>10 (76.92%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>2 (22.22%)</td>\n","      <td>12 (37.50%)</td>\n","      <td>5 (62.50%)</td>\n","      <td>1 (33.33%)</td>\n","      <td>3 (75.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>298 (33.90%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>2 (15.38%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>5 (55.56%)</td>\n","      <td>10 (31.25%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>6 (0.68%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>9</td>\n","      <td>32</td>\n","      <td>8</td>\n","      <td>3</td>\n","      <td>4</td>\n","      <td>1</td>\n","      <td>879</td>\n","      <td>6</td>\n","      <td>13</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                       AAC          AFR          AJ         CAH         CAS  \\\n","e1/e1            0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e2            0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e4            0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e2            0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e3           1 (11.11%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e4 or e1/e3   0 (0.00%)    3 (9.38%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e3/e3           1 (11.11%)   7 (21.88%)  3 (37.50%)  2 (66.67%)  1 (25.00%)   \n","e3/e4           2 (22.22%)  12 (37.50%)  5 (62.50%)  1 (33.33%)  3 (75.00%)   \n","e4/e4           5 (55.56%)  10 (31.25%)   0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","total                    9           32           8           3           4   \n","\n","                        EAS           EUR         MDE          SAS  \n","e1/e1             0 (0.00%)     0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e2             0 (0.00%)     0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e4             0 (0.00%)     0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e2/e2             0 (0.00%)     8 (0.91%)   0 (0.00%)    0 (0.00%)  \n","e2/e3             0 (0.00%)  141 (16.04%)   0 (0.00%)    1 (7.69%)  \n","e2/e4 or e1/e3    0 (0.00%)    59 (6.71%)   0 (0.00%)    0 (0.00%)  \n","e3/e3           1 (100.00%)  367 (41.75%)  4 (66.67%)  10 (76.92%)  \n","e3/e4             0 (0.00%)  298 (33.90%)  1 (16.67%)   2 (15.38%)  \n","e4/e4             0 (0.00%)     6 (0.68%)  1 (16.67%)    0 (0.00%)  \n","total                     1           879           6           13  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 299 of 299 bytes (100%) chr19_44905307_A_T/AD_apoe_genotype_counts.csv\n","ID                                file-Gq38580JYBPq1y2GX9bJ56Z8\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:20 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:21 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 741 of 741 bytes (100%) chr19_44905307_A_T/AD_apoe_genotype_percentages.csv\n","ID                                file-Gq3858QJYBPjp8Z6qvZ1B8xf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:22 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:23 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 970 of 970 bytes (100%) chr19_44905307_A_T/AD_apoe_genotype_combined.csv\n","ID                                file-Gq38590JYBPjp8Z6qvZ1B8xp\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              AD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:24 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:25 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","RD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>20</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>295</td>\n","      <td>1</td>\n","      <td>4</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>105</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>3</td>\n","      <td>11</td>\n","      <td>14</td>\n","      <td>1</td>\n","      <td>4</td>\n","      <td>7</td>\n","      <td>1</td>\n","      <td>670</td>\n","      <td>6</td>\n","      <td>12</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>5</td>\n","      <td>14</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>277</td>\n","      <td>1</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>5</td>\n","      <td>8</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>6</td>\n","      <td>0</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>16</td>\n","      <td>37</td>\n","      <td>18</td>\n","      <td>2</td>\n","      <td>6</td>\n","      <td>10</td>\n","      <td>1</td>\n","      <td>1373</td>\n","      <td>8</td>\n","      <td>20</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  AMR  CAH  CAS  EAS   EUR  MDE  SAS\n","e1/e1             0    0   0    0    0    0    0     0    0    0\n","e1/e2             0    0   0    0    0    0    0     0    0    0\n","e1/e4             0    0   0    0    0    0    0     0    0    0\n","e2/e2             0    0   0    0    0    0    0    20    0    0\n","e2/e3             0    1   1    0    0    0    0   295    1    4\n","e2/e4 or e1/e3    3    3   1    0    0    0    0   105    0    0\n","e3/e3             3   11  14    1    4    7    1   670    6   12\n","e3/e4             5   14   2    1    2    3    0   277    1    3\n","e4/e4             5    8   0    0    0    0    0     6    0    1\n","total            16   37  18    2    6   10    1  1373    8   20"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.00</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.00</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.00</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.00</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>1.456664</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0.00</td>\n","      <td>2.702703</td>\n","      <td>5.555556</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>21.485798</td>\n","      <td>12.5</td>\n","      <td>20.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>18.75</td>\n","      <td>8.108108</td>\n","      <td>5.555556</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>7.647487</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>18.75</td>\n","      <td>29.729730</td>\n","      <td>77.777778</td>\n","      <td>50.0</td>\n","      <td>66.666667</td>\n","      <td>70.0</td>\n","      <td>100.0</td>\n","      <td>48.798252</td>\n","      <td>75.0</td>\n","      <td>60.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>31.25</td>\n","      <td>37.837838</td>\n","      <td>11.111111</td>\n","      <td>50.0</td>\n","      <td>33.333333</td>\n","      <td>30.0</td>\n","      <td>0.0</td>\n","      <td>20.174800</td>\n","      <td>12.5</td>\n","      <td>15.0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>31.25</td>\n","      <td>21.621622</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.436999</td>\n","      <td>0.0</td>\n","      <td>5.0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>16.00</td>\n","      <td>37.000000</td>\n","      <td>18.000000</td>\n","      <td>2.0</td>\n","      <td>6.000000</td>\n","      <td>10.0</td>\n","      <td>1.0</td>\n","      <td>1373.000000</td>\n","      <td>8.0</td>\n","      <td>20.0</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                  AAC        AFR         AJ   AMR        CAH   CAS    EAS  \\\n","e1/e1            0.00   0.000000   0.000000   0.0   0.000000   0.0    0.0   \n","e1/e2            0.00   0.000000   0.000000   0.0   0.000000   0.0    0.0   \n","e1/e4            0.00   0.000000   0.000000   0.0   0.000000   0.0    0.0   \n","e2/e2            0.00   0.000000   0.000000   0.0   0.000000   0.0    0.0   \n","e2/e3            0.00   2.702703   5.555556   0.0   0.000000   0.0    0.0   \n","e2/e4 or e1/e3  18.75   8.108108   5.555556   0.0   0.000000   0.0    0.0   \n","e3/e3           18.75  29.729730  77.777778  50.0  66.666667  70.0  100.0   \n","e3/e4           31.25  37.837838  11.111111  50.0  33.333333  30.0    0.0   \n","e4/e4           31.25  21.621622   0.000000   0.0   0.000000   0.0    0.0   \n","total           16.00  37.000000  18.000000   2.0   6.000000  10.0    1.0   \n","\n","                        EUR   MDE   SAS  \n","e1/e1              0.000000   0.0   0.0  \n","e1/e2              0.000000   0.0   0.0  \n","e1/e4              0.000000   0.0   0.0  \n","e2/e2              1.456664   0.0   0.0  \n","e2/e3             21.485798  12.5  20.0  \n","e2/e4 or e1/e3     7.647487   0.0   0.0  \n","e3/e3             48.798252  75.0  60.0  \n","e3/e4             20.174800  12.5  15.0  \n","e4/e4              0.436999   0.0   5.0  \n","total           1373.000000   8.0  20.0  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>20 (1.46%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>1 (2.70%)</td>\n","      <td>1 (5.56%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>295 (21.49%)</td>\n","      <td>1 (12.50%)</td>\n","      <td>4 (20.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>3 (18.75%)</td>\n","      <td>3 (8.11%)</td>\n","      <td>1 (5.56%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>105 (7.65%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>3 (18.75%)</td>\n","      <td>11 (29.73%)</td>\n","      <td>14 (77.78%)</td>\n","      <td>1 (50.00%)</td>\n","      <td>4 (66.67%)</td>\n","      <td>7 (70.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>670 (48.80%)</td>\n","      <td>6 (75.00%)</td>\n","      <td>12 (60.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>5 (31.25%)</td>\n","      <td>14 (37.84%)</td>\n","      <td>2 (11.11%)</td>\n","      <td>1 (50.00%)</td>\n","      <td>2 (33.33%)</td>\n","      <td>3 (30.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>277 (20.17%)</td>\n","      <td>1 (12.50%)</td>\n","      <td>3 (15.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>5 (31.25%)</td>\n","      <td>8 (21.62%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>6 (0.44%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (5.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>16</td>\n","      <td>37</td>\n","      <td>18</td>\n","      <td>2</td>\n","      <td>6</td>\n","      <td>10</td>\n","      <td>1</td>\n","      <td>1373</td>\n","      <td>8</td>\n","      <td>20</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                       AAC          AFR           AJ         AMR         CAH  \\\n","e1/e1            0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e2            0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e4            0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e2            0 (0.00%)    0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e3            0 (0.00%)    1 (2.70%)    1 (5.56%)   0 (0.00%)   0 (0.00%)   \n","e2/e4 or e1/e3  3 (18.75%)    3 (8.11%)    1 (5.56%)   0 (0.00%)   0 (0.00%)   \n","e3/e3           3 (18.75%)  11 (29.73%)  14 (77.78%)  1 (50.00%)  4 (66.67%)   \n","e3/e4           5 (31.25%)  14 (37.84%)   2 (11.11%)  1 (50.00%)  2 (33.33%)   \n","e4/e4           5 (31.25%)   8 (21.62%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","total                   16           37           18           2           6   \n","\n","                       CAS          EAS           EUR         MDE          SAS  \n","e1/e1            0 (0.00%)    0 (0.00%)     0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e2            0 (0.00%)    0 (0.00%)     0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e1/e4            0 (0.00%)    0 (0.00%)     0 (0.00%)   0 (0.00%)    0 (0.00%)  \n","e2/e2            0 (0.00%)    0 (0.00%)    20 (1.46%)   0 (0.00%)    0 (0.00%)  \n","e2/e3            0 (0.00%)    0 (0.00%)  295 (21.49%)  1 (12.50%)   4 (20.00%)  \n","e2/e4 or e1/e3   0 (0.00%)    0 (0.00%)   105 (7.65%)   0 (0.00%)    0 (0.00%)  \n","e3/e3           7 (70.00%)  1 (100.00%)  670 (48.80%)  6 (75.00%)  12 (60.00%)  \n","e3/e4           3 (30.00%)    0 (0.00%)  277 (20.17%)  1 (12.50%)   3 (15.00%)  \n","e4/e4            0 (0.00%)    0 (0.00%)     6 (0.44%)   0 (0.00%)    1 (5.00%)  \n","total                   10            1          1373           8           20  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 330 of 330 bytes (100%) chr19_44905307_A_T/RD_apoe_genotype_counts.csv\n","ID                                file-Gq3859QJYBPX6JX5yk31F4Fx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:26 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:26 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 779 of 779 bytes (100%) chr19_44905307_A_T/RD_apoe_genotype_percentages.csv\n","ID                                file-Gq3859jJYBPX6JX5yk31F4GB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:27 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:28 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,076 of 1,076 bytes (100%) chr19_44905307_A_T/RD_apoe_genotype_combined.csv\n","ID                                file-Gq385B8JYBPgk6zfK995QVzb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              RD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:29 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:30 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Control\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>299</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>11</td>\n","      <td>18</td>\n","      <td>21</td>\n","      <td>2</td>\n","      <td>6</td>\n","      <td>11</td>\n","      <td>0</td>\n","      <td>4789</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>30</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>6</td>\n","      <td>17</td>\n","      <td>7</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>748</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>20</td>\n","      <td>83</td>\n","      <td>146</td>\n","      <td>12</td>\n","      <td>34</td>\n","      <td>65</td>\n","      <td>6</td>\n","      <td>9191</td>\n","      <td>6</td>\n","      <td>29</td>\n","      <td>167</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>22</td>\n","      <td>89</td>\n","      <td>21</td>\n","      <td>4</td>\n","      <td>14</td>\n","      <td>6</td>\n","      <td>4</td>\n","      <td>1810</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>25</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>5</td>\n","      <td>15</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>12</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>64</td>\n","      <td>222</td>\n","      <td>197</td>\n","      <td>18</td>\n","      <td>57</td>\n","      <td>85</td>\n","      <td>12</td>\n","      <td>16849</td>\n","      <td>8</td>\n","      <td>34</td>\n","      <td>227</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR   AJ  AMR  CAH  CAS  EAS    EUR  FIN  MDE  SAS\n","e1/e1             0    0    0    0    0    0    0      0    0    0    0\n","e1/e2             0    0    0    0    0    0    0      0    0    0    0\n","e1/e4             0    0    0    0    0    0    0      0    0    0    0\n","e2/e2             0    0    2    0    1    1    1    299    0    0    1\n","e2/e3            11   18   21    2    6   11    0   4789    1    3   30\n","e2/e4 or e1/e3    6   17    7    0    2    2    0    748    0    0    3\n","e3/e3            20   83  146   12   34   65    6   9191    6   29  167\n","e3/e4            22   89   21    4   14    6    4   1810    1    2   25\n","e4/e4             5   15    0    0    0    0    1     12    0    0    1\n","total            64  222  197   18   57   85   12  16849    8   34  227"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0000</td>\n","      <td>0.000000</td>\n","      <td>1.015228</td>\n","      <td>0.000000</td>\n","      <td>1.754386</td>\n","      <td>1.176471</td>\n","      <td>8.333333</td>\n","      <td>1.774586</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.440529</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>17.1875</td>\n","      <td>8.108108</td>\n","      <td>10.659898</td>\n","      <td>11.111111</td>\n","      <td>10.526316</td>\n","      <td>12.941176</td>\n","      <td>0.000000</td>\n","      <td>28.423052</td>\n","      <td>12.5</td>\n","      <td>8.823529</td>\n","      <td>13.215859</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>9.3750</td>\n","      <td>7.657658</td>\n","      <td>3.553299</td>\n","      <td>0.000000</td>\n","      <td>3.508772</td>\n","      <td>2.352941</td>\n","      <td>0.000000</td>\n","      <td>4.439433</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>1.321586</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>31.2500</td>\n","      <td>37.387387</td>\n","      <td>74.111675</td>\n","      <td>66.666667</td>\n","      <td>59.649123</td>\n","      <td>76.470588</td>\n","      <td>50.000000</td>\n","      <td>54.549231</td>\n","      <td>75.0</td>\n","      <td>85.294118</td>\n","      <td>73.568282</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>34.3750</td>\n","      <td>40.090090</td>\n","      <td>10.659898</td>\n","      <td>22.222222</td>\n","      <td>24.561404</td>\n","      <td>7.058824</td>\n","      <td>33.333333</td>\n","      <td>10.742477</td>\n","      <td>12.5</td>\n","      <td>5.882353</td>\n","      <td>11.013216</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>7.8125</td>\n","      <td>6.756757</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>8.333333</td>\n","      <td>0.071221</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.440529</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>64.0000</td>\n","      <td>222.000000</td>\n","      <td>197.000000</td>\n","      <td>18.000000</td>\n","      <td>57.000000</td>\n","      <td>85.000000</td>\n","      <td>12.000000</td>\n","      <td>16849.000000</td>\n","      <td>8.0</td>\n","      <td>34.000000</td>\n","      <td>227.000000</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                    AAC         AFR          AJ        AMR        CAH  \\\n","e1/e1            0.0000    0.000000    0.000000   0.000000   0.000000   \n","e1/e2            0.0000    0.000000    0.000000   0.000000   0.000000   \n","e1/e4            0.0000    0.000000    0.000000   0.000000   0.000000   \n","e2/e2            0.0000    0.000000    1.015228   0.000000   1.754386   \n","e2/e3           17.1875    8.108108   10.659898  11.111111  10.526316   \n","e2/e4 or e1/e3   9.3750    7.657658    3.553299   0.000000   3.508772   \n","e3/e3           31.2500   37.387387   74.111675  66.666667  59.649123   \n","e3/e4           34.3750   40.090090   10.659898  22.222222  24.561404   \n","e4/e4            7.8125    6.756757    0.000000   0.000000   0.000000   \n","total           64.0000  222.000000  197.000000  18.000000  57.000000   \n","\n","                      CAS        EAS           EUR   FIN        MDE  \\\n","e1/e1            0.000000   0.000000      0.000000   0.0   0.000000   \n","e1/e2            0.000000   0.000000      0.000000   0.0   0.000000   \n","e1/e4            0.000000   0.000000      0.000000   0.0   0.000000   \n","e2/e2            1.176471   8.333333      1.774586   0.0   0.000000   \n","e2/e3           12.941176   0.000000     28.423052  12.5   8.823529   \n","e2/e4 or e1/e3   2.352941   0.000000      4.439433   0.0   0.000000   \n","e3/e3           76.470588  50.000000     54.549231  75.0  85.294118   \n","e3/e4            7.058824  33.333333     10.742477  12.5   5.882353   \n","e4/e4            0.000000   8.333333      0.071221   0.0   0.000000   \n","total           85.000000  12.000000  16849.000000   8.0  34.000000   \n","\n","                       SAS  \n","e1/e1             0.000000  \n","e1/e2             0.000000  \n","e1/e4             0.000000  \n","e2/e2             0.440529  \n","e2/e3            13.215859  \n","e2/e4 or e1/e3    1.321586  \n","e3/e3            73.568282  \n","e3/e4            11.013216  \n","e4/e4             0.440529  \n","total           227.000000  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (1.02%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (1.75%)</td>\n","      <td>1 (1.18%)</td>\n","      <td>1 (8.33%)</td>\n","      <td>299 (1.77%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (0.44%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>11 (17.19%)</td>\n","      <td>18 (8.11%)</td>\n","      <td>21 (10.66%)</td>\n","      <td>2 (11.11%)</td>\n","      <td>6 (10.53%)</td>\n","      <td>11 (12.94%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>4789 (28.42%)</td>\n","      <td>1 (12.50%)</td>\n","      <td>3 (8.82%)</td>\n","      <td>30 (13.22%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>6 (9.38%)</td>\n","      <td>17 (7.66%)</td>\n","      <td>7 (3.55%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (3.51%)</td>\n","      <td>2 (2.35%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>748 (4.44%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>3 (1.32%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>20 (31.25%)</td>\n","      <td>83 (37.39%)</td>\n","      <td>146 (74.11%)</td>\n","      <td>12 (66.67%)</td>\n","      <td>34 (59.65%)</td>\n","      <td>65 (76.47%)</td>\n","      <td>6 (50.00%)</td>\n","      <td>9191 (54.55%)</td>\n","      <td>6 (75.00%)</td>\n","      <td>29 (85.29%)</td>\n","      <td>167 (73.57%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>22 (34.38%)</td>\n","      <td>89 (40.09%)</td>\n","      <td>21 (10.66%)</td>\n","      <td>4 (22.22%)</td>\n","      <td>14 (24.56%)</td>\n","      <td>6 (7.06%)</td>\n","      <td>4 (33.33%)</td>\n","      <td>1810 (10.74%)</td>\n","      <td>1 (12.50%)</td>\n","      <td>2 (5.88%)</td>\n","      <td>25 (11.01%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>5 (7.81%)</td>\n","      <td>15 (6.76%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (8.33%)</td>\n","      <td>12 (0.07%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (0.44%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>64</td>\n","      <td>222</td>\n","      <td>197</td>\n","      <td>18</td>\n","      <td>57</td>\n","      <td>85</td>\n","      <td>12</td>\n","      <td>16849</td>\n","      <td>8</td>\n","      <td>34</td>\n","      <td>227</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                        AAC          AFR            AJ          AMR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)     0 (0.00%)    0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)     0 (0.00%)    0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)     0 (0.00%)    0 (0.00%)   \n","e2/e2             0 (0.00%)    0 (0.00%)     2 (1.02%)    0 (0.00%)   \n","e2/e3           11 (17.19%)   18 (8.11%)   21 (10.66%)   2 (11.11%)   \n","e2/e4 or e1/e3    6 (9.38%)   17 (7.66%)     7 (3.55%)    0 (0.00%)   \n","e3/e3           20 (31.25%)  83 (37.39%)  146 (74.11%)  12 (66.67%)   \n","e3/e4           22 (34.38%)  89 (40.09%)   21 (10.66%)   4 (22.22%)   \n","e4/e4             5 (7.81%)   15 (6.76%)     0 (0.00%)    0 (0.00%)   \n","total                    64          222           197           18   \n","\n","                        CAH          CAS         EAS            EUR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)   0 (0.00%)      0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)   0 (0.00%)      0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)   0 (0.00%)      0 (0.00%)   \n","e2/e2             1 (1.75%)    1 (1.18%)   1 (8.33%)    299 (1.77%)   \n","e2/e3            6 (10.53%)  11 (12.94%)   0 (0.00%)  4789 (28.42%)   \n","e2/e4 or e1/e3    2 (3.51%)    2 (2.35%)   0 (0.00%)    748 (4.44%)   \n","e3/e3           34 (59.65%)  65 (76.47%)  6 (50.00%)  9191 (54.55%)   \n","e3/e4           14 (24.56%)    6 (7.06%)  4 (33.33%)  1810 (10.74%)   \n","e4/e4             0 (0.00%)    0 (0.00%)   1 (8.33%)     12 (0.07%)   \n","total                    57           85          12          16849   \n","\n","                       FIN          MDE           SAS  \n","e1/e1            0 (0.00%)    0 (0.00%)     0 (0.00%)  \n","e1/e2            0 (0.00%)    0 (0.00%)     0 (0.00%)  \n","e1/e4            0 (0.00%)    0 (0.00%)     0 (0.00%)  \n","e2/e2            0 (0.00%)    0 (0.00%)     1 (0.44%)  \n","e2/e3           1 (12.50%)    3 (8.82%)   30 (13.22%)  \n","e2/e4 or e1/e3   0 (0.00%)    0 (0.00%)     3 (1.32%)  \n","e3/e3           6 (75.00%)  29 (85.29%)  167 (73.57%)  \n","e3/e4           1 (12.50%)    2 (5.88%)   25 (11.01%)  \n","e4/e4            0 (0.00%)    0 (0.00%)     1 (0.44%)  \n","total                    8           34           227  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 385 of 385 bytes (100%) chr19_44905307_A_T/Control_apoe_genotype_counts.csv\n","ID                                file-Gq385BjJYBPq1y2GX9bJ56Zq\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              Control_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:31 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:32 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,177 of 1,177 bytes (100%) chr19_44905307_A_T/Control_apoe_genotype_percentages.csv\n","ID                                file-Gq385F8JYBPb05fV03x8z7bQ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              Control_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:33 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:33 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,205 of 1,205 bytes (100%) chr19_44905307_A_T/Control_apoe_genotype_combined.csv\n","ID                                file-Gq385FQJYBPkBqx8Gk11gGJY\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44905307_A_T\n","Name                              Control_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:34 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:35 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr19_44908756_C_A\n","AD\n","RD\n","Control\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>EUR</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>4</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                EUR\n","e1/e1             0\n","e1/e2             0\n","e1/e4             0\n","e2/e2             0\n","e2/e3             0\n","e2/e4 or e1/e3    0\n","e3/e3             3\n","e3/e4             1\n","e4/e4             0\n","total             4"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>EUR</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>75.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>25.0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>4.0</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                 EUR\n","e1/e1            0.0\n","e1/e2            0.0\n","e1/e4            0.0\n","e2/e2            0.0\n","e2/e3            0.0\n","e2/e4 or e1/e3   0.0\n","e3/e3           75.0\n","e3/e4           25.0\n","e4/e4            0.0\n","total            4.0"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>EUR</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>3 (75.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>1 (25.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>4</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                       EUR\n","e1/e1            0 (0.00%)\n","e1/e2            0 (0.00%)\n","e1/e4            0 (0.00%)\n","e2/e2            0 (0.00%)\n","e2/e3            0 (0.00%)\n","e2/e4 or e1/e3   0 (0.00%)\n","e3/e3           3 (75.00%)\n","e3/e4           1 (25.00%)\n","e4/e4            0 (0.00%)\n","total                    4"]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 94 of 94 bytes (100%) chr19_44908756_C_A/Control_apoe_genotype_counts.csv\n","ID                                file-Gq385G0JYBPkBqx8Gk11gGJg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44908756_C_A\n","Name                              Control_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:36 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:37 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 116 of 116 bytes (100%) chr19_44908756_C_A/Control_apoe_genotype_percentages.csv\n","ID                                file-Gq385GQJYBPYj1g24ZzB535f\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44908756_C_A\n","Name                              Control_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:38 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:39 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 168 of 168 bytes (100%) chr19_44908756_C_A/Control_apoe_genotype_combined.csv\n","ID                                file-Gq385J0JYBPgfjQbQQp194Kg\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr19_44908756_C_A\n","Name                              Control_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:40 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:40 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr21_25897620_C_T\n","AD\n","RD\n","Control\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>EUR</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>1</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                EUR\n","e1/e1             0\n","e1/e2             0\n","e1/e4             0\n","e2/e2             0\n","e2/e3             0\n","e2/e4 or e1/e3    0\n","e3/e3             1\n","e3/e4             0\n","e4/e4             0\n","total             1"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>EUR</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>100.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>1.0</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                  EUR\n","e1/e1             0.0\n","e1/e2             0.0\n","e1/e4             0.0\n","e2/e2             0.0\n","e2/e3             0.0\n","e2/e4 or e1/e3    0.0\n","e3/e3           100.0\n","e3/e4             0.0\n","e4/e4             0.0\n","total             1.0"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>EUR</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1 (100.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>1</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                        EUR\n","e1/e1             0 (0.00%)\n","e1/e2             0 (0.00%)\n","e1/e4             0 (0.00%)\n","e2/e2             0 (0.00%)\n","e2/e3             0 (0.00%)\n","e2/e4 or e1/e3    0 (0.00%)\n","e3/e3           1 (100.00%)\n","e3/e4             0 (0.00%)\n","e4/e4             0 (0.00%)\n","total                     1"]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 94 of 94 bytes (100%) chr21_25897620_C_T/Control_apoe_genotype_counts.csv\n","ID                                file-Gq385J8JYBPb05fV03x8z7bk\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_25897620_C_T\n","Name                              Control_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:41 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:42 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 116 of 116 bytes (100%) chr21_25897620_C_T/Control_apoe_genotype_percentages.csv\n","ID                                file-Gq385JjJYBPgfJXKzpk98fFp\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_25897620_C_T\n","Name                              Control_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:43 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:44 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 168 of 168 bytes (100%) chr21_25897620_C_T/Control_apoe_genotype_combined.csv\n","ID                                file-Gq385K8JYBPpZFv16Z88QkZ2\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_25897620_C_T\n","Name                              Control_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:45 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:45 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr21_26171645_A_G\n","AD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>8</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>3</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>58</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0</td>\n","      <td>7</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>82</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>25</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>2</td>\n","      <td>15</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>177</td>\n","      <td>3</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  CAH  CAS  EAS  EUR  SAS\n","e1/e1             0    0   0    0    0    0    0    0\n","e1/e2             0    0   0    0    0    0    0    0\n","e1/e4             0    0   0    0    0    0    0    0\n","e2/e2             0    0   0    0    0    0    1    0\n","e2/e3             0    2   0    0    0    0    8    0\n","e2/e4 or e1/e3    0    3   0    0    0    0    3    0\n","e3/e3             0    1   1    2    1    3   58    3\n","e3/e4             0    7   0    0    0    0   82    0\n","e4/e4             2    2   0    0    0    0   25    0\n","total             2   15   1    2    1    3  177    3"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.564972</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0.0</td>\n","      <td>13.333333</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>4.519774</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0.0</td>\n","      <td>20.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>1.694915</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0.0</td>\n","      <td>6.666667</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>32.768362</td>\n","      <td>100.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0.0</td>\n","      <td>46.666667</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>46.327684</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>100.0</td>\n","      <td>13.333333</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>14.124294</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>2.0</td>\n","      <td>15.000000</td>\n","      <td>1.0</td>\n","      <td>2.0</td>\n","      <td>1.0</td>\n","      <td>3.0</td>\n","      <td>177.000000</td>\n","      <td>3.0</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                  AAC        AFR     AJ    CAH    CAS    EAS         EUR  \\\n","e1/e1             0.0   0.000000    0.0    0.0    0.0    0.0    0.000000   \n","e1/e2             0.0   0.000000    0.0    0.0    0.0    0.0    0.000000   \n","e1/e4             0.0   0.000000    0.0    0.0    0.0    0.0    0.000000   \n","e2/e2             0.0   0.000000    0.0    0.0    0.0    0.0    0.564972   \n","e2/e3             0.0  13.333333    0.0    0.0    0.0    0.0    4.519774   \n","e2/e4 or e1/e3    0.0  20.000000    0.0    0.0    0.0    0.0    1.694915   \n","e3/e3             0.0   6.666667  100.0  100.0  100.0  100.0   32.768362   \n","e3/e4             0.0  46.666667    0.0    0.0    0.0    0.0   46.327684   \n","e4/e4           100.0  13.333333    0.0    0.0    0.0    0.0   14.124294   \n","total             2.0  15.000000    1.0    2.0    1.0    3.0  177.000000   \n","\n","                  SAS  \n","e1/e1             0.0  \n","e1/e2             0.0  \n","e1/e4             0.0  \n","e2/e2             0.0  \n","e2/e3             0.0  \n","e2/e4 or e1/e3    0.0  \n","e3/e3           100.0  \n","e3/e4             0.0  \n","e4/e4             0.0  \n","total             3.0  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (0.56%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>2 (13.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>8 (4.52%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>3 (20.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>3 (1.69%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>1 (6.67%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>2 (100.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>3 (100.00%)</td>\n","      <td>58 (32.77%)</td>\n","      <td>3 (100.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>7 (46.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>82 (46.33%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2 (100.00%)</td>\n","      <td>2 (13.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>25 (14.12%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>2</td>\n","      <td>15</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>177</td>\n","      <td>3</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                        AAC         AFR           AJ          CAH  \\\n","e1/e1             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e2             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e4             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e2/e2             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e2/e3             0 (0.00%)  2 (13.33%)    0 (0.00%)    0 (0.00%)   \n","e2/e4 or e1/e3    0 (0.00%)  3 (20.00%)    0 (0.00%)    0 (0.00%)   \n","e3/e3             0 (0.00%)   1 (6.67%)  1 (100.00%)  2 (100.00%)   \n","e3/e4             0 (0.00%)  7 (46.67%)    0 (0.00%)    0 (0.00%)   \n","e4/e4           2 (100.00%)  2 (13.33%)    0 (0.00%)    0 (0.00%)   \n","total                     2          15            1            2   \n","\n","                        CAS          EAS          EUR          SAS  \n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e2/e2             0 (0.00%)    0 (0.00%)    1 (0.56%)    0 (0.00%)  \n","e2/e3             0 (0.00%)    0 (0.00%)    8 (4.52%)    0 (0.00%)  \n","e2/e4 or e1/e3    0 (0.00%)    0 (0.00%)    3 (1.69%)    0 (0.00%)  \n","e3/e3           1 (100.00%)  3 (100.00%)  58 (32.77%)  3 (100.00%)  \n","e3/e4             0 (0.00%)    0 (0.00%)  82 (46.33%)    0 (0.00%)  \n","e4/e4             0 (0.00%)    0 (0.00%)  25 (14.12%)    0 (0.00%)  \n","total                     1            3          177            3  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 267 of 267 bytes (100%) chr21_26171645_A_G/AD_apoe_genotype_counts.csv\n","ID                                file-Gq385KQJYBPvJ3FY4Q1J5FFv\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:46 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:47 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 581 of 581 bytes (100%) chr21_26171645_A_G/AD_apoe_genotype_percentages.csv\n","ID                                file-Gq385P0JYBPgfjQbQQp194PB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:48 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:49 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 862 of 862 bytes (100%) chr21_26171645_A_G/AD_apoe_genotype_combined.csv\n","ID                                file-Gq385P8JYBPYj1g24ZzB5369\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              AD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:49 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:50 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","RD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>3</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>17</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>6</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1</td>\n","      <td>9</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>124</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>4</td>\n","      <td>8</td>\n","      <td>0</td>\n","      <td>4</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>74</td>\n","      <td>4</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>22</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>8</td>\n","      <td>24</td>\n","      <td>1</td>\n","      <td>7</td>\n","      <td>6</td>\n","      <td>4</td>\n","      <td>246</td>\n","      <td>11</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  CAH  CAS  EAS  EUR  SAS\n","e1/e1             0    0   0    0    0    0    0    0\n","e1/e2             0    0   0    0    0    0    0    0\n","e1/e4             0    0   0    0    0    0    0    0\n","e2/e2             0    0   0    0    0    0    3    0\n","e2/e3             1    3   0    0    1    1   17    2\n","e2/e4 or e1/e3    1    2   0    1    0    0    6    0\n","e3/e3             1    9   1    2    3    3  124    3\n","e3/e4             4    8   0    4    1    0   74    4\n","e4/e4             1    2   0    0    1    0   22    2\n","total             8   24   1    7    6    4  246   11"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>1.219512</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>12.5</td>\n","      <td>12.500000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>16.666667</td>\n","      <td>25.0</td>\n","      <td>6.910569</td>\n","      <td>18.181818</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>12.5</td>\n","      <td>8.333333</td>\n","      <td>0.0</td>\n","      <td>14.285714</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>2.439024</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>12.5</td>\n","      <td>37.500000</td>\n","      <td>100.0</td>\n","      <td>28.571429</td>\n","      <td>50.000000</td>\n","      <td>75.0</td>\n","      <td>50.406504</td>\n","      <td>27.272727</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>50.0</td>\n","      <td>33.333333</td>\n","      <td>0.0</td>\n","      <td>57.142857</td>\n","      <td>16.666667</td>\n","      <td>0.0</td>\n","      <td>30.081301</td>\n","      <td>36.363636</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>12.5</td>\n","      <td>8.333333</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>16.666667</td>\n","      <td>0.0</td>\n","      <td>8.943089</td>\n","      <td>18.181818</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>8.0</td>\n","      <td>24.000000</td>\n","      <td>1.0</td>\n","      <td>7.000000</td>\n","      <td>6.000000</td>\n","      <td>4.0</td>\n","      <td>246.000000</td>\n","      <td>11.000000</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                 AAC        AFR     AJ        CAH        CAS   EAS  \\\n","e1/e1            0.0   0.000000    0.0   0.000000   0.000000   0.0   \n","e1/e2            0.0   0.000000    0.0   0.000000   0.000000   0.0   \n","e1/e4            0.0   0.000000    0.0   0.000000   0.000000   0.0   \n","e2/e2            0.0   0.000000    0.0   0.000000   0.000000   0.0   \n","e2/e3           12.5  12.500000    0.0   0.000000  16.666667  25.0   \n","e2/e4 or e1/e3  12.5   8.333333    0.0  14.285714   0.000000   0.0   \n","e3/e3           12.5  37.500000  100.0  28.571429  50.000000  75.0   \n","e3/e4           50.0  33.333333    0.0  57.142857  16.666667   0.0   \n","e4/e4           12.5   8.333333    0.0   0.000000  16.666667   0.0   \n","total            8.0  24.000000    1.0   7.000000   6.000000   4.0   \n","\n","                       EUR        SAS  \n","e1/e1             0.000000   0.000000  \n","e1/e2             0.000000   0.000000  \n","e1/e4             0.000000   0.000000  \n","e2/e2             1.219512   0.000000  \n","e2/e3             6.910569  18.181818  \n","e2/e4 or e1/e3    2.439024   0.000000  \n","e3/e3            50.406504  27.272727  \n","e3/e4            30.081301  36.363636  \n","e4/e4             8.943089  18.181818  \n","total           246.000000  11.000000  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>3 (1.22%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1 (12.50%)</td>\n","      <td>3 (12.50%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>1 (25.00%)</td>\n","      <td>17 (6.91%)</td>\n","      <td>2 (18.18%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>1 (12.50%)</td>\n","      <td>2 (8.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (14.29%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>6 (2.44%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1 (12.50%)</td>\n","      <td>9 (37.50%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>2 (28.57%)</td>\n","      <td>3 (50.00%)</td>\n","      <td>3 (75.00%)</td>\n","      <td>124 (50.41%)</td>\n","      <td>3 (27.27%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>4 (50.00%)</td>\n","      <td>8 (33.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>4 (57.14%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>74 (30.08%)</td>\n","      <td>4 (36.36%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1 (12.50%)</td>\n","      <td>2 (8.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>22 (8.94%)</td>\n","      <td>2 (18.18%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>8</td>\n","      <td>24</td>\n","      <td>1</td>\n","      <td>7</td>\n","      <td>6</td>\n","      <td>4</td>\n","      <td>246</td>\n","      <td>11</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                       AAC         AFR           AJ         CAH         CAS  \\\n","e1/e1            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e4            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e3           1 (12.50%)  3 (12.50%)    0 (0.00%)   0 (0.00%)  1 (16.67%)   \n","e2/e4 or e1/e3  1 (12.50%)   2 (8.33%)    0 (0.00%)  1 (14.29%)   0 (0.00%)   \n","e3/e3           1 (12.50%)  9 (37.50%)  1 (100.00%)  2 (28.57%)  3 (50.00%)   \n","e3/e4           4 (50.00%)  8 (33.33%)    0 (0.00%)  4 (57.14%)  1 (16.67%)   \n","e4/e4           1 (12.50%)   2 (8.33%)    0 (0.00%)   0 (0.00%)  1 (16.67%)   \n","total                    8          24            1           7           6   \n","\n","                       EAS           EUR         SAS  \n","e1/e1            0 (0.00%)     0 (0.00%)   0 (0.00%)  \n","e1/e2            0 (0.00%)     0 (0.00%)   0 (0.00%)  \n","e1/e4            0 (0.00%)     0 (0.00%)   0 (0.00%)  \n","e2/e2            0 (0.00%)     3 (1.22%)   0 (0.00%)  \n","e2/e3           1 (25.00%)    17 (6.91%)  2 (18.18%)  \n","e2/e4 or e1/e3   0 (0.00%)     6 (2.44%)   0 (0.00%)  \n","e3/e3           3 (75.00%)  124 (50.41%)  3 (27.27%)  \n","e3/e4            0 (0.00%)   74 (30.08%)  4 (36.36%)  \n","e4/e4            0 (0.00%)    22 (8.94%)  2 (18.18%)  \n","total                    4           246          11  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 270 of 270 bytes (100%) chr21_26171645_A_G/RD_apoe_genotype_counts.csv\n","ID                                file-Gq385PjJYBPb05fV03x8z7fP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:51 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:52 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 711 of 711 bytes (100%) chr21_26171645_A_G/RD_apoe_genotype_percentages.csv\n","ID                                file-Gq385Q8JYBPb05fV03x8z7fb\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:53 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:54 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 871 of 871 bytes (100%) chr21_26171645_A_G/RD_apoe_genotype_combined.csv\n","ID                                file-Gq385QQJYBPX6JX5yk31F4Gp\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              RD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:54 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:55 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Control\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>5</td>\n","      <td>25</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>309</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>6</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>4</td>\n","      <td>11</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>52</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>18</td>\n","      <td>74</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>14</td>\n","      <td>15</td>\n","      <td>41</td>\n","      <td>1546</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>73</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>7</td>\n","      <td>36</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>7</td>\n","      <td>7</td>\n","      <td>9</td>\n","      <td>535</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>15</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1</td>\n","      <td>9</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>35</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>35</td>\n","      <td>155</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>27</td>\n","      <td>26</td>\n","      <td>55</td>\n","      <td>2496</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>97</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  AMR  CAH  CAS  EAS   EUR  FIN  MDE  SAS\n","e1/e1             0    0   0    0    0    0    0     0    0    0    0\n","e1/e2             0    0   0    0    0    0    0     0    0    0    0\n","e1/e4             0    0   0    0    0    0    0     0    0    0    0\n","e2/e2             0    0   0    0    0    0    0    19    0    0    0\n","e2/e3             5   25   0    0    2    3    3   309    0    0    6\n","e2/e4 or e1/e3    4   11   0    0    2    0    1    52    0    0    1\n","e3/e3            18   74   2    3   14   15   41  1546    1    1   73\n","e3/e4             7   36   1    0    7    7    9   535    0    0   15\n","e4/e4             1    9   0    0    2    1    1    35    0    0    2\n","total            35  155   3    3   27   26   55  2496    1    1   97"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.761218</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>14.285714</td>\n","      <td>16.129032</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>7.407407</td>\n","      <td>11.538462</td>\n","      <td>5.454545</td>\n","      <td>12.379808</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>6.185567</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>11.428571</td>\n","      <td>7.096774</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>7.407407</td>\n","      <td>0.000000</td>\n","      <td>1.818182</td>\n","      <td>2.083333</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>1.030928</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>51.428571</td>\n","      <td>47.741935</td>\n","      <td>66.666667</td>\n","      <td>100.0</td>\n","      <td>51.851852</td>\n","      <td>57.692308</td>\n","      <td>74.545455</td>\n","      <td>61.939103</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>75.257732</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>20.000000</td>\n","      <td>23.225806</td>\n","      <td>33.333333</td>\n","      <td>0.0</td>\n","      <td>25.925926</td>\n","      <td>26.923077</td>\n","      <td>16.363636</td>\n","      <td>21.434295</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>15.463918</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2.857143</td>\n","      <td>5.806452</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>7.407407</td>\n","      <td>3.846154</td>\n","      <td>1.818182</td>\n","      <td>1.402244</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>2.061856</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>35.000000</td>\n","      <td>155.000000</td>\n","      <td>3.000000</td>\n","      <td>3.0</td>\n","      <td>27.000000</td>\n","      <td>26.000000</td>\n","      <td>55.000000</td>\n","      <td>2496.000000</td>\n","      <td>1.0</td>\n","      <td>1.0</td>\n","      <td>97.000000</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                      AAC         AFR         AJ    AMR        CAH        CAS  \\\n","e1/e1            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e1/e2            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e1/e4            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e2/e2            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e2/e3           14.285714   16.129032   0.000000    0.0   7.407407  11.538462   \n","e2/e4 or e1/e3  11.428571    7.096774   0.000000    0.0   7.407407   0.000000   \n","e3/e3           51.428571   47.741935  66.666667  100.0  51.851852  57.692308   \n","e3/e4           20.000000   23.225806  33.333333    0.0  25.925926  26.923077   \n","e4/e4            2.857143    5.806452   0.000000    0.0   7.407407   3.846154   \n","total           35.000000  155.000000   3.000000    3.0  27.000000  26.000000   \n","\n","                      EAS          EUR    FIN    MDE        SAS  \n","e1/e1            0.000000     0.000000    0.0    0.0   0.000000  \n","e1/e2            0.000000     0.000000    0.0    0.0   0.000000  \n","e1/e4            0.000000     0.000000    0.0    0.0   0.000000  \n","e2/e2            0.000000     0.761218    0.0    0.0   0.000000  \n","e2/e3            5.454545    12.379808    0.0    0.0   6.185567  \n","e2/e4 or e1/e3   1.818182     2.083333    0.0    0.0   1.030928  \n","e3/e3           74.545455    61.939103  100.0  100.0  75.257732  \n","e3/e4           16.363636    21.434295    0.0    0.0  15.463918  \n","e4/e4            1.818182     1.402244    0.0    0.0   2.061856  \n","total           55.000000  2496.000000    1.0    1.0  97.000000  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>19 (0.76%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>5 (14.29%)</td>\n","      <td>25 (16.13%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (7.41%)</td>\n","      <td>3 (11.54%)</td>\n","      <td>3 (5.45%)</td>\n","      <td>309 (12.38%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>6 (6.19%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>4 (11.43%)</td>\n","      <td>11 (7.10%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (7.41%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (1.82%)</td>\n","      <td>52 (2.08%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (1.03%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>18 (51.43%)</td>\n","      <td>74 (47.74%)</td>\n","      <td>2 (66.67%)</td>\n","      <td>3 (100.00%)</td>\n","      <td>14 (51.85%)</td>\n","      <td>15 (57.69%)</td>\n","      <td>41 (74.55%)</td>\n","      <td>1546 (61.94%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>73 (75.26%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>7 (20.00%)</td>\n","      <td>36 (23.23%)</td>\n","      <td>1 (33.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>7 (25.93%)</td>\n","      <td>7 (26.92%)</td>\n","      <td>9 (16.36%)</td>\n","      <td>535 (21.43%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>15 (15.46%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1 (2.86%)</td>\n","      <td>9 (5.81%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (7.41%)</td>\n","      <td>1 (3.85%)</td>\n","      <td>1 (1.82%)</td>\n","      <td>35 (1.40%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (2.06%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>35</td>\n","      <td>155</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>27</td>\n","      <td>26</td>\n","      <td>55</td>\n","      <td>2496</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>97</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                        AAC          AFR          AJ          AMR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e2/e2             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e2/e3            5 (14.29%)  25 (16.13%)   0 (0.00%)    0 (0.00%)   \n","e2/e4 or e1/e3   4 (11.43%)   11 (7.10%)   0 (0.00%)    0 (0.00%)   \n","e3/e3           18 (51.43%)  74 (47.74%)  2 (66.67%)  3 (100.00%)   \n","e3/e4            7 (20.00%)  36 (23.23%)  1 (33.33%)    0 (0.00%)   \n","e4/e4             1 (2.86%)    9 (5.81%)   0 (0.00%)    0 (0.00%)   \n","total                    35          155           3            3   \n","\n","                        CAH          CAS          EAS            EUR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e2/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)     19 (0.76%)   \n","e2/e3             2 (7.41%)   3 (11.54%)    3 (5.45%)   309 (12.38%)   \n","e2/e4 or e1/e3    2 (7.41%)    0 (0.00%)    1 (1.82%)     52 (2.08%)   \n","e3/e3           14 (51.85%)  15 (57.69%)  41 (74.55%)  1546 (61.94%)   \n","e3/e4            7 (25.93%)   7 (26.92%)   9 (16.36%)   535 (21.43%)   \n","e4/e4             2 (7.41%)    1 (3.85%)    1 (1.82%)     35 (1.40%)   \n","total                    27           26           55           2496   \n","\n","                        FIN          MDE          SAS  \n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e2/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e2/e3             0 (0.00%)    0 (0.00%)    6 (6.19%)  \n","e2/e4 or e1/e3    0 (0.00%)    0 (0.00%)    1 (1.03%)  \n","e3/e3           1 (100.00%)  1 (100.00%)  73 (75.26%)  \n","e3/e4             0 (0.00%)    0 (0.00%)  15 (15.46%)  \n","e4/e4             0 (0.00%)    0 (0.00%)    2 (2.06%)  \n","total                     1            1           97  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 363 of 363 bytes (100%) chr21_26171645_A_G/Control_apoe_genotype_counts.csv\n","ID                                file-Gq385V0JYBPgp85fYz93yVG4\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              Control_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:56 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:57 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,096 of 1,096 bytes (100%) chr21_26171645_A_G/Control_apoe_genotype_percentages.csv\n","ID                                file-Gq385VQJYBPq1y2GX9bJ56bJ\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              Control_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:58 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:13:58 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,182 of 1,182 bytes (100%) chr21_26171645_A_G/Control_apoe_genotype_combined.csv\n","ID                                file-Gq385VjJYBPY9jV3VVF01139\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171645_A_G\n","Name                              Control_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:13:59 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:00 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","chr21_26171723_T_C\n","AD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>8</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>3</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>58</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0</td>\n","      <td>7</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>83</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>26</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>2</td>\n","      <td>15</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>179</td>\n","      <td>3</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  CAH  CAS  EAS  EUR  SAS\n","e1/e1             0    0   0    0    0    0    0    0\n","e1/e2             0    0   0    0    0    0    0    0\n","e1/e4             0    0   0    0    0    0    0    0\n","e2/e2             0    0   0    0    0    0    1    0\n","e2/e3             0    2   0    0    0    0    8    0\n","e2/e4 or e1/e3    0    3   0    0    0    0    3    0\n","e3/e3             0    1   1    2    1    3   58    3\n","e3/e4             0    7   0    0    0    0   83    0\n","e4/e4             2    2   0    0    0    0   26    0\n","total             2   15   1    2    1    3  179    3"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.558659</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0.0</td>\n","      <td>13.333333</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>4.469274</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0.0</td>\n","      <td>20.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>1.675978</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0.0</td>\n","      <td>6.666667</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>32.402235</td>\n","      <td>100.0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0.0</td>\n","      <td>46.666667</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>46.368715</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>100.0</td>\n","      <td>13.333333</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>14.525140</td>\n","      <td>0.0</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>2.0</td>\n","      <td>15.000000</td>\n","      <td>1.0</td>\n","      <td>2.0</td>\n","      <td>1.0</td>\n","      <td>3.0</td>\n","      <td>179.000000</td>\n","      <td>3.0</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                  AAC        AFR     AJ    CAH    CAS    EAS         EUR  \\\n","e1/e1             0.0   0.000000    0.0    0.0    0.0    0.0    0.000000   \n","e1/e2             0.0   0.000000    0.0    0.0    0.0    0.0    0.000000   \n","e1/e4             0.0   0.000000    0.0    0.0    0.0    0.0    0.000000   \n","e2/e2             0.0   0.000000    0.0    0.0    0.0    0.0    0.558659   \n","e2/e3             0.0  13.333333    0.0    0.0    0.0    0.0    4.469274   \n","e2/e4 or e1/e3    0.0  20.000000    0.0    0.0    0.0    0.0    1.675978   \n","e3/e3             0.0   6.666667  100.0  100.0  100.0  100.0   32.402235   \n","e3/e4             0.0  46.666667    0.0    0.0    0.0    0.0   46.368715   \n","e4/e4           100.0  13.333333    0.0    0.0    0.0    0.0   14.525140   \n","total             2.0  15.000000    1.0    2.0    1.0    3.0  179.000000   \n","\n","                  SAS  \n","e1/e1             0.0  \n","e1/e2             0.0  \n","e1/e4             0.0  \n","e2/e2             0.0  \n","e2/e3             0.0  \n","e2/e4 or e1/e3    0.0  \n","e3/e3           100.0  \n","e3/e4             0.0  \n","e4/e4             0.0  \n","total             3.0  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (0.56%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>2 (13.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>8 (4.47%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>3 (20.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>3 (1.68%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>0 (0.00%)</td>\n","      <td>1 (6.67%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>2 (100.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>3 (100.00%)</td>\n","      <td>58 (32.40%)</td>\n","      <td>3 (100.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>7 (46.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>83 (46.37%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2 (100.00%)</td>\n","      <td>2 (13.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>26 (14.53%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>2</td>\n","      <td>15</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>179</td>\n","      <td>3</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                        AAC         AFR           AJ          CAH  \\\n","e1/e1             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e2             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e1/e4             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e2/e2             0 (0.00%)   0 (0.00%)    0 (0.00%)    0 (0.00%)   \n","e2/e3             0 (0.00%)  2 (13.33%)    0 (0.00%)    0 (0.00%)   \n","e2/e4 or e1/e3    0 (0.00%)  3 (20.00%)    0 (0.00%)    0 (0.00%)   \n","e3/e3             0 (0.00%)   1 (6.67%)  1 (100.00%)  2 (100.00%)   \n","e3/e4             0 (0.00%)  7 (46.67%)    0 (0.00%)    0 (0.00%)   \n","e4/e4           2 (100.00%)  2 (13.33%)    0 (0.00%)    0 (0.00%)   \n","total                     2          15            1            2   \n","\n","                        CAS          EAS          EUR          SAS  \n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e2/e2             0 (0.00%)    0 (0.00%)    1 (0.56%)    0 (0.00%)  \n","e2/e3             0 (0.00%)    0 (0.00%)    8 (4.47%)    0 (0.00%)  \n","e2/e4 or e1/e3    0 (0.00%)    0 (0.00%)    3 (1.68%)    0 (0.00%)  \n","e3/e3           1 (100.00%)  3 (100.00%)  58 (32.40%)  3 (100.00%)  \n","e3/e4             0 (0.00%)    0 (0.00%)  83 (46.37%)    0 (0.00%)  \n","e4/e4             0 (0.00%)    0 (0.00%)  26 (14.53%)    0 (0.00%)  \n","total                     1            3          179            3  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 267 of 267 bytes (100%) chr21_26171723_T_C/AD_apoe_genotype_counts.csv\n","ID                                file-Gq385X8JYBPq1y2GX9bJ56bP\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:01 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:02 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 583 of 583 bytes (100%) chr21_26171723_T_C/AD_apoe_genotype_percentages.csv\n","ID                                file-Gq385XjJYBPgp85fYz93yVGB\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:03 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:03 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 862 of 862 bytes (100%) chr21_26171723_T_C/AD_apoe_genotype_combined.csv\n","ID                                file-Gq385Y0JYBPgfjQbQQp194PX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              AD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:04 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:05 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","RD\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>3</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>17</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>6</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1</td>\n","      <td>9</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>124</td>\n","      <td>3</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>5</td>\n","      <td>8</td>\n","      <td>0</td>\n","      <td>4</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>74</td>\n","      <td>4</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1</td>\n","      <td>3</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>23</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>9</td>\n","      <td>25</td>\n","      <td>1</td>\n","      <td>7</td>\n","      <td>6</td>\n","      <td>4</td>\n","      <td>247</td>\n","      <td>11</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  CAH  CAS  EAS  EUR  SAS\n","e1/e1             0    0   0    0    0    0    0    0\n","e1/e2             0    0   0    0    0    0    0    0\n","e1/e4             0    0   0    0    0    0    0    0\n","e2/e2             0    0   0    0    0    0    3    0\n","e2/e3             1    3   0    0    1    1   17    2\n","e2/e4 or e1/e3    1    2   0    1    0    0    6    0\n","e3/e3             1    9   1    2    3    3  124    3\n","e3/e4             5    8   0    4    1    0   74    4\n","e4/e4             1    3   0    0    1    0   23    2\n","total             9   25   1    7    6    4  247   11"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>1.214575</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>11.111111</td>\n","      <td>12.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>16.666667</td>\n","      <td>25.0</td>\n","      <td>6.882591</td>\n","      <td>18.181818</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>11.111111</td>\n","      <td>8.0</td>\n","      <td>0.0</td>\n","      <td>14.285714</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>2.429150</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>11.111111</td>\n","      <td>36.0</td>\n","      <td>100.0</td>\n","      <td>28.571429</td>\n","      <td>50.000000</td>\n","      <td>75.0</td>\n","      <td>50.202429</td>\n","      <td>27.272727</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>55.555556</td>\n","      <td>32.0</td>\n","      <td>0.0</td>\n","      <td>57.142857</td>\n","      <td>16.666667</td>\n","      <td>0.0</td>\n","      <td>29.959514</td>\n","      <td>36.363636</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>11.111111</td>\n","      <td>12.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>16.666667</td>\n","      <td>0.0</td>\n","      <td>9.311741</td>\n","      <td>18.181818</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>9.000000</td>\n","      <td>25.0</td>\n","      <td>1.0</td>\n","      <td>7.000000</td>\n","      <td>6.000000</td>\n","      <td>4.0</td>\n","      <td>247.000000</td>\n","      <td>11.000000</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                      AAC   AFR     AJ        CAH        CAS   EAS  \\\n","e1/e1            0.000000   0.0    0.0   0.000000   0.000000   0.0   \n","e1/e2            0.000000   0.0    0.0   0.000000   0.000000   0.0   \n","e1/e4            0.000000   0.0    0.0   0.000000   0.000000   0.0   \n","e2/e2            0.000000   0.0    0.0   0.000000   0.000000   0.0   \n","e2/e3           11.111111  12.0    0.0   0.000000  16.666667  25.0   \n","e2/e4 or e1/e3  11.111111   8.0    0.0  14.285714   0.000000   0.0   \n","e3/e3           11.111111  36.0  100.0  28.571429  50.000000  75.0   \n","e3/e4           55.555556  32.0    0.0  57.142857  16.666667   0.0   \n","e4/e4           11.111111  12.0    0.0   0.000000  16.666667   0.0   \n","total            9.000000  25.0    1.0   7.000000   6.000000   4.0   \n","\n","                       EUR        SAS  \n","e1/e1             0.000000   0.000000  \n","e1/e2             0.000000   0.000000  \n","e1/e4             0.000000   0.000000  \n","e2/e2             1.214575   0.000000  \n","e2/e3             6.882591  18.181818  \n","e2/e4 or e1/e3    2.429150   0.000000  \n","e3/e3            50.202429  27.272727  \n","e3/e4            29.959514  36.363636  \n","e4/e4             9.311741  18.181818  \n","total           247.000000  11.000000  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>3 (1.21%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>1 (11.11%)</td>\n","      <td>3 (12.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>1 (25.00%)</td>\n","      <td>17 (6.88%)</td>\n","      <td>2 (18.18%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>1 (11.11%)</td>\n","      <td>2 (8.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (14.29%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>6 (2.43%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>1 (11.11%)</td>\n","      <td>9 (36.00%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>2 (28.57%)</td>\n","      <td>3 (50.00%)</td>\n","      <td>3 (75.00%)</td>\n","      <td>124 (50.20%)</td>\n","      <td>3 (27.27%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>5 (55.56%)</td>\n","      <td>8 (32.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>4 (57.14%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>74 (29.96%)</td>\n","      <td>4 (36.36%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1 (11.11%)</td>\n","      <td>3 (12.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (16.67%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>23 (9.31%)</td>\n","      <td>2 (18.18%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>9</td>\n","      <td>25</td>\n","      <td>1</td>\n","      <td>7</td>\n","      <td>6</td>\n","      <td>4</td>\n","      <td>247</td>\n","      <td>11</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                       AAC         AFR           AJ         CAH         CAS  \\\n","e1/e1            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e1/e4            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e2            0 (0.00%)   0 (0.00%)    0 (0.00%)   0 (0.00%)   0 (0.00%)   \n","e2/e3           1 (11.11%)  3 (12.00%)    0 (0.00%)   0 (0.00%)  1 (16.67%)   \n","e2/e4 or e1/e3  1 (11.11%)   2 (8.00%)    0 (0.00%)  1 (14.29%)   0 (0.00%)   \n","e3/e3           1 (11.11%)  9 (36.00%)  1 (100.00%)  2 (28.57%)  3 (50.00%)   \n","e3/e4           5 (55.56%)  8 (32.00%)    0 (0.00%)  4 (57.14%)  1 (16.67%)   \n","e4/e4           1 (11.11%)  3 (12.00%)    0 (0.00%)   0 (0.00%)  1 (16.67%)   \n","total                    9          25            1           7           6   \n","\n","                       EAS           EUR         SAS  \n","e1/e1            0 (0.00%)     0 (0.00%)   0 (0.00%)  \n","e1/e2            0 (0.00%)     0 (0.00%)   0 (0.00%)  \n","e1/e4            0 (0.00%)     0 (0.00%)   0 (0.00%)  \n","e2/e2            0 (0.00%)     3 (1.21%)   0 (0.00%)  \n","e2/e3           1 (25.00%)    17 (6.88%)  2 (18.18%)  \n","e2/e4 or e1/e3   0 (0.00%)     6 (2.43%)   0 (0.00%)  \n","e3/e3           3 (75.00%)  124 (50.20%)  3 (27.27%)  \n","e3/e4            0 (0.00%)   74 (29.96%)  4 (36.36%)  \n","e4/e4            0 (0.00%)    23 (9.31%)  2 (18.18%)  \n","total                    4           247          11  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 270 of 270 bytes (100%) chr21_26171723_T_C/RD_apoe_genotype_counts.csv\n","ID                                file-Gq385YQJYBPvJ3FY4Q1J5FJ4\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:06 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:07 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 733 of 733 bytes (100%) chr21_26171723_T_C/RD_apoe_genotype_percentages.csv\n","ID                                file-Gq385Z0JYBPb05fV03x8z7gX\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:08 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:08 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 872 of 872 bytes (100%) chr21_26171723_T_C/RD_apoe_genotype_combined.csv\n","ID                                file-Gq385Z8JYBPvJ3FY4Q1J5FJG\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              RD_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:09 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:10 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","Control\n"]},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>19</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>5</td>\n","      <td>25</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>309</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>6</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>4</td>\n","      <td>11</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>0</td>\n","      <td>1</td>\n","      <td>52</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>1</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>18</td>\n","      <td>74</td>\n","      <td>2</td>\n","      <td>3</td>\n","      <td>15</td>\n","      <td>15</td>\n","      <td>41</td>\n","      <td>1548</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>73</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>7</td>\n","      <td>36</td>\n","      <td>1</td>\n","      <td>0</td>\n","      <td>7</td>\n","      <td>7</td>\n","      <td>9</td>\n","      <td>537</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>15</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1</td>\n","      <td>9</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","      <td>1</td>\n","      <td>1</td>\n","      <td>35</td>\n","      <td>0</td>\n","      <td>0</td>\n","      <td>2</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>35</td>\n","      <td>155</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>28</td>\n","      <td>26</td>\n","      <td>55</td>\n","      <td>2500</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>97</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                AAC  AFR  AJ  AMR  CAH  CAS  EAS   EUR  FIN  MDE  SAS\n","e1/e1             0    0   0    0    0    0    0     0    0    0    0\n","e1/e2             0    0   0    0    0    0    0     0    0    0    0\n","e1/e4             0    0   0    0    0    0    0     0    0    0    0\n","e2/e2             0    0   0    0    0    0    0    19    0    0    0\n","e2/e3             5   25   0    0    2    3    3   309    0    0    6\n","e2/e4 or e1/e3    4   11   0    0    2    0    1    52    0    0    1\n","e3/e3            18   74   2    3   15   15   41  1548    1    2   73\n","e3/e4             7   36   1    0    7    7    9   537    0    0   15\n","e4/e4             1    9   0    0    2    1    1    35    0    0    2\n","total            35  155   3    3   28   26   55  2500    1    2   97"]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.00</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.00</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.00</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.000000</td>\n","      <td>0.76</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>0.000000</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>14.285714</td>\n","      <td>16.129032</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>7.142857</td>\n","      <td>11.538462</td>\n","      <td>5.454545</td>\n","      <td>12.36</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>6.185567</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>11.428571</td>\n","      <td>7.096774</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>7.142857</td>\n","      <td>0.000000</td>\n","      <td>1.818182</td>\n","      <td>2.08</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>1.030928</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>51.428571</td>\n","      <td>47.741935</td>\n","      <td>66.666667</td>\n","      <td>100.0</td>\n","      <td>53.571429</td>\n","      <td>57.692308</td>\n","      <td>74.545455</td>\n","      <td>61.92</td>\n","      <td>100.0</td>\n","      <td>100.0</td>\n","      <td>75.257732</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>20.000000</td>\n","      <td>23.225806</td>\n","      <td>33.333333</td>\n","      <td>0.0</td>\n","      <td>25.000000</td>\n","      <td>26.923077</td>\n","      <td>16.363636</td>\n","      <td>21.48</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>15.463918</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>2.857143</td>\n","      <td>5.806452</td>\n","      <td>0.000000</td>\n","      <td>0.0</td>\n","      <td>7.142857</td>\n","      <td>3.846154</td>\n","      <td>1.818182</td>\n","      <td>1.40</td>\n","      <td>0.0</td>\n","      <td>0.0</td>\n","      <td>2.061856</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>35.000000</td>\n","      <td>155.000000</td>\n","      <td>3.000000</td>\n","      <td>3.0</td>\n","      <td>28.000000</td>\n","      <td>26.000000</td>\n","      <td>55.000000</td>\n","      <td>2500.00</td>\n","      <td>1.0</td>\n","      <td>2.0</td>\n","      <td>97.000000</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                      AAC         AFR         AJ    AMR        CAH        CAS  \\\n","e1/e1            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e1/e2            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e1/e4            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e2/e2            0.000000    0.000000   0.000000    0.0   0.000000   0.000000   \n","e2/e3           14.285714   16.129032   0.000000    0.0   7.142857  11.538462   \n","e2/e4 or e1/e3  11.428571    7.096774   0.000000    0.0   7.142857   0.000000   \n","e3/e3           51.428571   47.741935  66.666667  100.0  53.571429  57.692308   \n","e3/e4           20.000000   23.225806  33.333333    0.0  25.000000  26.923077   \n","e4/e4            2.857143    5.806452   0.000000    0.0   7.142857   3.846154   \n","total           35.000000  155.000000   3.000000    3.0  28.000000  26.000000   \n","\n","                      EAS      EUR    FIN    MDE        SAS  \n","e1/e1            0.000000     0.00    0.0    0.0   0.000000  \n","e1/e2            0.000000     0.00    0.0    0.0   0.000000  \n","e1/e4            0.000000     0.00    0.0    0.0   0.000000  \n","e2/e2            0.000000     0.76    0.0    0.0   0.000000  \n","e2/e3            5.454545    12.36    0.0    0.0   6.185567  \n","e2/e4 or e1/e3   1.818182     2.08    0.0    0.0   1.030928  \n","e3/e3           74.545455    61.92  100.0  100.0  75.257732  \n","e3/e4           16.363636    21.48    0.0    0.0  15.463918  \n","e4/e4            1.818182     1.40    0.0    0.0   2.061856  \n","total           55.000000  2500.00    1.0    2.0  97.000000  "]},"metadata":{},"output_type":"display_data"},{"data":{"text/html":["<div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>AAC</th>\n","      <th>AFR</th>\n","      <th>AJ</th>\n","      <th>AMR</th>\n","      <th>CAH</th>\n","      <th>CAS</th>\n","      <th>EAS</th>\n","      <th>EUR</th>\n","      <th>FIN</th>\n","      <th>MDE</th>\n","      <th>SAS</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>e1/e1</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e1/e4</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e2</th>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>19 (0.76%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e3</th>\n","      <td>5 (14.29%)</td>\n","      <td>25 (16.13%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (7.14%)</td>\n","      <td>3 (11.54%)</td>\n","      <td>3 (5.45%)</td>\n","      <td>309 (12.36%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>6 (6.19%)</td>\n","    </tr>\n","    <tr>\n","      <th>e2/e4 or e1/e3</th>\n","      <td>4 (11.43%)</td>\n","      <td>11 (7.10%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (7.14%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (1.82%)</td>\n","      <td>52 (2.08%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>1 (1.03%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e3</th>\n","      <td>18 (51.43%)</td>\n","      <td>74 (47.74%)</td>\n","      <td>2 (66.67%)</td>\n","      <td>3 (100.00%)</td>\n","      <td>15 (53.57%)</td>\n","      <td>15 (57.69%)</td>\n","      <td>41 (74.55%)</td>\n","      <td>1548 (61.92%)</td>\n","      <td>1 (100.00%)</td>\n","      <td>2 (100.00%)</td>\n","      <td>73 (75.26%)</td>\n","    </tr>\n","    <tr>\n","      <th>e3/e4</th>\n","      <td>7 (20.00%)</td>\n","      <td>36 (23.23%)</td>\n","      <td>1 (33.33%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>7 (25.00%)</td>\n","      <td>7 (26.92%)</td>\n","      <td>9 (16.36%)</td>\n","      <td>537 (21.48%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>15 (15.46%)</td>\n","    </tr>\n","    <tr>\n","      <th>e4/e4</th>\n","      <td>1 (2.86%)</td>\n","      <td>9 (5.81%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (7.14%)</td>\n","      <td>1 (3.85%)</td>\n","      <td>1 (1.82%)</td>\n","      <td>35 (1.40%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>0 (0.00%)</td>\n","      <td>2 (2.06%)</td>\n","    </tr>\n","    <tr>\n","      <th>total</th>\n","      <td>35</td>\n","      <td>155</td>\n","      <td>3</td>\n","      <td>3</td>\n","      <td>28</td>\n","      <td>26</td>\n","      <td>55</td>\n","      <td>2500</td>\n","      <td>1</td>\n","      <td>2</td>\n","      <td>97</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>"],"text/plain":["                        AAC          AFR          AJ          AMR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e2/e2             0 (0.00%)    0 (0.00%)   0 (0.00%)    0 (0.00%)   \n","e2/e3            5 (14.29%)  25 (16.13%)   0 (0.00%)    0 (0.00%)   \n","e2/e4 or e1/e3   4 (11.43%)   11 (7.10%)   0 (0.00%)    0 (0.00%)   \n","e3/e3           18 (51.43%)  74 (47.74%)  2 (66.67%)  3 (100.00%)   \n","e3/e4            7 (20.00%)  36 (23.23%)  1 (33.33%)    0 (0.00%)   \n","e4/e4             1 (2.86%)    9 (5.81%)   0 (0.00%)    0 (0.00%)   \n","total                    35          155           3            3   \n","\n","                        CAH          CAS          EAS            EUR  \\\n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)      0 (0.00%)   \n","e2/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)     19 (0.76%)   \n","e2/e3             2 (7.14%)   3 (11.54%)    3 (5.45%)   309 (12.36%)   \n","e2/e4 or e1/e3    2 (7.14%)    0 (0.00%)    1 (1.82%)     52 (2.08%)   \n","e3/e3           15 (53.57%)  15 (57.69%)  41 (74.55%)  1548 (61.92%)   \n","e3/e4            7 (25.00%)   7 (26.92%)   9 (16.36%)   537 (21.48%)   \n","e4/e4             2 (7.14%)    1 (3.85%)    1 (1.82%)     35 (1.40%)   \n","total                    28           26           55           2500   \n","\n","                        FIN          MDE          SAS  \n","e1/e1             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e1/e4             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e2/e2             0 (0.00%)    0 (0.00%)    0 (0.00%)  \n","e2/e3             0 (0.00%)    0 (0.00%)    6 (6.19%)  \n","e2/e4 or e1/e3    0 (0.00%)    0 (0.00%)    1 (1.03%)  \n","e3/e3           1 (100.00%)  2 (100.00%)  73 (75.26%)  \n","e3/e4             0 (0.00%)    0 (0.00%)  15 (15.46%)  \n","e4/e4             0 (0.00%)    0 (0.00%)    2 (2.06%)  \n","total                     1            2           97  "]},"metadata":{},"output_type":"display_data"},{"name":"stdout","output_type":"stream","text":["[===========================================================>] Uploaded 363 of 363 bytes (100%) chr21_26171723_T_C/Control_apoe_genotype_counts.csv\n","ID                                file-Gq385ZjJYBPgfJXKzpk98fKx\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              Control_apoe_genotype_counts.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:11 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:12 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,027 of 1,027 bytes (100%) chr21_26171723_T_C/Control_apoe_genotype_percentages.csv\n","ID                                file-Gq385b8JYBPYj1g24ZzB537K\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              Control_apoe_genotype_percentages.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:13 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:14 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n","[===========================================================>] Uploaded 1,182 of 1,182 bytes (100%) chr21_26171723_T_C/Control_apoe_genotype_combined.csv\n","ID                                file-Gq385bQJYBPvJ3FY4Q1J5FJf\n","Class                             file\n","Project                           project-Gbgg70jJYBPf3yZ26F3GVFjx\n","Folder                            /wgs_analysis/results/12_protective_variants/chr21_26171723_T_C\n","Name                              Control_apoe_genotype_combined.csv\n","State                             \u001b[33mclosing\u001b[0m\n","Visibility                        visible\n","Types                             -\n","Properties                        -\n","Tags                              -\n","Outgoing links                    -\n","Created                           Thu Aug 22 00:14:14 2024\n","Created by                        spencermg3\n"," via the job                      job-Gq34GgQJYBPxJQFZ2Y16j8KX\n","Last modified                     Thu Aug 22 00:14:15 2024\n","Media type                        \n","archivalState                     \"live\"\n","cloudAccount                      \"cloudaccount-dnanexus\"\n"]}],"source":["for var_id in [\"chr19_44892887_C_T\",\"chr19_44905307_A_T\",\"chr19_44908756_C_A\",\"chr21_25897620_C_T\",\"chr21_26171645_A_G\",\"chr21_26171723_T_C\"]:\n","    print(var_id)\n","    for cohort in [\"AD\",\"RD\",\"Control\"]:\n","        print(cohort)\n","        cohort_counts = []\n","        included_ancestries = []\n","        for ancestry in [\"AAC\",\"AFR\",\"AJ\",\"AMR\",\"CAH\",\"CAS\",\"EAS\",\"EUR\",\"FIN\",\"MDE\",\"SAS\"]:\n","            if os.path.exists(f\"{var_id}/apoe_final_{cohort}_{ancestry}.APOE_GENOTYPES.csv\"):\n","                apoe_geno = pd.read_csv(f\"{var_id}/apoe_final_{cohort}_{ancestry}.APOE_GENOTYPES.csv\")\n","                counts = apoe_geno['APOE_GENOTYPE'].value_counts()\n","                counts = counts.reindex([\"e1/e1\", \"e1/e2\", \"e1/e4\", \"e2/e2\", \"e2/e3\", \"e2/e4 or e1/e3\", \"e3/e3\", \"e3/e4\", \"e4/e4\"], fill_value=0)\n","                counts = counts.rename(ancestry)\n","                counts = pd.concat([counts, pd.Series([counts.sum()], index=['total'])])\n","                cohort_counts.append(counts)\n","                included_ancestries.append(ancestry)\n","        if len(cohort_counts) > 0:\n","            cohort_counts = pd.concat(cohort_counts, axis=1)\n","            cohort_counts.columns = included_ancestries\n","            display(cohort_counts)\n","            cohort_percentages = cohort_counts.div(cohort_counts.loc[\"total\"], axis=1) * 100\n","            cohort_percentages.loc[\"total\"] = cohort_counts.loc[\"total\"]\n","            display(cohort_percentages)\n","            cohort_combined = cohort_counts.applymap(str) + ' (' + cohort_percentages.applymap(lambda x: f'{x:.2f}%') + ')'\n","            cohort_combined.loc[\"total\"] = cohort_counts.loc[\"total\"]\n","            display(cohort_combined)\n","            cohort_counts.to_csv(f\"{var_id}/{cohort}_apoe_genotype_counts.csv\")\n","            cohort_percentages.to_csv(f\"{var_id}/{cohort}_apoe_genotype_percentages.csv\")\n","            cohort_combined.to_csv(f\"{var_id}/{cohort}_apoe_genotype_combined.csv\")\n","            ! dx upload {var_id}/{cohort}_apoe_genotype_counts.csv --path wgs_analysis/results/12_protective_variants/{var_id}/{cohort}_apoe_genotype_counts.csv\n","            ! dx upload {var_id}/{cohort}_apoe_genotype_percentages.csv --path wgs_analysis/results/12_protective_variants/{var_id}/{cohort}_apoe_genotype_percentages.csv\n","            ! dx upload {var_id}/{cohort}_apoe_genotype_combined.csv --path wgs_analysis/results/12_protective_variants/{var_id}/{cohort}_apoe_genotype_combined.csv\n"]}],"metadata":{"kernelspec":{"display_name":"Python 3 (ipykernel)","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.9.16"}},"nbformat":4,"nbformat_minor":5}
