{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "904cadb1",
   "metadata": {},
   "source": [
    "#  Visualize Data\n",
    "\n",
    "* **Project:** ADRD Genetic Diversity in Biobanks\n",
    "* **Version:** Python/3.10\n",
    "* **Last Updated:** 24-FEB-2025\n",
    "\n",
    "## Notebook Overview\n",
    "Visualizing the results of protective/resilience variants using a heatmap, Visualizing the results of allele frequencies using a heatmap, Visualizing the results of APOE genotypes using barplot, Visualizing mutation sites on predicted protein structures, Visualizing Beta values using Upset plot, Generating a PCA plot"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "271dcccf",
   "metadata": {},
   "source": [
    "## Variables used \n",
    "`${COHORT}` = AD, Dementia, Control\n",
    "\n",
    "`${Biobank}` = UKB, AoU, ADSP, AMP PD, 100KGP\n",
    "\n",
    "`${Value}` = data specific to each biobank"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3ef01688",
   "metadata": {},
   "source": [
    "### Using a Heatmap to Visualize Protective/Resilient Variants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "889b4026",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_z = pd.read_csv(\"Heatmap_protective_${COHORT}.csv\")\n",
    "df_z.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3bc3eae0-fa1b-4580-99eb-c5d4659f1d40",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load the CSV file\n",
    "df_z = pd.read_csv('Heatmap_protective_${COHORT}.csv')\n",
    "\n",
    "# Filter out rows where the 'Variants' column is empty\n",
    "df_z = df_z[df_z['Variants'].notna()]\n",
    "\n",
    "# Find the maximum value in the numeric columns\n",
    "max_value = df_z.iloc[:, 1:].max().max()\n",
    "\n",
    "# Print the maximum value\n",
    "print(\"The maximum value in the dataset is:\", max_value)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6349c9e1",
   "metadata": {},
   "outputs": [],
   "source": [
    "import seaborn as sns\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib.colors import Normalize, LinearSegmentedColormap\n",
    "import pandas as pd\n",
    "import matplotlib.patches as patches\n",
    "\n",
    "# Create a figure with three subplots (one row, three columns)\n",
    "fig, axs = plt.subplots(1, 3, figsize=(30, 8))  \n",
    "\n",
    "# Define file names, titles, and norms for each heatmap\n",
    "file_names = ['HeatmapPlot_protective_AD.csv', 'HeatmapPlot_protective_Dementia.csv', 'HeatmapPlot_protective_control.csv']\n",
    "titles = ['AD_e4 ', 'Related Dementias_e4', 'Controls_e4 ']\n",
    "norms = [Normalize(vmin=, vmax=), Normalize(vmin=, vmax=), Normalize(vmin=, vmax=)]\n",
    "text_positions = [0.8, 3.7, 1.9]  # Text positions for AD, Related Dementia, Controls\n",
    "\n",
    "# Custom colormap to set 0 as white\n",
    "colors = [(1, 1, 1), *sns.color_palette(\"Spectral_r\", 256)]\n",
    "custom_cmap = LinearSegmentedColormap.from_list(\"CustomSpectral\", colors, N=256)\n",
    "\n",
    "# Iterate over the files, titles, and subplots to create each heatmap\n",
    "for i, (file_name, title, norm, text_position) in enumerate(zip(file_names, titles, norms, text_positions)):\n",
    "    # Load the data\n",
    "    df_z = pd.read_csv(file_name)\n",
    "    df_z = df_z[df_z['Variants'].notna()]\n",
    "\n",
    "    # List of ancestry columns (all columns except the first one)\n",
    "    cols = list(df_z.columns)[1:]\n",
    "\n",
    "    # Create heatmap with custom colormap and no internal borders\n",
    "    sns.heatmap(\n",
    "        df_z[cols],\n",
    "        cmap=custom_cmap,\n",
    "        norm=norm,\n",
    "        cbar_kws={\"shrink\": 0.8},\n",
    "        ax=axs[i],\n",
    "        linewidths=0,  # No internal borders between cells\n",
    "    )\n",
    "\n",
    "    # Set axis labels\n",
    "    axs[i].set_ylabel('SNPs', fontsize=13, fontweight='bold')\n",
    "    axs[i].set_xlabel('Ancestry', fontsize=13, fontweight='bold')\n",
    "\n",
    "    # Adjust y-ticks to be centered\n",
    "    y_ticks = range(len(df_z))  \n",
    "    axs[i].set_yticks([tick + 0.5 for tick in y_ticks])\n",
    "    axs[i].set_yticklabels(df_z['Variants'], rotation=0, fontsize=13, ha='right', va='center')\n",
    "\n",
    "    # Keep x-ticks as default (top-aligned)\n",
    "    axs[i].set_xticklabels(axs[i].get_xticklabels(), rotation=45, ha='right', fontsize=11, va='top')\n",
    "\n",
    "    # Draw a border around the outer edge of the heatmap (4 sides)\n",
    "    border = patches.Rectangle((0, 0), 1, 1, transform=axs[i].transAxes, color='black', linewidth=2, fill=False)\n",
    "    axs[i].add_patch(border)\n",
    "\n",
    "    # Get the colorbar from the heatmap\n",
    "    colorbar = axs[i].collections[0].colorbar\n",
    "\n",
    "    # Adjust colorbar tick label size\n",
    "    colorbar.ax.tick_params(labelsize=13)\n",
    "\n",
    "    # Adjust text label position above the colorbar\n",
    "    colorbar.ax.text(text_position, 1.05, title, ha='center', va='center', fontsize=13, fontweight='bold', transform=colorbar.ax.transAxes)\n",
    "\n",
    "    # Add a label to the colorbar to indicate percentage\n",
    "    colorbar.ax.set_ylabel('Percentage', fontsize=12)\n",
    "\n",
    "# Adjust layout to prevent overlap\n",
    "plt.tight_layout()\n",
    "\n",
    "# Save the combined plot\n",
    "plt.savefig('combined_heatmap.png', dpi=300, bbox_inches='tight')\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "769b9018",
   "metadata": {},
   "outputs": [],
   "source": [
    "import seaborn as sns\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "for grouping in [\"all\",\"e4e4\",\"e4\"]:\n",
    "    df_control = pd.read_csv(f\"Control_{grouping}.csv\", index_col=0)\n",
    "    fig, axes = plt.subplots(1, 2, figsize=(20,10))\n",
    "    for i, pheno in enumerate([\"AD\",\"RD\"]):\n",
    "        df_case = pd.read_csv(f\"{pheno}_{grouping}.csv\", index_col=0)\n",
    "        df_ratio = df_case / df_control\n",
    "        arr_ratio = df_ratio.values\n",
    "\n",
    "        mask_inf = arr_ratio == np.inf\n",
    "        mask_zero = arr_ratio == 0\n",
    "        mask_na = np.isnan(arr_ratio)\n",
    "        mask_other = ~(mask_inf + mask_zero + mask_na)\n",
    "\n",
    "        arr_log = np.log2(arr_ratio)\n",
    "        max_abs = np.max(np.abs(arr_log[mask_other]))\n",
    "        arr_log[mask_inf] = max_abs + 1\n",
    "        arr_log[mask_zero] = (max_abs + 1) * -1\n",
    "        arr_log[mask_na] = (max_abs + 1) * -1\n",
    "\n",
    "        cmap = sns.color_palette(\"Spectral_r\", as_cmap=True)\n",
    "\n",
    "        ax = sns.heatmap(\n",
    "            arr_log,\n",
    "            cmap=cmap,\n",
    "            cbar=True,\n",
    "            xticklabels=df_ratio.columns.values,\n",
    "            yticklabels=df_ratio.index.values,\n",
    "            ax = axes[i],\n",
    "            mask=mask_na, \n",
    "        )\n",
    "\n",
    "        for _, spine in ax.spines.items():\n",
    "            spine.set_visible(True)\n",
    "            spine.set_color(\"black\")\n",
    "            spine.set_linewidth(1)\n",
    "\n",
    "        cbar = ax.collections[0].colorbar\n",
    "\n",
    "        min_val, max_val = np.ceil(arr_log.min()), np.floor(arr_log.max())\n",
    "        ticks = np.arange(int(min_val), int(max_val) + 1)\n",
    "        ticks = np.append(arr_log.min(), ticks)\n",
    "        ticks = np.append(ticks, arr_log.max())\n",
    "        cbar.set_ticks(ticks)\n",
    "        ticklabels = [\"Only controls\"] + [f\"{2**tick:.3f}\" for tick in np.arange(int(min_val), int(max_val) + 1, dtype=float)] + [\"Only cases\"]\n",
    "        cbar.set_ticklabels(ticklabels)\n",
    "\n",
    "        ax.set_xticklabels(ax.get_xticklabels(), rotation=45)\n",
    "        ax.set_ylabel('SNPs', fontweight=\"bold\")\n",
    "        ax.set_xlabel('Ancestry', fontweight=\"bold\")\n",
    "\n",
    "        if pheno == \"AD\":\n",
    "            ax.set_title(f\"{pheno} vs Control Frequency Ratio\")\n",
    "        else:\n",
    "            ax.set_title(f\"Related Dementias vs Control Frequency Ratio\")\n",
    "\n",
    "    plt.tight_layout()\n",
    "    plt.savefig(f\"Merged_{grouping}.png\")\n",
    "    plt.close()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0184a265",
   "metadata": {},
   "source": [
    "### Using a Heatmap to Visualize Allele Frequencies Across All Biobanks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cf95160a",
   "metadata": {},
   "outputs": [],
   "source": [
    "import seaborn as sns\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib.colors import ListedColormap, BoundaryNorm\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "# Load your data into a DataFrame from the specified CSV file\n",
    "df_z = pd.read_csv('${WORK_DIR}/${Biobank}.csv')\n",
    "\n",
    "# Filter out rows where the 'SNP' column is empty\n",
    "df_z = df_z[df_z['SNP'].notna()]\n",
    "\n",
    "# List of ancestry columns (all columns except the first one)\n",
    "cols = list(df_z.columns)[1:]\n",
    "\n",
    "# Create a colormap with a slightly darker color for zero and a linear scale for non-zero values\n",
    "zero_color = '#E0E0E0'  \n",
    "\n",
    "# Generate a colormap for non-zero values using the viridis color palette\n",
    "nonzero_colors = sns.color_palette(\"viridis\", n_colors=100)\n",
    "colors = [zero_color] + nonzero_colors\n",
    "custom_cmap = ListedColormap(colors)\n",
    "\n",
    "# Define boundaries for the colormap and normalization\n",
    "min_value = ${Value}\n",
    "max_value = ${Value}\n",
    "boundaries = np.linspace(min_value, max_value, 100)\n",
    "norm = BoundaryNorm(boundaries, custom_cmap.N, clip=True)\n",
    "\n",
    "# Increase the figure size \n",
    "plt.figure(figsize=(3, 6))  \n",
    "\n",
    "# Create a heatmap with custom color palette and normalization\n",
    "ax = sns.heatmap(df_z[cols], cmap=custom_cmap, norm=norm, cbar_kws={\"shrink\": 0.8})\n",
    "\n",
    "# Set axis labels with bold font\n",
    "ax.set_ylabel('SNPs', fontsize=6, fontweight='bold') \n",
    "ax.set_xlabel('Ancestry', fontsize=6, fontweight='bold')  \n",
    "\n",
    "# Add title \n",
    "plt.title('Allele frequencies of variants across all ancestries in ${Biobank}', fontsize=6, fontweight='bold')\n",
    "\n",
    "# Set y-ticks and y-tick labels to match the number of variants\n",
    "ax.set_yticks(np.arange(len(df_z)) + 0.5)  \n",
    "ax.set_yticklabels(df_z['SNP'], rotation=0, fontsize=5, ha='right')  \n",
    "\n",
    "# Adjust the y-axis label \n",
    "ax.yaxis.labelpad = 10  \n",
    "\n",
    "# Rotate x-axis labels \n",
    "plt.xticks(rotation=45, ha='right', fontsize=5)  \n",
    "\n",
    "# Adjust colorbar ticks and labels to handle zero properly and make the colorbar less cluttered\n",
    "colorbar = ax.collections[0].colorbar\n",
    "# Set fewer ticks to avoid clutter\n",
    "colorbar.set_ticks(np.linspace(min_value, max_value, 6))\n",
    "colorbar.set_ticklabels([f'{tick:.4f}' for tick in np.linspace(min_value, max_value, 6)])\n",
    "\n",
    "# Adjust colorbar font size \n",
    "colorbar.ax.tick_params(labelsize=3)  \n",
    "colorbar.ax.yaxis.set_tick_params(labelsize=2, width=0.1, color='black', labelcolor='black')\n",
    "colorbar.ax.yaxis.set_tick_params(width=1) \n",
    "for label in colorbar.ax.get_yticklabels():\n",
    "    label.set_fontsize(3)  \n",
    "    label.set_fontweight('bold')  \n",
    "\n",
    "# Save the plot \n",
    "plt.savefig('${Biobank}_Final.png', dpi=300, bbox_inches='tight')\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f305b887",
   "metadata": {},
   "source": [
    "### Using a Barplot to Visualize the Results of APOE Genotypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "56736325",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "import numpy as np\n",
    "\n",
    "# Define the data\n",
    "data = {\n",
    "    'Genotypes': ['AD_e4/e4', 'Related Dementias_e4/e4', 'Controls_e4/e4'],\n",
    "    'EUR': [13.03,7.53,1.7],\n",
    "    'AFR': [13.56,6.91,4.29],\n",
    "    'AMR': [4.58,4.09,0.96],\n",
    "    'EAS': [9.38,0,1.04],\n",
    "    'SAS': [2.64,5.21,1.19],\n",
    "    'MDE': [8.11,0,0.71],\n",
    "    'AJ': [8.02,8.88,1],\n",
    "    'FIN': [36.36,0,1.67],\n",
    "    'AAC': [13.08,7.74,3.31],\n",
    "    'CAS': [6.25,6.67,1.59],\n",
    "    'CAH': [7.56,3.7,2.07]\n",
    "}\n",
    "\n",
    "# Create a DataFrame\n",
    "df = pd.DataFrame(data)\n",
    "\n",
    "# Define bar width and x locations for the groups\n",
    "bar_width = 0.25\n",
    "x = np.arange(len(df.columns[1:]))  \n",
    "\n",
    "# Define the space between bars and data labels\n",
    "label_offset = 0.2\n",
    "\n",
    "# Plot\n",
    "plt.figure(figsize=(10, 10))\n",
    "\n",
    "# Plot bars for each genotype\n",
    "for i, genotype in enumerate(df['Genotypes']):\n",
    "    bars = plt.bar(x + i * bar_width, df.iloc[i, 1:], width=bar_width, label=genotype)\n",
    "    for bar, value in zip(bars, df.iloc[i, 1:]):\n",
    "        if value.is_integer():\n",
    "            plt.text(bar.get_x() + bar.get_width() / 2, bar.get_height() + label_offset, f'{int(value)}', ha='center', va='bottom', color='black', fontsize=7, fontweight='bold', rotation=90)\n",
    "        else:\n",
    "            plt.text(bar.get_x() + bar.get_width() / 2, bar.get_height() + label_offset, f'{value:.2f}', ha='center', va='bottom', color='black', fontsize=7, fontweight='bold', rotation=90)\n",
    "\n",
    "# Set labels and title\n",
    "plt.xlabel('Ancestries', fontweight='bold')\n",
    "plt.ylabel('Proportion (%)', fontweight='bold')\n",
    "plt.title('Proportions of APOE e4/e4 Across Different Ancestries')\n",
    "\n",
    "# Add x-ticks and labels\n",
    "plt.xticks(x + bar_width, df.columns[1:], rotation=45)\n",
    "\n",
    "# Add legend\n",
    "plt.legend(title='Genotype')\n",
    "\n",
    "# Show plot\n",
    "plt.tight_layout()\n",
    "plt.savefig('proportions_e4e4.png', dpi=300)\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6bd6fbbe",
   "metadata": {},
   "outputs": [],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "import numpy as np\n",
    "\n",
    "# Data for the first plot\n",
    "genotypes_first_plot = ['e2/e2', 'e2/e3', 'e2/e4 or e1/e3', 'e3/e3', 'e3/e4', 'e4/e4']\n",
    "populations = ['EUR', 'AFR', 'AMR', 'EAS', 'SAS', 'MDE', 'AJ', 'FIN', 'AAC', 'CAS', 'CAH']\n",
    "data_first_plot = {\n",
    "    'EUR': [0.17, 4.04, 2.29, 35.25, 45.22, 13.03],\n",
    "    'AFR': [0.67, 7.25, 5.23, 31.41, 41.88, 13.56],\n",
    "    'AMR': [0, 3.63, 0.86, 63.27, 27.66, 4.58],\n",
    "    'EAS': [0, 7.81, 1.56, 48.44, 32.81, 9.38],\n",
    "    'SAS': [0, 7.05, 1.76, 60.79, 27.75, 2.64],\n",
    "    'MDE': [2.7, 2.7, 0, 62.16, 24.32, 8.11],\n",
    "    'AJ': [0.33, 3.76, 1.96, 45.34, 40.59, 8.02],\n",
    "    'FIN': [0, 0, 0, 27.27, 36.36, 36.36],\n",
    "    'AAC': [0.43, 9.91, 3.68, 34.02, 38.89, 13.08],\n",
    "    'CAS': [0, 6.25, 3.13, 50, 34.38, 6.25],\n",
    "    'CAH': [0.28, 6.54, 2.15, 51.73, 31.75, 7.56]\n",
    "}\n",
    "\n",
    "# Data for the second plot\n",
    "genotypes_second_plot = ['e2/e2', 'e2/e3', 'e2/e4 or e1/e3', 'e3/e3', 'e3/e4', 'e4/e4']\n",
    "data_second_plot = {\n",
    "    'EUR': [0.42,8.22,2.94,46.87,34.03,7.53],\n",
    "    'AFR': [0.84,12.60,6.50,41.06,32.11,6.91],\n",
    "    'AMR': [0.92,8.64,0.45,60.91,25,4.09],\n",
    "    'EAS': [0,20,3.33,56.67,20,0],\n",
    "    'SAS': [0,8.33,1.04,54.17,31.25,5.21],\n",
    "    'MDE': [0,8,0,76,16,0],\n",
    "    'AJ': [0.39,8.49,3.47,51.74,27.03,8.88],\n",
    "    'FIN': [0,0,0,62.5,37.5,0],\n",
    "    'AAC': [0.6,11.31,5.36,44.05,30.95,7.74],\n",
    "    'CAS': [0,3.33,0,60,30,6.67],\n",
    "    'CAH': [0,0,7.41,48.15,40.74,3.70]\n",
    "}\n",
    "\n",
    "# Data for the third plot\n",
    "genotypes_third_plot = ['e1/e4', 'e2/e2', 'e2/e3', 'e2/e4 or e1/e3', 'e3/e3', 'e3/e4', 'e4/e4']\n",
    "data_third_plot = {\n",
    "    'EUR': [0,0.63,12.82,2.14,61.36,21.35,1.7],\n",
    "    'AFR': [0.03,1.25,15.38,5.11,45.44,28.5,4.29],\n",
    "    'AMR': [0,0.18,5.78,0.76,74.53,17.79,0.96],\n",
    "    'EAS': [0,0.69,15.45,0.87,66.67,15.28,1.04],\n",
    "    'SAS': [0,0.15,7.75,1.13,73.05,16.72,1.19],\n",
    "    'MDE': [0,0,13.48,1.42,75.18,9.22,0.71],\n",
    "    'AJ': [0,0.5,11.5,1.61,64.01,21.39,1],\n",
    "    'FIN': [0,0,15,0,66.67,16.67,1.67],\n",
    "    'AAC': [0,0.91,14.3,4.03,49.45,27.1,3.31],\n",
    "    'CAS': [0,0.8,10.34,1.33,71.09,14.85,1.59],\n",
    "    'CAH': [0,0.33,10,1.79,64.4,21.41,2.07]\n",
    "}\n",
    "\n",
    "# Define plot parameters\n",
    "barWidth = 0.13\n",
    "new_color = '#333333'\n",
    "\n",
    "# Plot\n",
    "fig, axs = plt.subplots(3, 1, figsize=(16, 30))\n",
    "\n",
    "# Plot the first plot\n",
    "r = np.arange(len(populations))\n",
    "bars_list = []\n",
    "\n",
    "for i, genotype in enumerate(genotypes_first_plot):\n",
    "    bars = [data_first_plot[pop][i] for pop in populations]\n",
    "    bars_list.append(axs[0].bar(r + i * barWidth, bars, width=barWidth, edgecolor='grey', label=genotype))\n",
    "    for j, bar in enumerate(bars):\n",
    "        value = int(bar) if isinstance(bar, int) or float(bar).is_integer() else round(bar, 2)\n",
    "        axs[0].annotate(str(value), xy=(r[j] + i * barWidth, bar), xytext=(0, 3), textcoords='offset points', ha='center', va='bottom', color='black', fontsize=8, fontweight='bold', rotation=90)\n",
    "axs[0].set_xlabel('Ancestries', fontweight='bold')\n",
    "axs[0].set_ylabel('Proportion (%)', fontweight='bold')\n",
    "axs[0].set_title('Proportions of APOE Genotypes Across Different Ancestries in AD')\n",
    "axs[0].set_xticks(r + barWidth * (len(genotypes_first_plot) - 1) / 2)\n",
    "axs[0].set_xticklabels(populations, rotation=45)\n",
    "axs[0].legend()\n",
    "\n",
    "# Extract colors used in the first plot\n",
    "colors_first_plot = [bar[0].get_facecolor() for bar in bars_list]\n",
    "\n",
    "# Plot the second plot\n",
    "for i, genotype in enumerate(genotypes_second_plot):\n",
    "    bars = [data_second_plot[pop][i] for pop in populations]\n",
    "    axs[1].bar(r + i * barWidth, bars, width=barWidth, edgecolor='grey', color=colors_first_plot[i], label=genotype)\n",
    "    for j, bar in enumerate(bars):\n",
    "        value = int(bar) if isinstance(bar, int) or float(bar).is_integer() else round(bar, 2)\n",
    "        axs[1].annotate(str(value), xy=(r[j] + i * barWidth, bar), xytext=(0, 3), textcoords='offset points', ha='center', va='bottom', color='black', fontsize=8, fontweight='bold',rotation=90)\n",
    "axs[1].set_xlabel('Ancestries', fontweight='bold')\n",
    "axs[1].set_ylabel('Proportion (%)', fontweight='bold')\n",
    "axs[1].set_title('Proportions of APOE Genotypes Across Different Ancestries in Related Dementias')\n",
    "axs[1].set_xticks(r + barWidth * (len(genotypes_second_plot) - 1) / 2)\n",
    "axs[1].set_xticklabels(populations, rotation=45)\n",
    "axs[1].legend()\n",
    "\n",
    "# Plot the third plot\n",
    "colors_third_plot = [new_color] + colors_first_plot\n",
    "for i, genotype in enumerate(genotypes_third_plot):\n",
    "    bars = [data_third_plot[pop][i] for pop in populations]\n",
    "    axs[2].bar(r + i * barWidth, bars, width=barWidth, edgecolor='grey', color=colors_third_plot[i], label=genotype)\n",
    "    for j, bar in enumerate(bars):\n",
    "        value = int(bar) if isinstance(bar, int) or float(bar).is_integer() else round(bar, 2)\n",
    "        axs[2].annotate(str(value), xy=(r[j] + i * barWidth, bar), xytext=(0, 3), textcoords='offset points', ha='center', va='bottom', color='black', fontsize=8, fontweight='bold', rotation=90)\n",
    "axs[2].set_xlabel('Ancestries', fontweight='bold')\n",
    "axs[2].set_ylabel('Proportion (%)', fontweight='bold')\n",
    "axs[2].set_title('Proportions of APOE Genotypes Across Different Ancestries in Controls')\n",
    "axs[2].set_xticks(r + barWidth * (len(genotypes_third_plot) - 1) / 2)\n",
    "axs[2].set_xticklabels(populations, rotation=45)\n",
    "axs[2].legend()\n",
    "\n",
    "plt.tight_layout()\n",
    "plt.savefig('APOE_genotypes.png', dpi=300)\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9b7a083e",
   "metadata": {},
   "source": [
    "### Visualizing mutation sites on predicted protein structures"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cd120c8b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# The code used for the APP protein was presented. This code was also used for data on PSEN1, PSEN2, TREM2, MAPT, GRN, SNCA, GBA1, TBK1, TARDBP, and APOE.\n",
    "# Load structure\n",
    "load af-app.cif\n",
    "\n",
    "# Modify the representation of the system\n",
    "hide all\n",
    "show cartoon\n",
    "\n",
    "# Customize the cartoon representation\n",
    "set cartoon_fancy_helices\n",
    "set cartoon_highlight_color, grey75\n",
    "color red, ss h\n",
    "color yellow, ss s\n",
    "color green, ss l+''\n",
    "\n",
    "# Select the residues of interest\n",
    "select P132S, resi 132 and name CA\n",
    "select A209T, resi 209 and name CA\n",
    "select V227L, resi 227 and name CA\n",
    "select I248V, resi 248 and name CA\n",
    "select P251S, resi 251 and name CA\n",
    "select S271G, resi 271 and name CA\n",
    "select L364F, resi 364 and name CA\n",
    "select T371A, resi 371 and name CA\n",
    "select V375I, resi 375 and name CA\n",
    "select S407T, resi 407 and name CA\n",
    "select R409C, resi 409 and name CA\n",
    "select G458R, resi 458 and name CA\n",
    "select D460N, resi 460 and name CA\n",
    "select D516N, resi 516 and name CA\n",
    "select R517G, resi 517 and name CA\n",
    "select Y538H, resi 538 and name CA\n",
    "select L597W, resi 597 and name CA\n",
    "select I600L, resi 600 and name CA\n",
    "select E693Q, resi 693 and name CA\n",
    "select A713T, resi 713 and name CA\n",
    "select V717L, resi 717 and name CA\n",
    "select A740V, resi 740 and name CA\n",
    "\n",
    "# Color the residues of interest\n",
    "color cyan, P132S\n",
    "color cyan, A209T\n",
    "color cyan, V227L\n",
    "color cyan, I248V\n",
    "color cyan, P251S\n",
    "color cyan, S271G\n",
    "color cyan, L364F\n",
    "color cyan, T371A\n",
    "color cyan, V375I\n",
    "color cyan, S407T\n",
    "color cyan, R409C\n",
    "color cyan, G458R\n",
    "color cyan, D460N\n",
    "color cyan, D516N\n",
    "color cyan, R517G\n",
    "color cyan, Y538H\n",
    "color cyan, L597W\n",
    "color cyan, I600L\n",
    "color cyan, E693Q\n",
    "color cyan, A713T\n",
    "color cyan, V717L\n",
    "color cyan, A740V\n",
    "\n",
    "# Change the representation of the residues of interest\n",
    "show spheres, P132S\n",
    "show spheres, A209T\n",
    "show spheres, V227L\n",
    "show spheres, I248V\n",
    "show spheres, P251S\n",
    "show spheres, S271G\n",
    "show spheres, L364F\n",
    "show spheres, T371A\n",
    "show spheres, V375I\n",
    "show spheres, S407T\n",
    "show spheres, R409C\n",
    "show spheres, G458R\n",
    "show spheres, D460N\n",
    "show spheres, D516N\n",
    "show spheres, R517G\n",
    "show spheres, Y538H\n",
    "show spheres, L597W\n",
    "show spheres, I600L\n",
    "show spheres, E693Q\n",
    "show spheres, A713T\n",
    "show spheres, V717L\n",
    "show spheres, A740V\n",
    "\n",
    "# Label the residues of interest\n",
    "label P132S, \"P132S\" \n",
    "label A209T, \"A209T\"\n",
    "label V227L, \"V227L\"\n",
    "label I248V, \"I248V\"\n",
    "label P251S, \"P251S\"\n",
    "label S271G, \"S271G\"\n",
    "label L364F, \"L364F\"\n",
    "label T371A, \"T371A\"\n",
    "label V375I, \"V375I\"\n",
    "label S407T, \"S407T\"\n",
    "label R409C, \"R409C\"\n",
    "label G458R, \"G458R\"\n",
    "label D460N, \"D460N\"\n",
    "label D516N, \"D516N\"\n",
    "label R517G, \"R517G\"\n",
    "label Y538H, \"Y538H\"\n",
    "label L597W, \"L597W\"\n",
    "label I600L, \"I600L\"\n",
    "label E693Q, \"E693Q\"\n",
    "label A713T, \"A713T\"\n",
    "label V717L, \"V717L\"\n",
    "label A740V, \"A740V\"\n",
    "\n",
    "# Adjust the size of the label\n",
    "set label_size=12\n",
    "set label_position=[7.0, 0.0, 30.0]\n",
    "set label_font_id=7\n",
    "\n",
    "# Change the background color\n",
    "bg_color white\n",
    "set cartoon_transparency=0.4\n",
    "\n",
    "# Render image and save\n",
    "set depth_cue=0\n",
    "orient\n",
    "zoom complete=1\n",
    "ray 3000, 3000\n",
    "#png app_labels.png, dpi=500\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b7f8bb7",
   "metadata": {},
   "source": [
    "### Using an Upset plot to Visualize the Beta Values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3b040437",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import matplotlib\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import seaborn as sns\n",
    "#import warnings\n",
    "from matplotlib import rcParams\n",
    "from matplotlib.ticker import FormatStrFormatter\n",
    "from matplotlib.collections import PatchCollection\n",
    "from matplotlib.colors import LinearSegmentedColormap\n",
    "import matplotlib.gridspec as gridspec\n",
    "#warnings.simplefilter(action='ignore', category=FutureWarning)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8d4138e7",
   "metadata": {},
   "outputs": [],
   "source": [
    "Protective = pd.read_csv('Protective_model_beta.csv')\n",
    "Protective.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9d9094a6",
   "metadata": {},
   "outputs": [],
   "source": [
    "for i,row in Protective.iterrows():\n",
    "    SNP = row['SNP']\n",
    "    for ANC in ['EUR','AFR','AMR', 'AJ','AAC', 'CAH']:\n",
    "        beta = row[ANC]\n",
    "        tempDF = pd.DataFrame({\n",
    "            'ANC':ANC,\n",
    "            'SNP':SNP,\n",
    "            'BETA':beta\n",
    "        }, index=[0])\n",
    "        if (i==0) & (ANC=='EUR'):\n",
    "            inputDF = tempDF\n",
    "        else:\n",
    "            inputDF = pd.concat([inputDF, tempDF]).reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "93fa7726",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "from matplotlib.colors import LinearSegmentedColormap\n",
    "\n",
    "# Convert 'BETA' to numeric, setting errors='coerce' will turn non-numeric values into NaN\n",
    "inputDF['BETA'] = pd.to_numeric(inputDF['BETA'], errors='coerce')\n",
    "\n",
    "# Drop rows with NaN in 'BETA' if necessary\n",
    "inputDF = inputDF.dropna(subset=['BETA'])\n",
    "\n",
    "# Define palette\n",
    "pal = [\"#6B1414\", \"#C14E4E\", \"#CF9FFF\", \"#1E90FF\", \"#000000\"]\n",
    "\n",
    "# Define NormalizeData function\n",
    "def NormalizeData(data):\n",
    "    return (data - np.min(data)) / (np.max(data) - np.min(data))\n",
    "\n",
    "# Calculate quantiles\n",
    "quantiles = inputDF[\"BETA\"].describe().loc[['min', '25%', '50%', '75%', 'max']].tolist()\n",
    "quantiles[2] = 0  # Set median (50%) to zero for color normalization\n",
    "quantiles_norm = NormalizeData(quantiles)\n",
    "\n",
    "# Create colormap\n",
    "cmp = LinearSegmentedColormap.from_list(\"\", list(zip(quantiles_norm, pal)))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f005de10",
   "metadata": {},
   "outputs": [],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "import matplotlib.gridspec as gridspec\n",
    "import seaborn as sns\n",
    "from matplotlib.cm import ScalarMappable\n",
    "\n",
    "# Extract only the rs part of SNP identifiers\n",
    "inputDF['SNP_rs'] = inputDF['SNP'].str.extract(r'(rs\\d+)', expand=False)\n",
    "\n",
    "# Set up the figure and axes\n",
    "fig, ax = plt.subplots(figsize=(2, 3), sharex='col')\n",
    "fig.clf()\n",
    "gs = gridspec.GridSpec(1, 2, width_ratios=(8, 0.7))\n",
    "\n",
    "ax1 = fig.add_subplot(gs[0, 0])\n",
    "ax2 = fig.add_subplot(gs[0, 1])\n",
    "\n",
    "sns.set_style({'font_scale': 1, 'axes.facecolor': 'white', 'grid.color': '.8'})\n",
    "sns.set_theme(style=\"white\", palette=None)\n",
    "\n",
    "# Set margins and spacing\n",
    "plt.margins(x=0.01, y=0.05)\n",
    "plt.subplots_adjust(wspace=0.1, hspace=0)\n",
    "\n",
    "# Create scatter plot using SNP_rs for x-axis labels\n",
    "plot_bottom = sns.scatterplot(data=inputDF, x='SNP_rs', y='ANC', hue='BETA', s=70,\n",
    "                              edgecolor='black', palette=cmp, legend='brief', ax=ax1)\n",
    "\n",
    "# Set title and remove x-axis label\n",
    "plot_bottom.set_title('Protective model', fontsize=8 ,fontweight='bold')\n",
    "plot_bottom.set_xlabel('')  \n",
    "\n",
    "# Customize plot appearance\n",
    "plot_bottom.spines[\"top\"].set_visible(False) \n",
    "plot_bottom.spines[\"bottom\"].set_visible(False) \n",
    "plot_bottom.spines[\"left\"].set_visible(False) \n",
    "plot_bottom.spines[\"right\"].set_visible(False) \n",
    "plot_bottom.set_xticklabels(inputDF.drop_duplicates(subset='SNP_rs')['SNP_rs'], rotation=45, ha='right')\n",
    "plot_bottom.get_legend().remove()\n",
    "\n",
    "# Adjust the font size of tick labels on x and y axes\n",
    "plot_bottom.tick_params(axis='x', labelsize=8)  \n",
    "plot_bottom.tick_params(axis='y', labelsize=8)  \n",
    "\n",
    "# Shade specific y-ranges\n",
    "plot_bottom.axhspan(ymin=0.5, ymax=1.5, color='steelblue', alpha=0.1, lw=0)\n",
    "plot_bottom.axhspan(ymin=2.5, ymax=3.5, color='steelblue', alpha=0.1, lw=0)\n",
    "plot_bottom.axhspan(ymin=4.5, ymax=5.5, color='steelblue', alpha=0.1, lw=0)\n",
    "plot_bottom.set_ylabel('')\n",
    "\n",
    "# Set up color bar with ScalarMappable\n",
    "sm = ScalarMappable(cmap=cmp)\n",
    "sm.set_array([])\n",
    "sm.set_clim(inputDF[\"BETA\"].min(), inputDF[\"BETA\"].max())\n",
    "\n",
    "min_beta = inputDF['BETA'].min()\n",
    "max_beta = inputDF['BETA'].max()\n",
    "\n",
    "cb_ticks = [inputDF['BETA'].min(), 0, inputDF['BETA'].max()]\n",
    "cb = plt.colorbar(sm, cax=ax2, ticks=[min_beta, 0, max_beta])\n",
    "cb.ax.set_yticklabels(cb_ticks)   \n",
    "cb.outline.set_color('black')\n",
    "cb.outline.set_linewidth(1)\n",
    "cb.ax.tick_params(labelsize=6, pad=1, width=0)\n",
    "cb.ax.set_title('Beta', size=8, pad=6)\n",
    "\n",
    "# Save the figure\n",
    "fig.savefig('Protective.png', facecolor='white', dpi=300, bbox_inches=\"tight\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b81b7e7f",
   "metadata": {},
   "source": [
    "### Generating a PCA plot"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1a5f6509",
   "metadata": {},
   "outputs": [],
   "source": [
    "def plot_3d(labeled_df, color, symbol=None, x='PC1', y='PC2', z='PC3', title=None, x_range=None, y_range=None, z_range=None, plot_out=None):\n",
    "    '''\n",
    "    Parameters: \n",
    "    labeled_df (Pandas dataframe): labeled ancestry dataframe\n",
    "    color (string): color of ancestry label. column name containing labels for ancestry in labeled_pcs_df\n",
    "    symbol (string): symbol of secondary label (for example, predicted vs reference ancestry). default: None\n",
    "    x (string): column name of x-dimension\n",
    "    y (string): column name of y-dimension\n",
    "    z (string): column name of z-dimension\n",
    "    title (string, optional): title of output scatterplot\n",
    "    x_range (list of floats [min, max], optional): range for x-axis\n",
    "    y_range (list of floats [min, max], optional): range for y-axis\n",
    "    z_range (list of floats [min, max], optional): range for z-axis\n",
    "\n",
    "    Returns:\n",
    "    3-D scatterplot (plotly.express.scatter_3d). If plot_out included, will write .png static image and .html interactive to plot_out filename\n",
    "                \n",
    "    '''\n",
    "\n",
    "    fig = px.scatter_3d(\n",
    "                labeled_df,\n",
    "                x=x,\n",
    "                y=y,\n",
    "                z=z,\n",
    "                color=color,\n",
    "                symbol=symbol,\n",
    "                title=title,\n",
    "                color_discrete_sequence=px.colors.qualitative.Bold,\n",
    "                range_x=x_range,\n",
    "                range_y=y_range,\n",
    "                range_z=z_range,\n",
    "                hover_name=\"IID\",\n",
    "                color_discrete_map={'AFR': \"#88CCEE\",\n",
    "                                    'SAS': \"#CC6677\",\n",
    "                                    'EAS': \"#DDCC77\",\n",
    "                                    'EUR':\"#117733\",\n",
    "                                    'AMR':\"#332288\",\n",
    "                                    'AJ': \"#D55E00\",\n",
    "                                    'AAC':\"#999933\",\n",
    "                                    'CAS':\"#882255\",\n",
    "                                    'MDE':\"#661100\",\n",
    "                                    'FIN':\"#F0E442\",\n",
    "                                    'CAH':\"#40B0A6\",\n",
    "                                    'new':\"#ababab\"}\n",
    "            )\n",
    "\n",
    "    fig.update_traces(marker={'size': 3})\n",
    "    \n",
    "    fig.show()\n",
    "    \n",
    "    if plot_out:\n",
    "        fig.write_html(f'{plot_out}.html')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e16461f7",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "import plotly.express as px\n",
    "\n",
    "new_pcs = pd.read_csv('${WORK_DIR}/FILTERED.merged_biallelic_ancestry_projected_new_pca.txt', sep='\\t') # Refer to (https://github.com/dvitale199/GenoTools) for generating this file\n",
    "new_pcs.head()\n",
    "\n",
    "labels = pd.read_csv('${WORK_DIR}/FILTERED.merged_biallelic_ancestry_umap_linearsvc_predicted_labels.txt',sep='\\t') # Refer to (https://github.com/dvitale199/GenoTools) for generating this file\n",
    "labels.head()\n",
    "\n",
    "merged_plot = pd.merge(new_pcs, labels, on=['FID', 'IID'])\n",
    "merged_plot.head()\n",
    "\n",
    "merged_plot_final = merged_plot[['FID', 'IID', 'PC1', 'PC2', 'PC3', 'label_y']]\n",
    "merged_plot_final.columns = ['FID', 'IID', 'PC1', 'PC2', 'PC3', 'label']\n",
    "merged_plot_final.head()\n",
    "\n",
    "plot_3d(merged_plot_final, color='label', plot_out='3d_plot_output_ADSP')\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "python/3.10",
   "language": "python",
   "name": "py3.10"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
